<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>THÈSE</title>
  <meta name="description" content="THÈSE">
  <meta name="generator" content="bookdown 0.5 and GitBook 2.6.7">

  <meta property="og:title" content="THÈSE" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="THÈSE" />
  
  
  

<meta name="author" content="Keurcien LUU">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="la-communalite.html">
<link rel="next" href="introgression-adaptative.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />









<style type="text/css">
div.sourceCode { overflow-x: auto; }
table.sourceCode, tr.sourceCode, td.lineNumbers, td.sourceCode {
  margin: 0; padding: 0; vertical-align: baseline; border: none; }
table.sourceCode { width: 100%; line-height: 100%; }
td.lineNumbers { text-align: right; padding-right: 4px; padding-left: 4px; color: #aaaaaa; border-right: 1px solid #aaaaaa; }
td.sourceCode { padding-left: 5px; }
code > span.kw { color: #007020; font-weight: bold; } /* Keyword */
code > span.dt { color: #902000; } /* DataType */
code > span.dv { color: #40a070; } /* DecVal */
code > span.bn { color: #40a070; } /* BaseN */
code > span.fl { color: #40a070; } /* Float */
code > span.ch { color: #4070a0; } /* Char */
code > span.st { color: #4070a0; } /* String */
code > span.co { color: #60a0b0; font-style: italic; } /* Comment */
code > span.ot { color: #007020; } /* Other */
code > span.al { color: #ff0000; font-weight: bold; } /* Alert */
code > span.fu { color: #06287e; } /* Function */
code > span.er { color: #ff0000; font-weight: bold; } /* Error */
code > span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
code > span.cn { color: #880000; } /* Constant */
code > span.sc { color: #4070a0; } /* SpecialChar */
code > span.vs { color: #4070a0; } /* VerbatimString */
code > span.ss { color: #bb6688; } /* SpecialString */
code > span.im { } /* Import */
code > span.va { color: #19177c; } /* Variable */
code > span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code > span.op { color: #666666; } /* Operator */
code > span.bu { } /* BuiltIn */
code > span.ex { } /* Extension */
code > span.pp { color: #bc7a00; } /* Preprocessor */
code > span.at { color: #7d9029; } /* Attribute */
code > span.do { color: #ba2121; font-style: italic; } /* Documentation */
code > span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code > span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code > span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#la-genetique-des-populations"><i class="fa fa-check"></i><b>1.1</b> La génétique des populations</a><ul>
<li class="chapter" data-level="1.1.1" data-path="introduction.html"><a href="introduction.html#levolution-comme-point-de-depart"><i class="fa fa-check"></i><b>1.1.1</b> L’évolution comme point de départ</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#a-lorigine-de-la-variabilite-genetique"><i class="fa fa-check"></i><b>1.2</b> À l’origine de la variabilité génétique</a><ul>
<li class="chapter" data-level="1.2.1" data-path="introduction.html"><a href="introduction.html#la-theorie-de-levolution"><i class="fa fa-check"></i><b>1.2.1</b> La théorie de l’évolution</a></li>
<li class="chapter" data-level="1.2.2" data-path="introduction.html"><a href="introduction.html#levolution-dune-theorie"><i class="fa fa-check"></i><b>1.2.2</b> L’évolution d’une théorie</a></li>
<li class="chapter" data-level="1.2.3" data-path="introduction.html"><a href="introduction.html#forces-evolutives"><i class="fa fa-check"></i><b>1.2.3</b> Forces évolutives</a></li>
<li class="chapter" data-level="1.2.4" data-path="introduction.html"><a href="introduction.html#adaptation-locale"><i class="fa fa-check"></i><b>1.2.4</b> Adaptation locale</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#donnees-de-polymorphismes-genetiques"><i class="fa fa-check"></i><b>1.3</b> Données de polymorphismes génétiques</a><ul>
<li class="chapter" data-level="1.3.1" data-path="introduction.html"><a href="introduction.html#des-donnees-en-grande-dimension"><i class="fa fa-check"></i><b>1.3.1</b> Des données en grande dimension</a></li>
<li class="chapter" data-level="1.3.2" data-path="introduction.html"><a href="introduction.html#les-marqueurs-genetiques"><i class="fa fa-check"></i><b>1.3.2</b> Les marqueurs génétiques</a></li>
<li class="chapter" data-level="1.3.3" data-path="introduction.html"><a href="introduction.html#encodage-des-donnees-genetiques"><i class="fa fa-check"></i><b>1.3.3</b> Encodage des données génétiques</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="introduction.html"><a href="introduction.html#les-motivations-de-la-these"><i class="fa fa-check"></i><b>1.4</b> Les motivations de la thèse</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html"><i class="fa fa-check"></i><b>2</b> Adaptation locale</a><ul>
<li class="chapter" data-level="2.1" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#letat-de-lart-pour-les-scans-genomiques"><i class="fa fa-check"></i><b>2.1</b> L’état de l’art pour les scans génomiques</a><ul>
<li class="chapter" data-level="2.1.1" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#modeles-demographiques"><i class="fa fa-check"></i><b>2.1.1</b> Modèles démographiques</a></li>
<li class="chapter" data-level="2.1.2" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#lindice-de-fixation"><i class="fa fa-check"></i><b>2.1.2</b> L’indice de fixation</a></li>
<li class="chapter" data-level="2.1.3" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#test-de-lewontin-krakauer"><i class="fa fa-check"></i><b>2.1.3</b> Test de Lewontin-Krakauer</a></li>
<li class="chapter" data-level="2.1.4" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#le-modele-f"><i class="fa fa-check"></i><b>2.1.4</b> Le modèle <span class="math inline">\(F\)</span></a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#lanalyse-en-composantes-principales-en-genetique-des-populations"><i class="fa fa-check"></i><b>2.2</b> L’Analyse en Composantes Principales en génétique des populations</a><ul>
<li class="chapter" data-level="2.2.1" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#principe-de-lanalyse-en-composantes-principales"><i class="fa fa-check"></i><b>2.2.1</b> Principe de l’Analyse en Composantes Principales</a></li>
<li class="chapter" data-level="2.2.2" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#apparentement-genetique-interindividuel"><i class="fa fa-check"></i><b>2.2.2</b> Apparentement génétique interindividuel</a></li>
<li class="chapter" data-level="2.2.3" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#applications-en-genetique-des-populations"><i class="fa fa-check"></i><b>2.2.3</b> Applications en génétique des populations</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="statistiques-de-test-basees-sur-lanalyse-en-composantes-principales.html"><a href="statistiques-de-test-basees-sur-lanalyse-en-composantes-principales.html"><i class="fa fa-check"></i><b>3</b> Statistiques de test basées sur l’Analyse en Composantes Principales</a></li>
<li class="chapter" data-level="4" data-path="la-communalite.html"><a href="la-communalite.html"><i class="fa fa-check"></i><b>4</b> La communalité</a><ul>
<li class="chapter" data-level="4.1" data-path="la-communalite.html"><a href="la-communalite.html#contributions"><i class="fa fa-check"></i><b>4.1</b> Contributions</a></li>
<li class="chapter" data-level="4.2" data-path="la-communalite.html"><a href="la-communalite.html#resume"><i class="fa fa-check"></i><b>4.2</b> Résumé</a></li>
<li class="chapter" data-level="4.3" data-path="la-communalite.html"><a href="la-communalite.html#article-1"><i class="fa fa-check"></i><b>4.3</b> Article 1</a><ul>
<li class="chapter" data-level="" data-path="la-communalite.html"><a href="la-communalite.html#abstract"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="la-communalite.html"><a href="la-communalite.html#significance-statement"><i class="fa fa-check"></i>Significance Statement</a></li>
<li class="chapter" data-level="" data-path="la-communalite.html"><a href="la-communalite.html#introduction-1"><i class="fa fa-check"></i>Introduction</a></li>
<li class="chapter" data-level="" data-path="la-communalite.html"><a href="la-communalite.html#new-method"><i class="fa fa-check"></i>New Method</a></li>
<li class="chapter" data-level="" data-path="la-communalite.html"><a href="la-communalite.html#results"><i class="fa fa-check"></i>Results</a></li>
<li class="chapter" data-level="" data-path="la-communalite.html"><a href="la-communalite.html#discussion"><i class="fa fa-check"></i>Discussion</a></li>
<li class="chapter" data-level="" data-path="la-communalite.html"><a href="la-communalite.html#materials-and-methods"><i class="fa fa-check"></i>Materials and Methods</a></li>
<li class="chapter" data-level="" data-path="la-communalite.html"><a href="la-communalite.html#acknowledgments"><i class="fa fa-check"></i>Acknowledgments</a></li>
<li class="chapter" data-level="" data-path="la-communalite.html"><a href="la-communalite.html#la-communalite-vue-comme-une-generalisation-de-lindice-de-fixation"><i class="fa fa-check"></i>La communalité vue comme une généralisation de l’indice de fixation</a></li>
<li class="chapter" data-level="4.3.1" data-path="la-communalite.html"><a href="la-communalite.html#limites-de-la-communalite"><i class="fa fa-check"></i><b>4.3.1</b> Limites de la communalité</a></li>
<li class="chapter" data-level="4.3.2" data-path="la-communalite.html"><a href="la-communalite.html#ajouter-le-choix-loadings-vs-z-scores"><i class="fa fa-check"></i><b>4.3.2</b> Ajouter le choix loadings vs z-scores</a></li>
<li class="chapter" data-level="4.3.3" data-path="la-communalite.html"><a href="la-communalite.html#ajouter-le-choix-mcd-vs-classique-vs-ogk"><i class="fa fa-check"></i><b>4.3.3</b> Ajouter le choix MCD vs classique vs OGK</a></li>
<li class="chapter" data-level="" data-path="la-communalite.html"><a href="la-communalite.html#controle-du-taux-de-fausse-decouverte"><i class="fa fa-check"></i>Contrôle du taux de fausse découverte</a></li>
<li class="chapter" data-level="4.3.4" data-path="la-communalite.html"><a href="la-communalite.html#bootstrap-acp"><i class="fa fa-check"></i><b>4.3.4</b> Bootstrap ACP</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="distance-robuste-de-mahalanobis.html"><a href="distance-robuste-de-mahalanobis.html"><i class="fa fa-check"></i><b>5</b> Distance robuste de Mahalanobis</a><ul>
<li class="chapter" data-level="5.1" data-path="distance-robuste-de-mahalanobis.html"><a href="distance-robuste-de-mahalanobis.html#article-2"><i class="fa fa-check"></i><b>5.1</b> Article 2</a><ul>
<li class="chapter" data-level="" data-path="distance-robuste-de-mahalanobis.html"><a href="distance-robuste-de-mahalanobis.html#abstract-1"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="distance-robuste-de-mahalanobis.html"><a href="distance-robuste-de-mahalanobis.html#introduction-2"><i class="fa fa-check"></i>Introduction</a></li>
<li class="chapter" data-level="" data-path="distance-robuste-de-mahalanobis.html"><a href="distance-robuste-de-mahalanobis.html#statistical-and-computational-approach"><i class="fa fa-check"></i>Statistical and computational approach</a></li>
<li class="chapter" data-level="" data-path="distance-robuste-de-mahalanobis.html"><a href="distance-robuste-de-mahalanobis.html#materials-and-methods-1"><i class="fa fa-check"></i>Materials and methods</a></li>
<li class="chapter" data-level="" data-path="distance-robuste-de-mahalanobis.html"><a href="distance-robuste-de-mahalanobis.html#results-1"><i class="fa fa-check"></i>Results</a></li>
<li class="chapter" data-level="" data-path="distance-robuste-de-mahalanobis.html"><a href="distance-robuste-de-mahalanobis.html#discussion-1"><i class="fa fa-check"></i>Discussion</a></li>
<li class="chapter" data-level="" data-path="distance-robuste-de-mahalanobis.html"><a href="distance-robuste-de-mahalanobis.html#acknowledgements"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="distance-robuste-de-mahalanobis.html"><a href="distance-robuste-de-mahalanobis.html#data-accessibility"><i class="fa fa-check"></i>Data accessibility</a></li>
<li class="chapter" data-level="5.1.1" data-path="distance-robuste-de-mahalanobis.html"><a href="distance-robuste-de-mahalanobis.html#estimation-robuste-de-la-matrice-de-covariance"><i class="fa fa-check"></i><b>5.1.1</b> Estimation robuste de la matrice de covariance</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html"><i class="fa fa-check"></i><b>6</b> Introgression adaptative</a><ul>
<li class="chapter" data-level="6.0.1" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#coefficients-de-metissage-globaux-et-locaux"><i class="fa fa-check"></i><b>6.0.1</b> Coefficients de métissage globaux et locaux</a></li>
<li class="chapter" data-level="6.0.2" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#lien-entre-analyse-en-composantes-principales-et-metissage-global."><i class="fa fa-check"></i><b>6.0.2</b> Lien entre Analyse en Composantes Principales et métissage global.</a></li>
<li class="chapter" data-level="6.0.3" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#analyse-en-composantes-principales-locale"><i class="fa fa-check"></i><b>6.0.3</b> Analyse en Composantes Principales locale</a></li>
<li class="chapter" data-level="6.0.4" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#sensibilite-a-limputation-des-donnees-manquantes"><i class="fa fa-check"></i><b>6.0.4</b> Sensibilité à l’imputation des données manquantes</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="introgression-adaptative-1.html"><a href="introgression-adaptative-1.html"><i class="fa fa-check"></i><b>7</b> Introgression adaptative</a><ul>
<li class="chapter" data-level="7.1" data-path="introgression-adaptative-1.html"><a href="introgression-adaptative-1.html#simulations"><i class="fa fa-check"></i><b>7.1</b> Simulations</a><ul>
<li class="chapter" data-level="7.1.1" data-path="introgression-adaptative-1.html"><a href="introgression-adaptative-1.html#donnees-de-peupliers"><i class="fa fa-check"></i><b>7.1.1</b> Données de peupliers</a></li>
<li class="chapter" data-level="7.1.2" data-path="introgression-adaptative-1.html"><a href="introgression-adaptative-1.html#generation-aleatoire-dindividus-hybrides"><i class="fa fa-check"></i><b>7.1.2</b> Génération aléatoire d’individus hybrides</a></li>
<li class="chapter" data-level="7.1.3" data-path="introgression-adaptative-1.html"><a href="introgression-adaptative-1.html#simulations-a-partir-de-ms-et-seq-gen"><i class="fa fa-check"></i><b>7.1.3</b> Simulations à partir de ms et Seq-Gen</a></li>
<li class="chapter" data-level="7.1.4" data-path="introgression-adaptative-1.html"><a href="introgression-adaptative-1.html#resultats-de-la-comparaison-des-logiciels"><i class="fa fa-check"></i><b>7.1.4</b> Résultats de la comparaison des logiciels</a></li>
</ul></li>
<li class="chapter" data-level="7.2" data-path="introgression-adaptative-1.html"><a href="introgression-adaptative-1.html#article-3"><i class="fa fa-check"></i><b>7.2</b> Article 3</a><ul>
<li class="chapter" data-level="" data-path="introgression-adaptative-1.html"><a href="introgression-adaptative-1.html#introduction-3"><i class="fa fa-check"></i>Introduction</a></li>
<li class="chapter" data-level="" data-path="introgression-adaptative-1.html"><a href="introgression-adaptative-1.html#materials-and-methods-2"><i class="fa fa-check"></i>Materials and methods</a></li>
<li class="chapter" data-level="" data-path="introgression-adaptative-1.html"><a href="introgression-adaptative-1.html#results-2"><i class="fa fa-check"></i>Results</a></li>
<li class="chapter" data-level="" data-path="introgression-adaptative-1.html"><a href="introgression-adaptative-1.html#discussion-2"><i class="fa fa-check"></i>Discussion</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="8" data-path="aspect-computationnel.html"><a href="aspect-computationnel.html"><i class="fa fa-check"></i><b>8</b> Aspect computationnel</a><ul>
<li class="chapter" data-level="8.1" data-path="aspect-computationnel.html"><a href="aspect-computationnel.html#donnees-dentree"><i class="fa fa-check"></i><b>8.1</b> Données d’entrée</a></li>
<li class="chapter" data-level="8.2" data-path="aspect-computationnel.html"><a href="aspect-computationnel.html#memory-mapping"><i class="fa fa-check"></i><b>8.2</b> Memory-mapping</a></li>
<li class="chapter" data-level="8.3" data-path="aspect-computationnel.html"><a href="aspect-computationnel.html#interface-shiny"><i class="fa fa-check"></i><b>8.3</b> Interface Shiny</a></li>
<li class="chapter" data-level="8.4" data-path="aspect-computationnel.html"><a href="aspect-computationnel.html#acp-et-donnees-manquantes"><i class="fa fa-check"></i><b>8.4</b> ACP et données manquantes</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="conclusion.html"><a href="conclusion.html"><i class="fa fa-check"></i>Conclusion</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="details.html"><a href="details.html"><i class="fa fa-check"></i><b>A</b> Details</a></li>
<li class="chapter" data-level="B" data-path="supplementary-information.html"><a href="supplementary-information.html"><i class="fa fa-check"></i><b>B</b> Supplementary Information</a><ul>
<li class="chapter" data-level="" data-path="supplementary-information.html"><a href="supplementary-information.html#article-1-1"><i class="fa fa-check"></i>Article 1 </a></li>
<li class="chapter" data-level="" data-path="supplementary-information.html"><a href="supplementary-information.html#article-2-1"><i class="fa fa-check"></i>Article 2</a></li>
<li class="chapter" data-level="" data-path="supplementary-information.html"><a href="supplementary-information.html#article-3-1"><i class="fa fa-check"></i>Article 3</a></li>
</ul></li>
<li class="chapter" data-level="C" data-path="r-python.html"><a href="r-python.html"><i class="fa fa-check"></i><b>C</b> R &amp; Python</a><ul>
<li class="chapter" data-level="C.1" data-path="r-python.html"><a href="r-python.html#simulations-et-modeles-demographiques"><i class="fa fa-check"></i><b>C.1</b> Simulations et modèles démographiques</a><ul>
<li class="chapter" data-level="C.1.1" data-path="r-python.html"><a href="r-python.html#modele-en-ile"><i class="fa fa-check"></i><b>C.1.1</b> Modèle en île</a></li>
<li class="chapter" data-level="C.1.2" data-path="r-python.html"><a href="r-python.html#modele-de-divergence"><i class="fa fa-check"></i><b>C.1.2</b> Modèle de divergence</a></li>
</ul></li>
<li class="chapter" data-level="C.2" data-path="details.html"><a href="details.html#details"><i class="fa fa-check"></i><b>C.2</b> Détails</a><ul>
<li class="chapter" data-level="C.2.1" data-path="r-python.html"><a href="r-python.html#donnees-manquantes"><i class="fa fa-check"></i><b>C.2.1</b> Données manquantes</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="bibliographie.html"><a href="bibliographie.html"><i class="fa fa-check"></i>Bibliographie</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">THÈSE</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="distance-robuste-de-mahalanobis" class="section level1">
<h1><span class="header-section-number">Chapter 5</span> Distance robuste de Mahalanobis</h1>
<p>Nous avons vu dans la partie précédente que la communalité avait tendance à accorder beaucoup d’importance aux premières composantes principales, du fait de la pondération de chaque composante par la proportion de variance expliquée. Pour remédier à cela, la statistique <span class="math inline">\(h^{\prime}\)</span> a été introduite. La distribution des <em>loadings</em> n’étant pas gaussienne, du fait de leur normalisation (<span class="math inline">\(\sum_{j=1}^p V_{jk}^2 = 1\)</span>), il n’est pas clair que la statistique <span class="math inline">\(h^{\prime}\)</span> suive une distribution connue.</p>
<p>Ainsi, ne pas normaliser les loadings permet de produire des <span class="math inline">\(t\)</span>-scores, si l’on considère que les loadings non-normalisés résultent de la régression de la matrice <span class="math inline">\(G\)</span> par les scores de l’ACP.</p>
<span class="math display" id="eq:svd-G">\[\begin{equation}
  G = U \Sigma V^T
  \tag{5.1}
\end{equation}\]</span>
<p><span class="math inline">\(V\)</span>, la matrice des loadings, peut être vue comme la solution du problème d’optimisation sous contrainte suivant :</p>
<span class="math display" id="eq:minimization-G">\[\begin{equation}
  \min_{V} ||G - U \Sigma V^T||_{2} \\
  \sum_{j=1}^p V_{jk}^2 = 1, \;  k = 1, \dots, K
  \tag{5.2}
\end{equation}\]</span>
<p>En retirant cette contrainte, le problème de minimisation est équivalent à un problème de régression linéaire multiple, dont la distribution des coefficients de régression est bien mieux connue. La solution au problème de régression linéaire multiple est explicite :</p>
<span class="math display" id="eq:linReg-solution">\[\begin{equation}
  \beta = U(U^T U)^{-1}U^TG
  \tag{5.3}
\end{equation}\]</span>
<p>Ce qui dans notre cas donne, grâce à l’orthonormalité de <span class="math inline">\(U\)</span> :</p>
<span class="math display" id="eq:linReg-solution-simplifiee">\[\begin{equation}
  \beta = UU^TG
  \tag{5.4}
\end{equation}\]</span>
<p>Ici, nous cherchons à tester l’hypothèse <span class="math inline">\(\beta_j = 0\)</span> contre l’hypothèse <span class="math inline">\(\beta_j \neq 0\)</span>, pour chaque <span class="math inline">\(\beta_j\)</span>, nous pouvons écrire le <span class="math inline">\(z\)</span>-score de la régrssion associé :</p>
<span class="math display" id="eq:linReg-zscore">\[\begin{equation}
  z_j = \frac{\beta_j}{\sqrt{\frac{||G_j - U_k \beta_j||}{n - p - 1}}}
  \tag{5.5}
\end{equation}\]</span>
<div id="article-2" class="section level2">
<h2><span class="header-section-number">5.1</span> Article 2</h2>
<p><img src="figure/molecol.png" width="\textwidth" style="display: block; margin: auto;" /></p>
<div id="abstract-1" class="section level3 unnumbered">
<h3>Abstract</h3>
<p>The R package <em>pcadapt</em> performs genome scans to detect genes under selection based on population genomic data. It assumes that candidate markers are outliers with respect to how they are related to population structure. Because population structure is ascertained with principal component analysis, the package is fast and works with large-scale data. It can handle missing data and pooled sequencing data. By contrast to population-based approaches, the package handle admixed individuals and does not require grouping individuals into populations. Since its first release, <em>pcadapt</em> has evolved in terms of both statistical approach and software implementation. We present results obtained with robust Mahalanobis distance, which is a new statistic for genome scans available in the 2.0 and later versions of the package. When hierarchical population structure occurs, Mahalanobis distance is more powerful than the communality statistic that was implemented in the first version of the package. Using simulated data, we compare <em>pcadapt</em> to other computer programs for genome scans (<em>BayeScan</em>, <em>hapflk</em>, <em>OutFLANK</em>, <em>sNMF</em>). We find that the proportion of false discoveries is around a nominal false discovery rate set at 10% with the exception of <em>BayeScan</em> that generates 40% of false discoveries. We also find that the power of <em>BayeScan</em> is severely impacted by the presence of admixed individuals whereas <em>pcadapt</em> is not impacted. Last, we find that <em>pcadapt</em> and <em>hapflk</em> are the most powerful in scenarios of population divergence and range expansion. Because <em>pcadapt</em> handles next-generation sequencing data, it is a valuable tool for data analysis in molecular ecology.</p>
</div>
<div id="introduction-2" class="section level3 unnumbered">
<h3>Introduction</h3>
<p>Looking for variants with unexpectedly large differences of allele frequencies between populations is a common approach to detect signals of natural selection <span class="citation">(Lewontin &amp; Krakauer, <a href="#ref-lewontin1973distribution">1973</a>)</span>. When variants confer a selective advantage in the local environment, allele frequency changes are triggered by natural selection leading to unexpectedly large differences of allele frequencies between populations. To detect variants with large differences of allele frequencies, numerous test statistics have been proposed, which are usually based on chi-square approximations of <span class="math inline">\(F_{ST}\)</span>-related test statistics <span class="citation">(François, Martins, Caye, &amp; Schoville, <a href="#ref-franccois2016controlling">2016</a>)</span>.</p>
<p>Statistical approaches for detecting selection should address several challenges. The first challenge is to account for hierarchical population structure that arises when genetic differentiation between populations is not identical between all pairs of populations. Statistical tests based on <span class="math inline">\(F_{ST}\)</span> that do not account for hierarchical structure, when it occurs, generate a large excess of false-positive loci <span class="citation">(Bierne et al., <a href="#ref-bierne2013pervasive">2013</a>; Excoffier, Hofer, &amp; Foll, <a href="#ref-excoffier2009detecting">2009</a>)</span>.</p>
<p>A second challenge arises because approaches based on <span class="math inline">\(F_{ST}\)</span>-related measures require to group individuals into populations, although defining populations is a difficult task <span class="citation">(Waples &amp; Gaggiotti, <a href="#ref-waples2006invited">2006</a>)</span>. Individual sampling may not be population based but based on more continuous sampling schemes <span class="citation">(Lotterhos &amp; Whitlock, <a href="#ref-lotterhos2015relative">2015</a>)</span>. Additionally assigning an admixed individual to a single population involves some arbitrariness because different regions of its genome might come from different populations <span class="citation">(Pritchard, Stephens, &amp; Donnelly, <a href="#ref-pritchard2000inference">2000</a>)</span>. Several individual-based methods of genome scans have already been proposed to address this challenge and they are based on related techniques of multivariate analysis including principal component analysis (PCA), factor models and non-negative matrix factorization <span class="citation">(G. Chen et al., <a href="#ref-chen2016eigengwas">2016</a>; Duforet-Frebourg et al., <a href="#ref-duforet2014genome">2014</a>, <a href="#ref-duforet2015detecting">2015</a>; Galinsky et al., <a href="#ref-galinsky2016fast">2016</a>; Hao, Song, &amp; Storey, <a href="#ref-hao2015probabilistic">2015</a>; Martins, Caye, Luu, Blum, &amp; Francois, <a href="#ref-martins2016identifying">2016</a>)</span>.</p>
<p>The last challenge arises from the nature of multilocus data sets generated from next-generation sequencing platforms. Because data sets are massive with a large number of molecular markers, Monte Carlo methods usually implemented in Bayesian statistics may be prohibitively slow <span class="citation">(Lange, Papp, Sinsheimer, &amp; Sobel, <a href="#ref-lange2014next">2014</a>)</span>. Additionally, next-generation sequencing data may contain a substantial proportion of missing data that should be accounted for <span class="citation">(B. Arnold, Corbett-Detig, Hartl, &amp; Bomblies, <a href="#ref-arnold2013radseq">2013</a>; Gautier et al., <a href="#ref-gautier2013effect">2013</a>)</span>.</p>
<p>To address the aforementioned challenges, we have developed the computer program <em>pcadapt</em> and the R package <em>pcadapt</em>. The computer program <em>pcadapt</em> is now deprecated and the R package only is maintained. <em>pcadapt</em> assumes that markers excessively related to population structure are candidates for local adaptation. Since its first release, <em>pcadapt</em> has substantially evolved in terms of both statistical approach and implementation (Table <a href="distance-robuste-de-mahalanobis.html#tab:table1">5.1</a>).</p>

<table>
<caption><span id="tab:table1">Table 5.1: </span>Summary of the different statistical methods and implementations of <em>pcadapt</em>. Pop. structure stands for population structure and dist. stands for distance</caption>
<thead>
<tr class="header">
<th align="left">Test statistic</th>
<th align="left">Pop. structure</th>
<th align="left">Language</th>
<th align="left">Command line</th>
<th align="left">Versions of the R package</th>
<th align="left">References</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Bayes factor</td>
<td align="left">Factor model</td>
<td align="left">C</td>
<td align="left">PCAdapt</td>
<td align="left">NA</td>
<td align="left">Duforet-Frebourg et al. (2014)</td>
</tr>
<tr class="even">
<td align="left">Communality</td>
<td align="left">PCA</td>
<td align="left">C and R</td>
<td align="left">PCAdapt fast</td>
<td align="left">1. x</td>
<td align="left">Duforet-Frebourg et al. (2016)</td>
</tr>
<tr class="odd">
<td align="left">Mahalanobis dist.</td>
<td align="left">PCA</td>
<td align="left">R</td>
<td align="left">NA</td>
<td align="left">2. x and 3. x</td>
<td align="left">This study</td>
</tr>
</tbody>
</table>
<p>The first release of <em>pcadapt</em> was a command line computer program written in C. It implemented a Monte Carlo approach based on a Bayesian factor model <span class="citation">(Duforet-Frebourg et al., <a href="#ref-duforet2014genome">2014</a>)</span>. The test statistic for outlier detection was a Bayes factor. Because Monte Carlo methods can be computationally prohibitive with massive NGS data, we then developed an alternative approach based on PCA. The first statistic based on PCA was the communality statistic, which measures the percentage of variation of a single-nucleotide polymorphism (SNP) explained by the first <span class="math inline">\(K\)</span> principal components <span class="citation">(Duforet-Frebourg et al., <a href="#ref-duforet2015detecting">2015</a>)</span>. It was initially implemented with a command line computer program (the <em>pcadapt fast</em> command) before being implemented in the <em>pcadapt</em> R package. We do not maintain C versions of <em>pcadapt</em> anymore. The whole analysis that goes from reading genotype files to detecting outlier SNPs can now be performed in R <span class="citation">(Team, <a href="#ref-team2015r">2015</a>)</span>.</p>
<p>The 2.0 and following versions of the R package implement a more powerful statistic for genome scans. The test statistic is a robust Mahalanobis distance. A vector containing <span class="math inline">\(K\)</span> <span class="math inline">\(z\)</span>-scores measures to what extent a SNP is related to the first <span class="math inline">\(K\)</span> principal components. The Mahalanobis distance is then computed for each SNP to detect outliers for which the vector of <span class="math inline">\(z\)</span>-scores does not follow the distribution of the main bulk of points. The term robust refers to the fact that the estimators of the mean and of the covariance matrix of <span class="math inline">\(z\)</span>, which are required to compute the Mahalanobis distances, are not sensitive to the presence of outliers in the data set <span class="citation">(Maronna &amp; Zamar, <a href="#ref-maronna2002robust">2002</a>)</span>. In the following, we provide a comparison of statistical power that shows that Mahalanobis distance provides more powerful genome scans compared with the communality statistic and with the Bayes factor that were implemented in previous versions of <em>pcadapt</em>.</p>
<p>In addition to comparing the different test statistics that were implemented in <em>pcadapt</em>, we compare statistic performance obtained with the 3.0 version of <em>pcadapt</em> and with other computer programs for genome scans. We use simulated data to compare computer programs in terms of false discovery rate (FDR) and statistical power. We consider data simulated under different demographic models including island model, divergence model and range expansion. To perform comparisons, we include programs that require to group individuals into populations: <em>BayeScan</em> <span class="citation">(Foll &amp; Gaggiotti, <a href="#ref-foll2008genome">2008</a>)</span>, the <span class="math inline">\(F_{LK}\)</span> statistic as implemented in the <em>hapflk</em> computer program <span class="citation">(Bonhomme et al., <a href="#ref-bonhomme2010detecting">2010</a>)</span>, and <em>OutFLANK</em> that provides a robust estimation of the null distribution of a <span class="math inline">\(F_{ST}\)</span> test statistic <span class="citation">(Whitlock &amp; Lotterhos, <a href="#ref-whitlock2015reliable">2015</a>)</span>. We additionally consider the <em>sNMF</em> computer program that implements another individual-based test statistic for genome scans <span class="citation">(Frichot, Mathieu, Trouillon, Bouchard, &amp; François, <a href="#ref-frichot2014fast">2014</a>; Martins et al., <a href="#ref-martins2016identifying">2016</a>)</span>.</p>
</div>
<div id="statistical-and-computational-approach" class="section level3 unnumbered">
<h3>Statistical and computational approach</h3>
<div id="input-data" class="section level4 unnumbered">
<h4>Input data</h4>
<p>The R package can handle different data formats for the genotype data matrix. In the version 3.0 that is currently available on CRAN, the package can handle genotype data files in the <em>vcf</em>, <em>ped</em> and <em>lfmm</em> formats. In addition, the package can also handle a <em>pcadapt</em> format, which is a text file where each line contains the allele counts of all individuals at a given locus. When reading a genotype data matrix with the <em>read.pcadapt</em> function, a <em>.pcadapt</em> file is generated, which contains the genotype data in the <em>pcadapt</em> format.</p>
</div>
<div id="choosing-the-number-of-principal-components" class="section level4 unnumbered">
<h4>Choosing the number of principal components</h4>
<p>In the following, we denote by <span class="math inline">\(n\)</span> the number of individuals, by <span class="math inline">\(p\)</span> the number of genetic markers and by <span class="math inline">\(G\)</span> the genotype matrix that is composed of <span class="math inline">\(n\)</span> lines and <span class="math inline">\(p\)</span> columns. The genotypic information at locus <span class="math inline">\(j\)</span> for individual <span class="math inline">\(i\)</span> is encoded by the allele count <span class="math inline">\(G_{ij}\)</span>, <span class="math inline">\(1 \leq i \leq n\)</span> and <span class="math inline">\(1 \leq j \leq p\)</span>, which is a value in 0,1 for haploid species and in 0,1,2 for diploid species. The current 3.0.2 version of the package can handle haploid and diploid data only.</p>
<p>First, we normalize the genotype matrix columnwise. For diploid data, we consider the usual normalization in population genomics where <span class="math inline">\(\tilde{G}_{ij} = (G_{ij} - p_j)/(2 \times p_j(1 - p_j))^{1/2}\)</span>, and <span class="math inline">\(p_j\)</span> denotes the minor allele frequency for locus <span class="math inline">\(j\)</span> <span class="citation">(N. Patterson et al., <a href="#ref-patterson2006population">2006</a>)</span>. The normalization for haploid data is similar except that the denominator is given by <span class="math inline">\((p_j(1 - p_j))^{1/2}\)</span></p>
<p>Then, we use the normalized genotype matrix math formula to ascertain population structure with PCA <span class="citation">(N. Patterson et al., <a href="#ref-patterson2006population">2006</a>)</span>. The number of principal components to consider is denoted <span class="math inline">\(K\)</span> and is a parameter that should be chosen by the user. In order to choose <span class="math inline">\(K\)</span>, we recommend to consider the graphical approach based on the scree plot <span class="citation">(Jackson, <a href="#ref-jackson1993stopping">1993</a>)</span>. The scree plot displays the eigenvalues of the covariance matrix <span class="math inline">\(\Omega\)</span> in descending order. Up to a constant, eigenvalues are proportional to the proportion of variance explained by each principal component. The eigenvalues that correspond to random variation lie on a straight line whereas the ones corresponding to population structure depart from the line. We recommend to use Cattell’s rule that states that components corresponding to eigenvalues to the left of the straight line should be kept <span class="citation">(Cattell, <a href="#ref-cattell1966scree">1966</a>)</span>.</p>
</div>
<div id="test-statistic" class="section level4 unnumbered">
<h4>Test statistic</h4>
<p>We now detail how the package computes the test statistic. We consider multiple linear regressions by regressing each of the <span class="math inline">\(p\)</span> SNPs by the <span class="math inline">\(K\)</span> principal components <span class="math inline">\(X_1, \dots, X_K\)</span></p>
<span class="math display" id="eq:multiple-reg">\[\begin{equation}
  G_j = \sum_{k=1}^K \beta_{jk} X_k + \epsilon_j, \; j = 1, \dots, p,
  \tag{5.6}
\end{equation}\]</span>
<p>where <span class="math inline">\(\beta_{jk}\)</span> is the regression coefficient corresponding to the <span class="math inline">\(j\)</span>-th SNP regressed by the <span class="math inline">\(k\)</span>-th principal component, and <span class="math inline">\(\epsilon_j\)</span> is the residuals vector. To summarize the result of the regression analysis for the <span class="math inline">\(j\)</span>-th SNP, we return a vector of <span class="math inline">\(z\)</span>-scores <span class="math inline">\(z_j = (z_{j1}, \dots, z_{jK})\)</span> where <span class="math inline">\(z_{jk}\)</span> corresponds to the <span class="math inline">\(z\)</span>-score obtained when regressing the <span class="math inline">\(j\)</span>-th SNP by the <span class="math inline">\(k\)</span>-th principal component.</p>
<p>The next step is to look for outliers based on the vector of <span class="math inline">\(z\)</span>-scores. We consider a classical approach in multivariate analysis for outlier detection. The test statistic is a robust Mahalanobis distance <span class="math inline">\(D\)</span> defined as</p>
<span class="math display" id="eq:maha-def">\[\begin{equation}
  D_j^2 = (z_j - \bar{z})^T \Sigma^{-1} (z_j - \bar{z}),
  \tag{5.7}
\end{equation}\]</span>
<p>where <span class="math inline">\(\Sigma\)</span> is the <span class="math inline">\((K \times K)\)</span> covariance matrix of the <span class="math inline">\(z\)</span>-scores and <span class="math inline">\(\bar{z}\)</span> is the vector of the <span class="math inline">\(K\)</span> <span class="math inline">\(z\)</span>-score means <span class="citation">(Maronna &amp; Zamar, <a href="#ref-maronna2002robust">2002</a>)</span>. When <span class="math inline">\(K &gt; 1\)</span>, the covariance matrix <span class="math inline">\(\Sigma\)</span> is estimated with the orthogonalized Gnanadesikan–Kettenring method that is a robust estimate of the covariance able to handle large-scale data <span class="citation">(Maronna &amp; Zamar, <a href="#ref-maronna2002robust">2002</a>)</span> (<em>covRob</em> function of the <em>robust</em> R package). When <span class="math inline">\(K = 1\)</span>, the variance is estimated with another robust estimate (<em>cov.rob</em> function of the <em>MASS</em> R package).</p>
</div>
<div id="genomic-inflation-factor" class="section level4 unnumbered">
<h4>Genomic inflation factor</h4>
<p>To perform multiple hypothesis testing, Mahalanobis distances should be transformed into <span class="math inline">\(P\)</span>-values. If the <span class="math inline">\(z\)</span>-scores were truly multivariate Gaussian, the Mahalanobis distances <span class="math inline">\(D\)</span> should be chi-square distributed with <span class="math inline">\(K\)</span> degrees of freedom. However, as usual for genome scans, there are confounding factors that inflate values of the test statistic and that would lead to an excess of false positives <span class="citation">(François et al., <a href="#ref-franccois2016controlling">2016</a>)</span>. To account for the inflation of test statistics, we divide Mahalanobis distances by a constant <span class="math inline">\(\lambda\)</span> to obtain a statistic that can be approximated by a chi-square distribution with <span class="math inline">\(K\)</span> degrees of freedom. This constant is estimated by the genomic inflation factor defined here as the median of the Mahalanobis distances divided by the median of the chi-square distribution with <span class="math inline">\(K\)</span> degrees of freedom <span class="citation">(Devlin &amp; Roeder, <a href="#ref-devlin1999genomic">1999</a>)</span>.</p>
</div>
<div id="control-of-the-false-discovery-rate-fdr" class="section level4 unnumbered">
<h4>Control of the false discovery rate (FDR)</h4>
<p>Once <span class="math inline">\(P\)</span>-values are computed, there is a problem of decision-making related to the choice of a threshold for <span class="math inline">\(P\)</span>-values. We recommend to use the FDR approach where the objective is to provide a list of candidate genes with an expected proportion of false discoveries smaller than a specified value. For controlling the FDR, we consider the <span class="math inline">\(q\)</span>-value procedure as implemented in the <em>qvalue</em> R package that is less conservative than Bonferroni or Benjamini–Hochberg correction <span class="citation">(Storey &amp; Tibshirani, <a href="#ref-storey2003statistical">2003</a>)</span>. The <em>qvalue</em> R package transforms the <span class="math inline">\(P\)</span>-values into <span class="math inline">\(q\)</span>-values and the user can control a specified value <span class="math inline">\(\alpha\)</span> of FDR by considering as candidates the SNPs with <span class="math inline">\(q\)</span>-values smaller than <span class="math inline">\(\alpha\)</span>.</p>
</div>
<div id="numerical-computations-1" class="section level4 unnumbered">
<h4>Numerical computations</h4>
<p>PCA is performed using a C routine that allows to compute scores and eigenvalues efficiently with minimum RAM access <span class="citation">(Duforet-Frebourg et al., <a href="#ref-duforet2015detecting">2015</a>)</span>. Computing the covariance matrix <span class="math inline">\(\Omega\)</span> is the most computationally demanding part. To provide a fast routine, we compute the <span class="math inline">\(n \times n\)</span> covariance matrix <span class="math inline">\(\Omega\)</span> instead of the much larger <span class="math inline">\(p \times p\)</span> covariance matrix. We compute the covariance <span class="math inline">\(\Omega\)</span> incrementally by adding small storable covariance blocks successively. Multiple linear regression is then solved directly by computing an explicit solution, written as a matrix product. Using the fact that the <span class="math inline">\((n,K)\)</span> score matrix <span class="math inline">\(X\)</span> is orthogonal, the <span class="math inline">\((p,K)\)</span> matrix <span class="math inline">\(\beta\)</span> of regression coefficients is given by <span class="math inline">\(G^TX\)</span> and the <span class="math inline">\((n,p)\)</span> matrix of residuals is given by <span class="math inline">\(G-XX^TG\)</span>. The <span class="math inline">\(z\)</span>-scores are then computed using the standard formula for multiple regression</p>
<span class="math display" id="eq:z-def">\[\begin{equation}
  z_{jk} = \hat{\beta}_{jk}\sqrt{\frac{\sum_{i=1}^n x_{ik}^2}{\sigma_j^2}}
  \tag{5.8}
\end{equation}\]</span>
<p>where <span class="math inline">\(\sigma_j^2\)</span> is an estimate of the residual variance for the <span class="math inline">\(j^{th}\)</span> SNP, and <span class="math inline">\(x_{ik}\)</span> is the score of the <span class="math inline">\(k^{th}\)</span> principal component for the <span class="math inline">\(i^{th}\)</span> individual.</p>
</div>
<div id="missing-data" class="section level4 unnumbered">
<h4>Missing data</h4>
<p>Missing data should be accounted for when computing principal components and when computing the matrix of <span class="math inline">\(z\)</span>-scores. There are many methods to account for missing data in PCA, and we consider the pairwise covariance approach <span class="citation">(Dray &amp; Josse, <a href="#ref-dray2015principal">2015</a>)</span>. It consists in estimating the covariance between each pair of individuals using only the markers that are available for both individuals. To compute <span class="math inline">\(z\)</span>-scores, we account for missing data in formula <a href="distance-robuste-de-mahalanobis.html#eq:z-def">(5.8)</a>. The term in the numerator <span class="math inline">\(\sum_{i=1}^n x_{ik}^2\)</span> depends on the quantity of missing data. If there are no missing data, it is equal to 1 by definition of the scores obtained with PCA. As the quantity of missing data grows, this term and the <span class="math inline">\(z\)</span>-score decrease such that it becomes more difficult to detect outlier markers.</p>
</div>
<div id="pooled-sequence-data" class="section level4 unnumbered">
<h4>Pooled sequence data</h4>
<p>When data are sequenced in pool, the Mahalanobis distance is based on the matrix of allele frequency computed in each pool instead of the matrix of z-scores.</p>
</div>
</div>
<div id="materials-and-methods-1" class="section level3 unnumbered">
<h3>Materials and methods</h3>
<div id="simulated-data" class="section level4 unnumbered">
<h4>Simulated data</h4>
<p>We simulated SNPs under an island model, under a divergence model and we downloaded simulations of range expansion <span class="citation">(Lotterhos &amp; Whitlock, <a href="#ref-lotterhos2015relative">2015</a>)</span>. All data we simulated were composed of 3 populations, each of them containing 50 sampled diploid individuals (Table <a href="distance-robuste-de-mahalanobis.html#tab:table2">5.2</a>). SNPs were simulated assuming no linkage disequilibrium. SNPs with minor allele frequencies lower than 5% were discarded from the data sets. The mean <span class="math inline">\(F_{ST}\)</span> for each simulation was comprised between 5% and 10%. Using the simulations based on an island and a divergence model, we also created data sets composed of admixed individuals. We assumed that an instantaneous admixture event occurs at the present time so that all sampled individuals are the results of this admixture event. Admixed individuals were generated by drawing randomly admixture proportions using a Dirichlet distribution of parameter <span class="math inline">\((\alpha, \alpha, \alpha)\)</span> (<span class="math inline">\(\alpha\)</span> ranging from 0.005 to 1 depending on the simulation).</p>

<table>
<caption><span id="tab:table2">Table 5.2: </span>Summary of the simulations. The table above shows the average number of individuals, of SNPs, of adaptive markers and the total number of simulations per scenario</caption>
<thead>
<tr class="header">
<th></th>
<th align="right">Individuals</th>
<th align="right">SNPs</th>
<th align="right">Adaptive SNPs</th>
<th align="right">Simulations</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Island model</td>
<td align="right">150</td>
<td align="right">472</td>
<td align="right">27</td>
<td align="right">35</td>
</tr>
<tr class="even">
<td>Divergence model</td>
<td align="right">150</td>
<td align="right">3000</td>
<td align="right">100</td>
<td align="right">6</td>
</tr>
<tr class="odd">
<td>Island model (hybrids)</td>
<td align="right">150</td>
<td align="right">472</td>
<td align="right">30</td>
<td align="right">27</td>
</tr>
<tr class="even">
<td>Divergence model (hybrids)</td>
<td align="right">150</td>
<td align="right">3000</td>
<td align="right">100</td>
<td align="right">9</td>
</tr>
<tr class="odd">
<td>Range expansion</td>
<td align="right">1200</td>
<td align="right">9999</td>
<td align="right">99</td>
<td align="right">6</td>
</tr>
</tbody>
</table>
</div>
<div id="island-model-1" class="section level4 unnumbered">
<h4>Island model</h4>
<p>We used <em>ms</em> to create simulations under an island model (Fig. <a href="supplementary-information.html#fig:FigureSI1">B.1</a>). We set a lower migration rate for the 50 adaptive SNPs compared with the 950 neutral ones to mimic diversifying selection <span class="citation">(Bazin et al., <a href="#ref-bazin2010likelihood">2010</a>)</span>. For a given locus, migration from population <span class="math inline">\(i\)</span> to <span class="math inline">\(j\)</span> was specified by choosing a value of the effective migration rate that is set to <span class="math inline">\(M_{\text{neutral}} = 10\)</span> for neutral SNPs and to <span class="math inline">\(M_{\text{adaptive}}\)</span> for adaptive ones. We simulated 35 data sets in the island model with different strengths of selection, where the strength of selection corresponds to the ratio <span class="math inline">\(M_{\text{neutral}} / M_{\text{adaptive}}\)</span> that varies from 10 to 1000. The <em>ms</em> command lines for neutral and adaptive SNPs are given by (<span class="math inline">\(M_{\text{adaptive}} = 0.01\)</span> and <span class="math inline">\(M_{\text{neutral}} = 10\)</span>).</p>
<div class="sourceCode"><pre class="sourceCode bash"><code class="sourceCode bash"><span class="kw">./ms</span> 300 950 -s 1 -I 3 100 100 100

<span class="kw">-ma</span> x 10 10 10 x 10 10 10 x

<span class="kw">./ms</span> 300 50 -s 1 -I 3 100 100 100

<span class="kw">-ma</span> x 0.01 0.01 0.01 x 0.01 0.01 0.01 x</code></pre></div>
</div>
<div id="divergence-model-1" class="section level4 unnumbered">
<h4>Divergence model</h4>
<p>To perform simulations under a divergence model, we used the package <em>simuPOP</em>, which is an individual-based population genetic simulation environment <span class="citation">(B. Peng &amp; Kimmel, <a href="#ref-peng2005simupop">2005</a>)</span>. We assumed that an ancestral panmictic population evolved during 20 generations before splitting into two subpopulations. The second subpopulation then split into subpopulations 2 and 3 at time <span class="math inline">\(T &gt; 20\)</span>. All 3 subpopulations continued to evolve until 200 generations have been reached, without migration between them (Figure <a href="supplementary-information.html#fig:FigureSI1">B.1</a>). A total of 50 diploid individuals were sampled in each population. Selection only occurred in the branch associated with population 2 and selection was simulated by assuming an additive model (fitness is equal to <span class="math inline">\(1 - 2s\)</span>, <span class="math inline">\(1 - s\)</span>, <span class="math inline">\(1\)</span> depending on the genotypes). We simulated a total of 3000 SNPs comprising of 100 adaptive ones for which the selection coefficient is of <span class="math inline">\(s = 0.1\)</span>.</p>
</div>
<div id="range-expansion" class="section level4 unnumbered">
<h4>Range expansion</h4>
<p>We downloaded in the <em>Dryad Digital Repository</em> six simulations of range expansion with two glacial refugia <span class="citation">(Lotterhos &amp; Whitlock, <a href="#ref-lotterhos2015relative">2015</a>)</span>. Adaptation occurred during the recolonization phase of the species range from the two refugia. We considered six different simulated data with 30 populations and a number of sampled individuals per location that varies from 20 to 60.</p>
</div>
<div id="parameter-settings-for-the-different-computer-programs" class="section level4 unnumbered">
<h4>Parameter settings for the different computer programs</h4>
<p>When using <em>hapflk</em>, we set <span class="math inline">\(K = 1\)</span> that corresponds to the computation of the <span class="math inline">\(F_{LK}\)</span> statistic. When using <em>BayeScan</em> and <em>OutFLANK</em>, we used the default parameter values. For <em>sNMF</em>, we used <span class="math inline">\(K = 3\)</span> for the island and divergence model and <span class="math inline">\(K = 5\)</span> for range expansion as indicated by the cross-entropy criterion. The regularization parameter of <em>sNMF</em> was set to <span class="math inline">\(\alpha = 1000\)</span>. For <em>sNMF</em> and <em>hapflk</em>, we used the genomic inflation factor to recalibrate <span class="math inline">\(p\)</span>-values. When using population-based methods with admixed individuals, we assigned each individual to the population with maximum amount of ancestry.</p>
</div>
</div>
<div id="results-1" class="section level3 unnumbered">
<h3>Results</h3>
<div id="choosing-the-number-of-principal-components-1" class="section level4 unnumbered">
<h4>Choosing the number of principal components</h4>
<p>We evaluate Cattell’s graphical rule to choose the number of principal components. For the island and divergence model, the choice of <span class="math inline">\(K\)</span> is evident (Fig. <a href="distance-robuste-de-mahalanobis.html#fig:choice">5.1</a>). For <span class="math inline">\(K \geq 3\)</span>, the eigenvalues follow a straight line. As a consequence, Cattell’s rule indicates <span class="math inline">\(K = 2\)</span>, which is expected because there are three populations <span class="citation">(N. Patterson et al., <a href="#ref-patterson2006population">2006</a>)</span>. For the model of range expansion, applying Cattell’s rule to choose <span class="math inline">\(K\)</span> is more difficult (Fig. <a href="distance-robuste-de-mahalanobis.html#fig:choice">5.1</a>). Ideally, the eigenvalues that correspond to random variation lie on a straight line whereas the ones corresponding to population structure depart from the line. However, there is no obvious point at which eigenvalues depart from the straight line. Choosing a value of <span class="math inline">\(K\)</span> between 5 and 8 is compatible with Cattell’s rule. Using the package qvalue to control 10% of FDR, we find that the actual proportion of false discoveries as well as statistical power is weakly impacted when varying the number of principal components from <span class="math inline">\(K = 5\)</span> to <span class="math inline">\(K = 8\)</span> (Figure <a href="supplementary-information.html#fig:FigureSI2">B.2</a>).</p>


<div class="figure" style="text-align: center"><span id="fig:choice"></span>
<img src="thesis_files/figure-html/choice-1.png" alt="Determining \(K\) with the scree plot. To choose \(K\), we recommend to use Cattell’s rule that states that components corresponding to eigenvalues to the left of the straight line should be kept. According to Cattell’s rule, the eigenvalues that correspond to random variation lie on the straight line whereas the ones corresponding to population structure depart from the line. For the island and divergence model, the choice of \(K\) is evident. For the model or range expansion, a value of \(K\) between 5 and 8 is compatible with Cattell’s rule." width="\textwidth" />
<p class="caption">
Figure 5.1: Determining <span class="math inline">\(K\)</span> with the scree plot. To choose <span class="math inline">\(K\)</span>, we recommend to use Cattell’s rule that states that components corresponding to eigenvalues to the left of the straight line should be kept. According to Cattell’s rule, the eigenvalues that correspond to random variation lie on the straight line whereas the ones corresponding to population structure depart from the line. For the island and divergence model, the choice of <span class="math inline">\(K\)</span> is evident. For the model or range expansion, a value of <span class="math inline">\(K\)</span> between 5 and 8 is compatible with Cattell’s rule.
</p>
</div>
</div>
<div id="an-example-of-genome-scans-performed-with-pcadapt" class="section level4 unnumbered">
<h4>An example of genome scans performed with <em>pcadapt</em></h4>
<p>To provide an example of results, we apply <em>pcadapt</em> with <span class="math inline">\(K = 6\)</span> in the model of range expansion. Population structure captured by the first two principal components is displayed in Fig. <a href="distance-robuste-de-mahalanobis.html#fig:scanex">5.2</a>. <span class="math inline">\(P\)</span>-values are well calibrated because they are distributed as a mixture of a uniform distribution and of a peaky distribution around 0, which corresponds to outlier loci (Fig. <a href="distance-robuste-de-mahalanobis.html#fig:scanex">5.2</a>). Using a FDR threshold of 10% with the <em>qvalue</em> package, we find 122 outliers among 10 000 SNPs, resulting in 23% actual false discoveries and a power of 95%.</p>

<div class="figure" style="text-align: center"><span id="fig:scanex"></span>
<img src="thesis_files/figure-html/scanex-1.png" alt="Population structure (first 2 principal components) and distribution of \(p\)-values obtained with pcadapt for a simulation of range expansion. \(P\)-values are well calibrated because they are distributed as a mixture of a uniform distribution and of a peaky distribution around 0, which corresponds to outlier loci. In the left panel, each colour corresponds to individuals sampled from the same population." width="\textwidth" />
<p class="caption">
Figure 5.2: Population structure (first 2 principal components) and distribution of <span class="math inline">\(p\)</span>-values obtained with <em>pcadapt</em> for a simulation of range expansion. <span class="math inline">\(P\)</span>-values are well calibrated because they are distributed as a mixture of a uniform distribution and of a peaky distribution around 0, which corresponds to outlier loci. In the left panel, each colour corresponds to individuals sampled from the same population.
</p>
</div>
</div>
<div id="control-of-the-false-discovery-rate" class="section level4">
<h4><span class="header-section-number">5.1.0.1</span> Control of the false discovery rate</h4>
<p>We evaluate to what extent using the packages <em>pcadapt</em> and <em>qvalue</em> control a FDR set at 10% (Fig. <a href="distance-robuste-de-mahalanobis.html#fig:jitter">5.3</a>). All SNPs with a <span class="math inline">\(q\)</span>-value smaller than 10% were considered as candidate SNPs. For the island model, we find that the proportion of false discoveries is 8% and it increases to 10% when including admixture. For the divergence model, the proportion of false discoveries is 11% and it increases to 22% when including admixture. The largest proportion of false discoveries is obtained under range expansion and is equal to 25%.</p>

<div class="figure" style="text-align: center"><span id="fig:jitter"></span>
<img src="thesis_files/figure-html/jitter-1.png" alt="Control of the FDR for different computer programs for genome scans. We find that the median proportion of false discoveries is around the nominal FDR set at 10% (6% for hapflk, 11% for both OutFLANK and pcadapt and 19% for sNMF) with the exception of BayeScan that generates 41% of false discoveries." width="\textwidth" />
<p class="caption">
Figure 5.3: Control of the FDR for different computer programs for genome scans. We find that the median proportion of false discoveries is around the nominal FDR set at 10% (6% for <em>hapflk</em>, 11% for both <em>OutFLANK</em> and <em>pcadapt</em> and 19% for <em>sNMF</em>) with the exception of <em>BayeScan</em> that generates 41% of false discoveries.
</p>
</div>
<p>We then evaluate the proportion of false discoveries obtained with <em>BayeScan</em>, <em>hapflk</em>, <em>OutFLANK</em> and <em>sNMF</em> (Fig. <a href="distance-robuste-de-mahalanobis.html#fig:jitter">5.3</a>). We find that <em>hapflk</em> is the most conservative approach (FDR = 6%) followed by <em>OutFLANK</em> and <em>pcadapt</em> (FDR = 11%). The computer program <em>sNMF</em> is more liberal (FDR = 19%) and <em>BayeScan</em> generates the largest proportion of false discoveries (FDR = 41%). When not recalibrating the p-values of <em>hapflk</em>, we find that the test is even more conservative (results not shown). For all programs, the range expansion scenario is the one that generates the largest proportion of false discoveries. Proportion of false discoveries under range expansion ranges from 22% (<em>OutFLANK</em>) to 93% (<em>BayeScan</em>).</p>
</div>
<div id="statistical-power" class="section level4 unnumbered">
<h4>Statistical power</h4>
<p>To provide a fair comparison between methods and computer programs, we compare statistical power for equal values of the observed proportion of false discoveries. Then we compute statistical power averaged over observed proportion of false discoveries ranging from 0% to 50%.</p>
<p>We first compare statistical power obtained with the different statistical methods that have been implemented in <em>pcadapt</em> (Table <a href="distance-robuste-de-mahalanobis.html#tab:table1">5.1</a>). For the island model, Bayes factor, communality statistic and Mahalanobis distance have similar power (Fig. <a href="distance-robuste-de-mahalanobis.html#fig:bayes-comm-maha">5.4</a>). For the divergence model, the power obtained with Mahalanobis distance is 20% whereas the power obtained with the communality statistic and with the Bayes factor is, respectively, 4% and 2% (Fig. <a href="distance-robuste-de-mahalanobis.html#fig:bayes-comm-maha">5.4</a>). Similarly, for range expansion, the power obtained with Mahalanobis distance is 46% whereas the power obtained with the communality statistic and with the Bayes factor is 34% and 13%. We additionally investigate to what extent increasing sample size in each population from 20 to 60 individuals affects power. For range expansion, the power obtained with the Mahalanobis distance hardly changes ranging from 44% to 47%. However, the power obtained with the other two statistics changes importantly. The power obtained with the communality statistic increases from 27% to 39% when increasing the sample size and the power obtained with the Bayes factor increases from 0% to 44%.</p>
<p></p>

<div class="figure" style="text-align: center"><span id="fig:bayes-comm-maha"></span>
<img src="thesis_files/figure-html/bayes-comm-maha-1.png" alt="Bayes factor corresponds to the test statistic implemented in the Bayesian version of pcadapt (Duforet-Frebourg et al., 2014); the communality statistic was the default statistic in version 1.x of the R package pcadapt (Duforet-Frebourg et al., 2015), and Mahalanobis distances are available since the release of the 2.0 version of the package. When there is hierarchical population structure (divergence model and range expansion), the Mahalanobis distance provides more powerful genome scans compared with the test statistic previously implemented in pcadapt. The abbreviation dist. stands for distance. Statistical power is averaged over the observed proportion of false discoveries (ranging between 0% and 50%)." width="\textwidth" />
<p class="caption">
Figure 5.4: Bayes factor corresponds to the test statistic implemented in the Bayesian version of <em>pcadapt</em> <span class="citation">(Duforet-Frebourg et al., <a href="#ref-duforet2014genome">2014</a>)</span>; the communality statistic was the default statistic in version 1.x of the R package <em>pcadapt</em> <span class="citation">(Duforet-Frebourg et al., <a href="#ref-duforet2015detecting">2015</a>)</span>, and Mahalanobis distances are available since the release of the 2.0 version of the package. When there is hierarchical population structure (divergence model and range expansion), the Mahalanobis distance provides more powerful genome scans compared with the test statistic previously implemented in pcadapt. The abbreviation dist. stands for distance. Statistical power is averaged over the observed proportion of false discoveries (ranging between 0% and 50%).
</p>
</div>
<p>Then we describe our comparison of computer programs for genome scans. For the simulations obtained with the island model where there is no hierarchical population structure, the statistical power is similar for all programs (Figure <a href="supplementary-information.html#fig:FigureSI3">B.3</a> and <a href="supplementary-information.html#fig:FigureSI4">B.4</a>). Including admixed individuals hardly changes their statistical power (Figure <a href="supplementary-information.html#fig:FigureSI3">B.3</a>).</p>
<p>Then, we compare statistical power in a divergence model where adaptation took place in one of the external branches of the population divergence tree. The programs <em>pcadapt</em> and <em>hapflk</em>, which account for hierarchical population structure, as well as <em>BayeScan</em> are the most powerful in that setting (Fig. <a href="distance-robuste-de-mahalanobis.html#fig:power-div">5.5</a> and Figure <a href="supplementary-information.html#fig:FigureSI5">B.5</a>). The values of power in decreasing order are of 23% for <em>BayeScan</em>, of 20% for <em>pcadapt</em>, of 17% for <em>hapflk</em>, of 7% for <em>sNMF</em> and of 1% for <em>OutFLANK</em>. When including admixed individuals, the power of <em>hapflk</em> and of <em>pcadapt</em> hardly decreases whereas the power of <em>BayeScan</em> decreases to 6% (Fig. <a href="distance-robuste-de-mahalanobis.html#fig:power-div">5.5</a>).</p>

<div class="figure" style="text-align: center"><span id="fig:power-div"></span>
<img src="thesis_files/figure-html/power-div-1.png" alt="Statistical power averaged over the expected proportion of false discoveries (ranging between 0% and 50%) for the divergence model with three populations. We assume that adaptation took place in an external branch that follows the most recent population divergence event." width="\textwidth" />
<p class="caption">
Figure 5.5: Statistical power averaged over the expected proportion of false discoveries (ranging between 0% and 50%) for the divergence model with three populations. We assume that adaptation took place in an external branch that follows the most recent population divergence event.
</p>
</div>
<p>The last model we consider is the model of range expansion. The package <em>pcadapt</em> is the most powerful approach in this setting (Fig. <a href="distance-robuste-de-mahalanobis.html#fig:power-rexp">5.6</a> and <a href="supplementary-information.html#fig:FigureSI6">B.6</a>). Other computer programs also discover many true-positive loci with the exception of <em>BayeScan</em> that provides no true discovery when the observed FDR is smaller than 50% (Fig. <a href="distance-robuste-de-mahalanobis.html#fig:power-rexp">5.6</a> and <a href="supplementary-information.html#fig:FigureSI6">B.6</a>). The values of power in decreasing order are of 46% for <em>pcadapt</em>, of 41% for <em>hapflk</em>, of 37% for <em>OutFLANK</em>, of 30% for <em>sNMF</em> and of 0% for <em>BayeScan</em>.</p>

<div class="figure" style="text-align: center"><span id="fig:power-rexp"></span>
<img src="thesis_files/figure-html/power-rexp-1.png" alt="Statistical power averaged over the expected proportion of false discoveries (ranging between 0% and 50%) for a range expansion model with two refugia. Adaptation took place during the recolonization event." width="\textwidth" />
<p class="caption">
Figure 5.6: Statistical power averaged over the expected proportion of false discoveries (ranging between 0% and 50%) for a range expansion model with two refugia. Adaptation took place during the recolonization event.
</p>
</div>
</div>
<div id="running-time-of-the-different-computer-programs" class="section level4 unnumbered">
<h4>Running time of the different computer programs</h4>
<p>Last, we compare running times. The characteristics of the computer we used to perform comparisons are the following: OSX El Capitan 10.11.3, 2,5 GHz Intel Core i5, 8 Go 1600 MHz DDR3. We discard <em>BayeScan</em> as it is too time-consuming. For instance, running <em>BayeScan</em> on a genotype matrix containing 150 individuals and 3000 SNPs takes 9h whereas it takes less than one second with <em>pcadapt</em>. The different programs were run on genotype matrices containing 300 individuals and from 500 to 50 000 SNPs. <em>OutFLANK</em> is the computer program for which the runtime increases the most rapidly with the number of markers. <em>OutFLANK</em> takes around 25 min to analyse 50 000 SNPs (Figure <a href="supplementary-information.html#fig:FigureSI7">B.7</a>). For the other 3 computer programs (<em>hapflk</em>, <em>pcadapt</em>, <em>sNMF</em>), analysing 50 000 SNPs takes less than 3 min.</p>
</div>
</div>
<div id="discussion-1" class="section level3 unnumbered">
<h3>Discussion</h3>
<p>The R package <em>pcadapt</em> implements a fast method to perform genome scans with next-generation sequencing data. It can handle data sets where population structure is continuous or data sets containing admixed individuals. It can handle missing data as well as pooled sequencing data. The 2.0 and later versions of the R package implements a robust Mahalanobis distance as a test statistic. When hierarchical population structure occurs, Mahalanobis distance provides more powerful genome scans compared with the communality statistic that was implemented in the first version of the package <span class="citation">(Duforet-Frebourg et al., <a href="#ref-duforet2015detecting">2015</a>)</span>. In the divergence model, adaptation occurs along an external branch of the divergence tree that corresponds to the second principal component. When outlier SNPs are not related to the first principal component, the Mahalanobis distance provides a better ranking of the SNPs compared with the communality statistic.</p>
<p>Simulations show that the R package <em>pcadapt</em> compares favourably to other computer programs for genome scans. When data were simulated under an island model, population structure is not hierarchical because genetic differentiation is the same for all pairs of populations. Statistical power and control of the FDR were similar for all computer programs. In the presence of hierarchical population structure (divergence model) where genetic differentiation varies between pairs of populations, the ranking of the SNPs depends on the computer program. <em>pcadapt</em> and <em>hapflk</em> provide the most powerful scans whether or not simulations include admixed individuals. <em>OutFLANK</em> implements a <span class="math inline">\(F_{ST}\)</span> statistic and because adaptation does not correspond to the most differentiated populations, it fails to capture adaptive SNPs (Fig. <a href="distance-robuste-de-mahalanobis.html#fig:power-div">5.5</a>) <span class="citation">(Bonhomme et al., <a href="#ref-bonhomme2010detecting">2010</a>; Duforet-Frebourg et al., <a href="#ref-duforet2015detecting">2015</a>)</span>. <em>BayeScan</em> does not assume equal differentiation between all pairs of populations, which may explain why it has a good statistical power for the divergence model. However, its statistical power is severely impacted by the presence of admixed individuals because its power decreases from 24% to 6% (Fig. <a href="distance-robuste-de-mahalanobis.html#fig:power-div">5.5</a>). Understanding why <em>BayeScan</em> is severely impacted by admixture is out of the scope of this study. In the range expansion model, <em>BayeScan</em> returns many null q-values (between 376 and 809 SNPs of 9899 neutral and 100 adaptive SNPs) such that the observed FDR is always larger than 50%. Overall, we find that <em>pcadapt</em> and <em>hapflk</em> provide comparable statistical power. They provide optimal or near optimal ranking of the SNPs in different scenarios including hierarchical population structure and admixed individuals. The main difference between the two computer programs concerns the control of the FDR because hapflk is found to be more conservative.</p>
<p>Because NGS data become more and more massive, careful numerical implementation is crucial. There are different options to implement PCA and <em>pcadapt</em> uses a numerical routine based on the computation of the covariance matrix <span class="math inline">\(\Omega\)</span>. The algorithmic complexity to compute the covariance matrix is proportional to <span class="math inline">\(pn^2\)</span> where <span class="math inline">\(p\)</span> is the number of markers and <span class="math inline">\(n\)</span> is the number of individuals. The computation of the first <span class="math inline">\(K\)</span> eigenvectors of the covariance matrix <span class="math inline">\(\Omega\)</span> has a complexity proportional to <span class="math inline">\(n^3\)</span>. This second step is usually more rapid than the computation of the covariance because the number of markers is usually large compared with the number of individuals. In brief, computing the covariance matrix <span class="math inline">\(\Omega\)</span> is by far the most costly operation when computing principal components. Although we have implemented PCA in C to obtain fast computations, an improvement in speed could be envisioned for future versions. When the number of individuals becomes large (e.g. <span class="math inline">\(n \geq 10 000\)</span>), there are faster algorithms to compute principal components <span class="citation">(Abraham &amp; Inouye, <a href="#ref-abraham2014fast">2014</a>; Halko, Martinsson, &amp; Tropp, <a href="#ref-halko2011finding">2011</a>)</span>. In addition to running time, numerical implementations also impact the effect of missing data on principal components <span class="citation">(Dray &amp; Josse, <a href="#ref-dray2015principal">2015</a>)</span>. Achieving a good trade-off between fast computations and accurate evaluation of population structure in the face of large amount of missing data is a challenge for modern numerical methods in molecular ecology.</p>
</div>
<div id="acknowledgements" class="section level3 unnumbered">
<h3>Acknowledgements</h3>
<p>This work has been supported by the LabEx PERSYVAL-Lab (ANR-11-LABX-0025-01) and the ANR AGRHUM project (ANR-14-CE02-0003-01). We want to thank two anonymous reviewers and Stéphane Dray for their critical reading of our manuscript.</p>
<p>K.L., E.B. and M.G.B.B. designed and performed the research.</p>
</div>
<div id="data-accessibility" class="section level3 unnumbered">
<h3>Data accessibility</h3>
<p>Island and divergence model data: doi: 10.5061/dryad.8290n.</p>
<p>Range expansion simulated data: doi: 10.5061/dryad.mh67v. Files:</p>
<p><code>2R_R30_1351142954_453_2_NumPops=30_NumInd=20</code></p>
<p><code>2R_R30_1351142954_453_2_NumPops=30_NumInd=60</code></p>
<p><code>2R_R30_1351142970_988_6_NumPops=30_NumInd=20</code></p>
<p><code>2R_R30_1351142970_988_6_NumPops=30_NumInd=60</code></p>
<p><code>2R_R30_1351142986_950_10_NumPops=30_NumInd=20</code></p>
<p><code>2R_R30_1351142986_950_10_NumPops=30_NumInd=60</code></p>
</div>
<div id="estimation-robuste-de-la-matrice-de-covariance" class="section level3">
<h3><span class="header-section-number">5.1.1</span> Estimation robuste de la matrice de covariance</h3>
<p><img src="thesis_files/figure-html/unnamed-chunk-7-1.png" width="\textwidth" style="display: block; margin: auto;" /></p>

</div>
</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-lewontin1973distribution">
<p>Lewontin, R., &amp; Krakauer, J. (1973). Distribution of gene frequency as a test of the theory of the selective neutrality of polymorphisms. <em>Genetics</em>, <em>74</em>(1), 175–195.</p>
</div>
<div id="ref-franccois2016controlling">
<p>François, O., Martins, H., Caye, K., &amp; Schoville, S. D. (2016). Controlling false discoveries in genome scans for selection. <em>Molecular Ecology</em>, <em>25</em>(2), 454–469.</p>
</div>
<div id="ref-bierne2013pervasive">
<p>Bierne, N., Roze, D., &amp; Welch, J. J. (2013). Pervasive selection or is it…? Why are fst outliers sometimes so frequent? <em>Molecular Ecology</em>, <em>22</em>(8), 2061–2064.</p>
</div>
<div id="ref-excoffier2009detecting">
<p>Excoffier, L., Hofer, T., &amp; Foll, M. (2009). Detecting loci under selection in a hierarchically structured population. <em>Heredity</em>, <em>103</em>(4), 285.</p>
</div>
<div id="ref-waples2006invited">
<p>Waples, R. S., &amp; Gaggiotti, O. (2006). INVITED review: What is a population? An empirical evaluation of some genetic methods for identifying the number of gene pools and their degree of connectivity. <em>Molecular Ecology</em>, <em>15</em>(6), 1419–1439.</p>
</div>
<div id="ref-lotterhos2015relative">
<p>Lotterhos, K. E., &amp; Whitlock, M. C. (2015). The relative power of genome scans to detect local adaptation depends on sampling design and statistical method. <em>Molecular Ecology</em>, <em>24</em>(5), 1031–1046.</p>
</div>
<div id="ref-pritchard2000inference">
<p>Pritchard, J. K., Stephens, M., &amp; Donnelly, P. (2000). Inference of population structure using multilocus genotype data. <em>Genetics</em>, <em>155</em>(2), 945–959.</p>
</div>
<div id="ref-chen2016eigengwas">
<p>Chen, G., Lee, S., Zhu, Z., Benyamin, B., &amp; Robinson, M. (2016). EigenGWAS: Finding loci under selection through genome-wide association studies of eigenvectors in structured populations. <em>Heredity</em>, <em>117</em>(1), 51.</p>
</div>
<div id="ref-duforet2014genome">
<p>Duforet-Frebourg, N., Bazin, E., &amp; Blum, M. G. (2014). Genome scans for detecting footprints of local adaptation using a bayesian factor model. <em>Molecular Biology and Evolution</em>, <em>31</em>(9), 2483–2495.</p>
</div>
<div id="ref-duforet2015detecting">
<p>Duforet-Frebourg, N., Luu, K., Laval, G., Bazin, E., &amp; Blum, M. G. (2015). Detecting genomic signatures of natural selection with principal component analysis: Application to the 1000 genomes data. <em>Molecular Biology and Evolution</em>, <em>33</em>(4), 1082–1093.</p>
</div>
<div id="ref-galinsky2016fast">
<p>Galinsky, K. J., Bhatia, G., Loh, P.-R., Georgiev, S., Mukherjee, S., Patterson, N. J., &amp; Price, A. L. (2016). Fast principal-component analysis reveals convergent evolution of adh1b in europe and east asia. <em>The American Journal of Human Genetics</em>, <em>98</em>(3), 456–472.</p>
</div>
<div id="ref-hao2015probabilistic">
<p>Hao, W., Song, M., &amp; Storey, J. D. (2015). Probabilistic models of genetic variation in structured populations applied to global human studies. <em>Bioinformatics</em>, <em>32</em>(5), 713–721.</p>
</div>
<div id="ref-martins2016identifying">
<p>Martins, H., Caye, K., Luu, K., Blum, M. G., &amp; Francois, O. (2016). Identifying outlier loci in admixed and in continuous populations using ancestral population differentiation statistics. <em>Molecular Ecology</em>, <em>25</em>(20), 5029–5042.</p>
</div>
<div id="ref-lange2014next">
<p>Lange, K., Papp, J. C., Sinsheimer, J. S., &amp; Sobel, E. M. (2014). Next-generation statistical genetics: Modeling, penalization, and optimization in high-dimensional data. <em>Annual Review of Statistics and Its Application</em>, <em>1</em>, 279–300.</p>
</div>
<div id="ref-arnold2013radseq">
<p>Arnold, B., Corbett-Detig, R. B., Hartl, D., &amp; Bomblies, K. (2013). RADseq underestimates diversity and introduces genealogical biases due to nonrandom haplotype sampling. <em>Molecular Ecology</em>, <em>22</em>(11), 3179–3190.</p>
</div>
<div id="ref-gautier2013effect">
<p>Gautier, M., Gharbi, K., Cezard, T., Foucaud, J., Kerdelhué, C., Pudlo, P., … Estoup, A. (2013). The effect of rad allele dropout on the estimation of genetic variation within and between populations. <em>Molecular Ecology</em>, <em>22</em>(11), 3165–3178.</p>
</div>
<div id="ref-team2015r">
<p>Team, R. C. (2015). R: A language and environment for statistical computing [internet]. vienna, austria: R foundation for statistical computing; 2014.</p>
</div>
<div id="ref-maronna2002robust">
<p>Maronna, R. A., &amp; Zamar, R. H. (2002). Robust estimates of location and dispersion for high-dimensional datasets. <em>Technometrics</em>, <em>44</em>(4), 307–317.</p>
</div>
<div id="ref-foll2008genome">
<p>Foll, M., &amp; Gaggiotti, O. (2008). A genome-scan method to identify selected loci appropriate for both dominant and codominant markers: A bayesian perspective. <em>Genetics</em>, <em>180</em>(2), 977–993.</p>
</div>
<div id="ref-bonhomme2010detecting">
<p>Bonhomme, M., Chevalet, C., Servin, B., Boitard, S., Abdallah, J., Blott, S., &amp; SanCristobal, M. (2010). Detecting selection in population trees: The lewontin and krakauer test extended. <em>Genetics</em>, <em>186</em>(1), 241–262.</p>
</div>
<div id="ref-whitlock2015reliable">
<p>Whitlock, M. C., &amp; Lotterhos, K. E. (2015). Reliable detection of loci responsible for local adaptation: Inference of a null model through trimming the distribution of f st. <em>The American Naturalist</em>, <em>186</em>(S1), S24–S36.</p>
</div>
<div id="ref-frichot2014fast">
<p>Frichot, E., Mathieu, F., Trouillon, T., Bouchard, G., &amp; François, O. (2014). Fast and efficient estimation of individual ancestry coefficients. <em>Genetics</em>, <em>196</em>(4), 973–983.</p>
</div>
<div id="ref-patterson2006population">
<p>Patterson, N., Price, A. L., &amp; Reich, D. (2006). Population structure and eigenanalysis. <em>PLoS Genetics</em>, <em>2</em>(12), e190.</p>
</div>
<div id="ref-jackson1993stopping">
<p>Jackson, D. A. (1993). Stopping rules in principal components analysis: A comparison of heuristical and statistical approaches. <em>Ecology</em>, <em>74</em>(8), 2204–2214.</p>
</div>
<div id="ref-cattell1966scree">
<p>Cattell, R. B. (1966). The scree test for the number of factors. <em>Multivariate Behavioral Research</em>, <em>1</em>(2), 245–276.</p>
</div>
<div id="ref-devlin1999genomic">
<p>Devlin, B., &amp; Roeder, K. (1999). Genomic control for association studies. <em>Biometrics</em>, <em>55</em>(4), 997–1004.</p>
</div>
<div id="ref-storey2003statistical">
<p>Storey, J. D., &amp; Tibshirani, R. (2003). Statistical significance for genomewide studies. <em>Proceedings of the National Academy of Sciences</em>, <em>100</em>(16), 9440–9445.</p>
</div>
<div id="ref-dray2015principal">
<p>Dray, S., &amp; Josse, J. (2015). Principal component analysis with missing values: A comparative survey of methods. <em>Plant Ecology</em>, <em>216</em>(5), 657–667.</p>
</div>
<div id="ref-bazin2010likelihood">
<p>Bazin, E., Dawson, K. J., &amp; Beaumont, M. A. (2010). Likelihood-free inference of population structure and local adaptation in a bayesian hierarchical model. <em>Genetics</em>, <em>185</em>(2), 587–602.</p>
</div>
<div id="ref-peng2005simupop">
<p>Peng, B., &amp; Kimmel, M. (2005). SimuPOP: A forward-time population genetics simulation environment. <em>Bioinformatics</em>, <em>21</em>(18), 3686–3687.</p>
</div>
<div id="ref-abraham2014fast">
<p>Abraham, G., &amp; Inouye, M. (2014). Fast principal component analysis of large-scale genome-wide data. <em>PloS One</em>, <em>9</em>(4), e93766.</p>
</div>
<div id="ref-halko2011finding">
<p>Halko, N., Martinsson, P.-G., &amp; Tropp, J. A. (2011). Finding structure with randomness: Probabilistic algorithms for constructing approximate matrix decompositions. <em>SIAM Review</em>, <em>53</em>(2), 217–288.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="la-communalite.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="introgression-adaptative.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"download": [["thesis.pdf", "PDF"], ["thesis.epub", "EPUB"], ["thesis.docx", "Word"]],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(script.src))
      script.src  = script.src.replace(/^https?:/, '');
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
