<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Application de l’Analyse en Composantes Principales pour étudier l’adaptation biologique en génomique des populations.</title>
  <meta name="description" content="Application de l’Analyse en Composantes Principales pour étudier l’adaptation biologique en génomique des populations.">
  <meta name="generator" content="bookdown 0.5 and GitBook 2.6.7">

  <meta property="og:title" content="Application de l’Analyse en Composantes Principales pour étudier l’adaptation biologique en génomique des populations." />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Application de l’Analyse en Composantes Principales pour étudier l’adaptation biologique en génomique des populations." />
  
  
  

<meta name="author" content="Keurcien LUU">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="details.html">
<link rel="next" href="r-python.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />









<style type="text/css">
div.sourceCode { overflow-x: auto; }
table.sourceCode, tr.sourceCode, td.lineNumbers, td.sourceCode {
  margin: 0; padding: 0; vertical-align: baseline; border: none; }
table.sourceCode { width: 100%; line-height: 100%; }
td.lineNumbers { text-align: right; padding-right: 4px; padding-left: 4px; color: #aaaaaa; border-right: 1px solid #aaaaaa; }
td.sourceCode { padding-left: 5px; }
code > span.kw { color: #007020; font-weight: bold; } /* Keyword */
code > span.dt { color: #902000; } /* DataType */
code > span.dv { color: #40a070; } /* DecVal */
code > span.bn { color: #40a070; } /* BaseN */
code > span.fl { color: #40a070; } /* Float */
code > span.ch { color: #4070a0; } /* Char */
code > span.st { color: #4070a0; } /* String */
code > span.co { color: #60a0b0; font-style: italic; } /* Comment */
code > span.ot { color: #007020; } /* Other */
code > span.al { color: #ff0000; font-weight: bold; } /* Alert */
code > span.fu { color: #06287e; } /* Function */
code > span.er { color: #ff0000; font-weight: bold; } /* Error */
code > span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
code > span.cn { color: #880000; } /* Constant */
code > span.sc { color: #4070a0; } /* SpecialChar */
code > span.vs { color: #4070a0; } /* VerbatimString */
code > span.ss { color: #bb6688; } /* SpecialString */
code > span.im { } /* Import */
code > span.va { color: #19177c; } /* Variable */
code > span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code > span.op { color: #666666; } /* Operator */
code > span.bu { } /* BuiltIn */
code > span.ex { } /* Extension */
code > span.pp { color: #bc7a00; } /* Preprocessor */
code > span.at { color: #7d9029; } /* Attribute */
code > span.do { color: #ba2121; font-style: italic; } /* Documentation */
code > span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code > span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code > span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#la-genetique-des-populations"><i class="fa fa-check"></i><b>1.1</b> La génétique des populations</a><ul>
<li class="chapter" data-level="1.1.1" data-path="introduction.html"><a href="introduction.html#levolution-comme-point-de-depart"><i class="fa fa-check"></i><b>1.1.1</b> L’évolution comme point de départ</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#a-lorigine-de-la-variabilite-genetique"><i class="fa fa-check"></i><b>1.2</b> À l’origine de la variabilité génétique</a><ul>
<li class="chapter" data-level="1.2.1" data-path="introduction.html"><a href="introduction.html#la-theorie-de-levolution"><i class="fa fa-check"></i><b>1.2.1</b> La théorie de l’évolution</a></li>
<li class="chapter" data-level="1.2.2" data-path="introduction.html"><a href="introduction.html#levolution-dune-theorie"><i class="fa fa-check"></i><b>1.2.2</b> L’évolution d’une théorie</a></li>
<li class="chapter" data-level="1.2.3" data-path="introduction.html"><a href="introduction.html#forces-evolutives"><i class="fa fa-check"></i><b>1.2.3</b> Forces évolutives</a></li>
<li class="chapter" data-level="1.2.4" data-path="introduction.html"><a href="introduction.html#adaptation-locale"><i class="fa fa-check"></i><b>1.2.4</b> Adaptation locale</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#donnees-de-polymorphismes-genetiques"><i class="fa fa-check"></i><b>1.3</b> Données de polymorphismes génétiques</a><ul>
<li class="chapter" data-level="1.3.1" data-path="introduction.html"><a href="introduction.html#des-donnees-en-grande-dimension"><i class="fa fa-check"></i><b>1.3.1</b> Des données en grande dimension</a></li>
<li class="chapter" data-level="1.3.2" data-path="introduction.html"><a href="introduction.html#les-marqueurs-genetiques"><i class="fa fa-check"></i><b>1.3.2</b> Les marqueurs génétiques</a></li>
<li class="chapter" data-level="1.3.3" data-path="introduction.html"><a href="introduction.html#encodage-des-donnees-genetiques"><i class="fa fa-check"></i><b>1.3.3</b> Encodage des données génétiques</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="introduction.html"><a href="introduction.html#les-motivations-de-la-these"><i class="fa fa-check"></i><b>1.4</b> Les motivations de la thèse</a><ul>
<li class="chapter" data-level="1.4.1" data-path="introduction.html"><a href="introduction.html#resultats-principaux-et-organisation-du-manuscript"><i class="fa fa-check"></i><b>1.4.1</b> Résultats principaux et organisation du manuscript</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html"><i class="fa fa-check"></i><b>2</b> Adaptation locale</a><ul>
<li class="chapter" data-level="2.1" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#letat-de-lart-pour-les-scans-genomiques"><i class="fa fa-check"></i><b>2.1</b> L’état de l’art pour les scans génomiques</a><ul>
<li class="chapter" data-level="2.1.1" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#modeles-demographiques"><i class="fa fa-check"></i><b>2.1.1</b> Modèles démographiques</a></li>
<li class="chapter" data-level="2.1.2" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#lindice-de-fixation"><i class="fa fa-check"></i><b>2.1.2</b> L’indice de fixation</a></li>
<li class="chapter" data-level="2.1.3" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#test-de-lewontin-krakauer"><i class="fa fa-check"></i><b>2.1.3</b> Test de Lewontin-Krakauer</a></li>
<li class="chapter" data-level="2.1.4" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#le-modele-f"><i class="fa fa-check"></i><b>2.1.4</b> Le modèle <span class="math inline">\(F\)</span></a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#lanalyse-en-composantes-principales-en-genetique-des-populations"><i class="fa fa-check"></i><b>2.2</b> L’Analyse en Composantes Principales en génétique des populations</a><ul>
<li class="chapter" data-level="2.2.1" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#principe-de-lanalyse-en-composantes-principales"><i class="fa fa-check"></i><b>2.2.1</b> Principe de l’Analyse en Composantes Principales</a></li>
<li class="chapter" data-level="2.2.2" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#apparentement-genetique-interindividuel"><i class="fa fa-check"></i><b>2.2.2</b> Apparentement génétique interindividuel</a></li>
<li class="chapter" data-level="2.2.3" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#applications-en-genetique-des-populations"><i class="fa fa-check"></i><b>2.2.3</b> Applications en génétique des populations</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#statistiques-de-test-basees-sur-lanalyse-en-composantes-principales"><i class="fa fa-check"></i><b>2.3</b> Statistiques de test basées sur l’Analyse en Composantes Principales</a><ul>
<li class="chapter" data-level="2.3.1" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#la-communalite"><i class="fa fa-check"></i><b>2.3.1</b> La communalité</a></li>
<li class="chapter" data-level="2.3.2" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#la-distance-robuste-de-mahalanobis"><i class="fa fa-check"></i><b>2.3.2</b> La distance robuste de Mahalanobis</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#article-1"><i class="fa fa-check"></i><b>2.4</b> Article 1</a><ul>
<li class="chapter" data-level="" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#abstract"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#significance-statement"><i class="fa fa-check"></i>Significance Statement</a></li>
<li class="chapter" data-level="" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#introduction-1"><i class="fa fa-check"></i>Introduction</a></li>
<li class="chapter" data-level="" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#new-method"><i class="fa fa-check"></i>New Method</a></li>
<li class="chapter" data-level="" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#results"><i class="fa fa-check"></i>Results</a></li>
<li class="chapter" data-level="" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#discussion"><i class="fa fa-check"></i>Discussion</a></li>
<li class="chapter" data-level="" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#materials-and-methods"><i class="fa fa-check"></i>Materials and Methods</a></li>
<li class="chapter" data-level="" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#acknowledgments"><i class="fa fa-check"></i>Acknowledgments</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#article-2"><i class="fa fa-check"></i><b>2.5</b> Article 2</a><ul>
<li class="chapter" data-level="" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#abstract-1"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#introduction-2"><i class="fa fa-check"></i>Introduction</a></li>
<li class="chapter" data-level="" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#statistical-and-computational-approach"><i class="fa fa-check"></i>Statistical and computational approach</a></li>
<li class="chapter" data-level="" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#materials-and-methods-1"><i class="fa fa-check"></i>Materials and methods</a></li>
<li class="chapter" data-level="" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#results-1"><i class="fa fa-check"></i>Results</a></li>
<li class="chapter" data-level="" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#discussion-1"><i class="fa fa-check"></i>Discussion</a></li>
<li class="chapter" data-level="" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#acknowledgements"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#data-accessibility"><i class="fa fa-check"></i>Data accessibility</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html"><i class="fa fa-check"></i><b>3</b> Introgression adaptative</a><ul>
<li class="chapter" data-level="3.1" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#introgression-par-metissage"><i class="fa fa-check"></i><b>3.1</b> Introgression par métissage</a><ul>
<li class="chapter" data-level="3.1.1" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#coefficients-de-metissage-locaux"><i class="fa fa-check"></i><b>3.1.1</b> Coefficients de métissage locaux</a></li>
<li class="chapter" data-level="3.1.2" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#statistique-de-test"><i class="fa fa-check"></i><b>3.1.2</b> Statistique de test</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#introgression-par-flux-de-genes"><i class="fa fa-check"></i><b>3.2</b> Introgression par flux de gènes</a><ul>
<li class="chapter" data-level="3.2.1" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#diversite-nucleotidique-par-paires-de-sequences"><i class="fa fa-check"></i><b>3.2.1</b> Diversité nucléotidique par paires de séquences</a></li>
<li class="chapter" data-level="3.2.2" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#le-modele-abba-baba"><i class="fa fa-check"></i><b>3.2.2</b> Le modèle ABBA-BABA</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#une-nouvelle-statistique-pour-les-scans-dintrogression"><i class="fa fa-check"></i><b>3.3</b> Une nouvelle statistique pour les scans d’introgression</a><ul>
<li class="chapter" data-level="3.3.1" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#analyse-en-composantes-principales-locale"><i class="fa fa-check"></i><b>3.3.1</b> Analyse en Composantes Principales locale</a></li>
<li class="chapter" data-level="3.3.2" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#resultats-principaux"><i class="fa fa-check"></i><b>3.3.2</b> Résultats principaux</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#article-3"><i class="fa fa-check"></i><b>3.4</b> Article 3</a><ul>
<li class="chapter" data-level="" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#scanning-genomes-for-adaptive-introgression-using-principal-component-analysis"><i class="fa fa-check"></i>Scanning genomes for adaptive introgression using principal component analysis</a></li>
<li class="chapter" data-level="" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#introduction-3"><i class="fa fa-check"></i>Introduction</a></li>
<li class="chapter" data-level="" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#a-pca-based-approach-to-detect-local-introgression"><i class="fa fa-check"></i>A PCA-based approach to detect local introgression</a></li>
<li class="chapter" data-level="" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#materials-and-methods-2"><i class="fa fa-check"></i>Materials and Methods</a></li>
<li class="chapter" data-level="" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#results-2"><i class="fa fa-check"></i>Results</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="aspect-computationnel.html"><a href="aspect-computationnel.html"><i class="fa fa-check"></i><b>4</b> Aspect computationnel</a><ul>
<li class="chapter" data-level="4.1" data-path="aspect-computationnel.html"><a href="aspect-computationnel.html#du-langage-c-au-langage-r"><i class="fa fa-check"></i><b>4.1</b> Du langage C au langage R</a></li>
<li class="chapter" data-level="4.2" data-path="aspect-computationnel.html"><a href="aspect-computationnel.html#du-calcul-de-la-matrice-de-covariance-a-lalgorithme-iram"><i class="fa fa-check"></i><b>4.2</b> Du calcul de la matrice de covariance à l’algorithme IRAM</a></li>
<li class="chapter" data-level="4.3" data-path="aspect-computationnel.html"><a href="aspect-computationnel.html#acp-et-valeurs-manquantes"><i class="fa fa-check"></i><b>4.3</b> ACP et valeurs manquantes</a><ul>
<li class="chapter" data-level="" data-path="aspect-computationnel.html"><a href="aspect-computationnel.html#algorithme-iram-et-donnees-manquantes"><i class="fa fa-check"></i>Algorithme IRAM et données manquantes</a></li>
<li class="chapter" data-level="" data-path="aspect-computationnel.html"><a href="aspect-computationnel.html#precision-de-la-svd-en-presence-de-donnees-manquantes"><i class="fa fa-check"></i>Précision de la SVD en présence de données manquantes</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="aspect-computationnel.html"><a href="aspect-computationnel.html#du-format-.pcadapt-au-format-.bed"><i class="fa fa-check"></i><b>4.4</b> Du format .pcadapt au format .bed</a></li>
<li class="chapter" data-level="4.5" data-path="aspect-computationnel.html"><a href="aspect-computationnel.html#interface-shiny"><i class="fa fa-check"></i><b>4.5</b> Interface Shiny</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="perspectives-et-conclusions.html"><a href="perspectives-et-conclusions.html"><i class="fa fa-check"></i><b>5</b> Perspectives et conclusions</a><ul>
<li class="chapter" data-level="5.1" data-path="perspectives-et-conclusions.html"><a href="perspectives-et-conclusions.html#substitution-de-lanalyse-en-composantes-principales"><i class="fa fa-check"></i><b>5.1</b> Substitution de l’Analyse en Composantes Principales</a></li>
<li class="chapter" data-level="5.2" data-path="perspectives-et-conclusions.html"><a href="perspectives-et-conclusions.html#utilisation-du-desequilibre-de-liaison-pour-ameliorer-linference-de-la-structure"><i class="fa fa-check"></i><b>5.2</b> Utilisation du déséquilibre de liaison pour améliorer l’inférence de la structure</a></li>
<li class="chapter" data-level="5.3" data-path="perspectives-et-conclusions.html"><a href="perspectives-et-conclusions.html#utilisation-de-variables-environnementales"><i class="fa fa-check"></i><b>5.3</b> Utilisation de variables environnementales</a></li>
<li class="chapter" data-level="5.4" data-path="perspectives-et-conclusions.html"><a href="perspectives-et-conclusions.html#scans-pour-lintrogression-et-donnees-manquantes"><i class="fa fa-check"></i><b>5.4</b> Scans pour l’introgression et données manquantes</a></li>
<li class="chapter" data-level="5.5" data-path="perspectives-et-conclusions.html"><a href="perspectives-et-conclusions.html#lapproche-iram-pour-les-donnees-manquantes"><i class="fa fa-check"></i><b>5.5</b> L’approche IRAM pour les données manquantes</a></li>
<li class="chapter" data-level="5.6" data-path="perspectives-et-conclusions.html"><a href="perspectives-et-conclusions.html#conclusion"><i class="fa fa-check"></i><b>5.6</b> Conclusion</a></li>
</ul></li>
<li class="appendix"><span><b>Appendice</b></span></li>
<li class="chapter" data-level="A" data-path="details.html"><a href="details.html"><i class="fa fa-check"></i><b>A</b> Détails</a><ul>
<li class="chapter" data-level="" data-path="details.html"><a href="details.html#rapport-entre-la-communalite-et-lindice-de-fixation"><i class="fa fa-check"></i>Rapport entre la communalité et l’indice de fixation</a></li>
<li><a href="details.html#une-generalisation-de-la-statistique-t_f-lk">Une généralisation de la statistique <span class="math inline">\(T_{F-LK}\)</span></a></li>
</ul></li>
<li class="chapter" data-level="B" data-path="informations-supplementaires.html"><a href="informations-supplementaires.html"><i class="fa fa-check"></i><b>B</b> Informations supplémentaires</a><ul>
<li class="chapter" data-level="" data-path="informations-supplementaires.html"><a href="informations-supplementaires.html#article-1-1"><i class="fa fa-check"></i>Article 1 </a></li>
<li class="chapter" data-level="" data-path="informations-supplementaires.html"><a href="informations-supplementaires.html#article-2-1"><i class="fa fa-check"></i>Article 2</a></li>
<li class="chapter" data-level="" data-path="informations-supplementaires.html"><a href="informations-supplementaires.html#article-3-1"><i class="fa fa-check"></i>Article 3</a></li>
</ul></li>
<li class="chapter" data-level="C" data-path="r-python.html"><a href="r-python.html"><i class="fa fa-check"></i><b>C</b> R &amp; Python</a><ul>
<li class="chapter" data-level="C.1" data-path="r-python.html"><a href="r-python.html#simulations-et-modeles-demographiques"><i class="fa fa-check"></i><b>C.1</b> Simulations et modèles démographiques</a><ul>
<li class="chapter" data-level="C.1.1" data-path="r-python.html"><a href="r-python.html#modele-en-iles-1"><i class="fa fa-check"></i><b>C.1.1</b> Modèle en îles</a></li>
<li class="chapter" data-level="C.1.2" data-path="r-python.html"><a href="r-python.html#modele-de-divergence"><i class="fa fa-check"></i><b>C.1.2</b> Modèle de divergence</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="bibliographie.html"><a href="bibliographie.html"><i class="fa fa-check"></i>Bibliographie</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Application de l’Analyse en Composantes Principales pour étudier l’adaptation biologique en génomique des populations.</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="informations-supplementaires" class="section level1">
<h1><span class="header-section-number">B</span> Informations supplémentaires</h1>
<div id="article-1-1" class="section level2 unnumbered">
<h2>Article 1 </h2>

<div class="figure" style="text-align: center"><span id="fig:ndf-figS1"></span>
<img src="figure/ndf-figS1.png" alt="Principal component analysis for SNP data simulated under the divergence model depicted in Figures 1 and 2 of the main text." width="\textwidth" />
<p class="caption">
Figure B.1: Principal component analysis for SNP data simulated under the divergence model depicted in Figures 1 and 2 of the main text.
</p>
</div>


<div class="figure" style="text-align: center"><span id="fig:ndf-figS2"></span>
<img src="figure/ndf-figS2.png" alt="False discovery rate of the 1% top-ranked SNPs obtained with \(h^2\) and with \(F_{ST}\) under an island model." width="\textwidth" />
<p class="caption">
Figure B.2: False discovery rate of the 1% top-ranked SNPs obtained with <span class="math inline">\(h^2\)</span> and with <span class="math inline">\(F_{ST}\)</span> under an island model.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:ndf-figS3"></span>
<img src="figure/ndf-figS3.png" alt="Decay of eigenvalues of the covariance matrix for divergence models, the 1000 Genome data, and POPRES." width="\textwidth" />
<p class="caption">
Figure B.3: Decay of eigenvalues of the covariance matrix for divergence models, the 1000 Genome data, and POPRES.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:ndf-figS4"></span>
<img src="figure/ndf-figS4.png" alt="Communality statistic as a function of \(F_{ST}\) for SNPs simulated with a divergence model." width="\textwidth" />
<p class="caption">
Figure B.4: Communality statistic as a function of <span class="math inline">\(F_{ST}\)</span> for SNPs simulated with a divergence model.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:ndf-figS5"></span>
<img src="figure/ndf-figS5.png" alt="Histograms of the squared normalized loadings \(V_{jk}^2\), \(k = 1, 2\), obtained for SNPs simulated with divergence models and for SNPs of the 1000 Genomes data." width="\textwidth" />
<p class="caption">
Figure B.5: Histograms of the squared normalized loadings <span class="math inline">\(V_{jk}^2\)</span>, <span class="math inline">\(k = 1, 2\)</span>, obtained for SNPs simulated with divergence models and for SNPs of the 1000 Genomes data.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:ndf-figS6"></span>
<img src="figure/ndf-figS6.png" alt="Repartition of the 5% top-ranked SNPs of each PCA-based statistic under the two divergence models." width="\textwidth" />
<p class="caption">
Figure B.6: Repartition of the 5% top-ranked SNPs of each PCA-based statistic under the two divergence models.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:ndf-figS7"></span>
<img src="figure/ndf-figS7.png" alt="False discovery rate as a function of the selection coefficient under the two divergence models." width="\textwidth" />
<p class="caption">
Figure B.7: False discovery rate as a function of the selection coefficient under the two divergence models.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:ndf-figS8"></span>
<img src="figure/ndf-figS8.png" alt="False discovery rate as a function of the number \(K\) of principal components under the two divergence models." width="\textwidth" />
<p class="caption">
Figure B.8: False discovery rate as a function of the number <span class="math inline">\(K\)</span> of principal components under the two divergence models.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:ndf-figS9"></span>
<img src="figure/ndf-figs9.png" alt="Comparison between \(h^2\) and \(F_{ST}\) in a isolation-with-migration model." width="75%" />
<p class="caption">
Figure B.9: Comparison between <span class="math inline">\(h^2\)</span> and <span class="math inline">\(F_{ST}\)</span> in a isolation-with-migration model.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:ndf-figS10"></span>
<img src="figure/ndf-figS10.png" alt="Principal component analysis of SNPs simulated with an isolation-with-migration model." width="\textwidth" />
<p class="caption">
Figure B.10: Principal component analysis of SNPs simulated with an isolation-with-migration model.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:ndf-figS11"></span>
<img src="figure/ndf-figS11.png" alt="Principal component analysis of the 1000 Genome data." width="\textwidth" />
<p class="caption">
Figure B.11: Principal component analysis of the 1000 Genome data.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:ndf-figS12"></span>
<img src="figure/ndf-figS12.png" alt="Number of contiguous windows containing one or more outlier SNPs." width="\textwidth" />
<p class="caption">
Figure B.12: Number of contiguous windows containing one or more outlier SNPs.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:ndf-figS13"></span>
<img src="figure/ndf-figS13.png" alt="Manhattan plot for the POPRES data of the squared loadings \(\rho_{j2}^2\) with the second principal component." width="\textwidth" />
<p class="caption">
Figure B.13: Manhattan plot for the POPRES data of the squared loadings <span class="math inline">\(\rho_{j2}^2\)</span> with the second principal component.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:ndf-figS14"></span>
<img src="figure/ndf-figS14.png" alt="Q-Q plots of the \(P\)-values, which are based on the communality \(h^2\) statistic, under an island and a divergence model." width="\textwidth" />
<p class="caption">
Figure B.14: Q-Q plots of the <span class="math inline">\(P\)</span>-values, which are based on the communality <span class="math inline">\(h^2\)</span> statistic, under an island and a divergence model.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:ndf-figS15"></span>
<img src="figure/ndf-figS15.png" alt="Control of the false discovery rate for SNPs simulated under an island and a divergence model." width="\textwidth" />
<p class="caption">
Figure B.15: Control of the false discovery rate for SNPs simulated under an island and a divergence model.
</p>
</div>
</div>
<div id="article-2-1" class="section level2 unnumbered">
<h2>Article 2</h2>

<div class="figure" style="text-align: center"><span id="fig:FigureSI1"></span>
<img src="thesis_files/figure-html/FigureSI1-1.png" alt="Schematic description of the island and divergence model. For the island model, adaptation occurs simultaneously in each population. For the divergence model, adaptation takes place in the branch leading to the second population." width="\textwidth" />
<p class="caption">
Figure B.16: Schematic description of the island and divergence model. For the island model, adaptation occurs simultaneously in each population. For the divergence model, adaptation takes place in the branch leading to the second population.
</p>
</div>


<div class="figure" style="text-align: center"><span id="fig:FigureSI2"></span>
<img src="thesis_files/figure-html/FigureSI2-1.png" alt="Proportion of false discoveries and statistical power as a function of the number of principal components in a model of range expansion." width="\textwidth" />
<p class="caption">
Figure B.17: Proportion of false discoveries and statistical power as a function of the number of principal components in a model of range expansion.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:FigureSI3"></span>
<img src="thesis_files/figure-html/FigureSI3-1.png" alt="Statistical power averaged over the expected proportion of false discoveries (ranging between 0% and 50%) for the island model." width="\textwidth" />
<p class="caption">
Figure B.18: Statistical power averaged over the expected proportion of false discoveries (ranging between 0% and 50%) for the island model.
</p>
</div>


<div class="figure" style="text-align: center"><span id="fig:FigureSI4"></span>
<img src="thesis_files/figure-html/FigureSI4-1.png" alt="Statistical power as a function of the proportion of false discoveries for the island model." width="\textwidth" />
<p class="caption">
Figure B.19: Statistical power as a function of the proportion of false discoveries for the island model.
</p>
</div>


<div class="figure" style="text-align: center"><span id="fig:FigureSI5"></span>
<img src="thesis_files/figure-html/FigureSI5-1.png" alt="Statistical power as a function of the proportion of false discoveries for the divergence model." width="\textwidth" />
<p class="caption">
Figure B.20: Statistical power as a function of the proportion of false discoveries for the divergence model.
</p>
</div>


<div class="figure" style="text-align: center"><span id="fig:FigureSI6"></span>
<img src="thesis_files/figure-html/FigureSI6-1.png" alt="Statistical power as a function of the proportion of false discoveries for the model of range expansion." width="\textwidth" />
<p class="caption">
Figure B.21: Statistical power as a function of the proportion of false discoveries for the model of range expansion.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:FigureSI7"></span>
<img src="thesis_files/figure-html/FigureSI7-1.png" alt="Running times of the different computer programs. The different programs were run on genotype matrices containing 300 individuals and from 500 to 50,000 SNPs. The characteristics of the computer we used to perform comparisons are the following: OSX El Capitan 10.11.3, 2,5 GHz Intel Core i5, 8 Go 1600 MHz DDR3." width="\textwidth" />
<p class="caption">
Figure B.22: Running times of the different computer programs. The different programs were run on genotype matrices containing 300 individuals and from 500 to 50,000 SNPs. The characteristics of the computer we used to perform comparisons are the following: OSX El Capitan 10.11.3, 2,5 GHz Intel Core i5, 8 Go 1600 MHz DDR3.
</p>
</div>
</div>
<div id="article-3-1" class="section level2 unnumbered">
<h2>Article 3</h2>

<div class="figure" style="text-align: center"><span id="fig:ancestry"></span>
<img src="figure/ancestry.png" alt="Standardized average ancestry coefficient computed with pcadapt and different LAI methods (EILA, Loter and RFMix) for a population of simulated admixed individuals. Simulations use phased genotype data from \(25\) Populus balsamifera individuals and \(25\) Populus trichocarpa individuals and assume that admixture took place \(\lambda=100\) generations ago. A total of \(25\) admixed individuals were generated assuming \(30\%\) of Populus balsamifera ancestry except from the 5 outlier \(500\) SNP regions where Populus balsamifera ancestry is of \(50\%\)." width="\textwidth" />
<p class="caption">
Figure B.23: Standardized average ancestry coefficient computed with pcadapt and different LAI methods (EILA, Loter and RFMix) for a population of simulated admixed individuals. Simulations use phased genotype data from <span class="math inline">\(25\)</span> <em>Populus balsamifera</em> individuals and <span class="math inline">\(25\)</span> <em>Populus trichocarpa</em> individuals and assume that admixture took place <span class="math inline">\(\lambda=100\)</span> generations ago. A total of <span class="math inline">\(25\)</span> admixed individuals were generated assuming <span class="math inline">\(30\%\)</span> of <em>Populus balsamifera</em> ancestry except from the 5 outlier <span class="math inline">\(500\)</span> SNP regions where <em>Populus balsamifera</em> ancestry is of <span class="math inline">\(50\%\)</span>.
</p>
</div>


<div class="figure" style="text-align: center"><span id="fig:losspower-admix"></span>
<img src="figure/meanloss_admixture.png" alt="Relative loss of power of the different methods when compared to an ideal method called oracle, which would know ancestry chunks for each admixed individual. The relative power is averaged over the difference \(\Delta_q\) of ancestry between neutral and outlier regions. Simulations use phased genotype data from \(25\) Populus balsamifera individuals and \(25\) Populus trichocarpa individuals and assume that admixture took place \(\lambda\) generations ago. A total of \(25\) admixed individuals were generated assuming \(70\%\) of Populus balsamifera ancestry except from the 5 outlier \(500\) SNP regions where Populus balsamifera ancestry is of \(70\%-\Delta_q\) where \(\Delta_q\) is equal to \(0.05\), \(0.10\), \(0.15\), \(0.20\), \(0.30\), \(0.40\), or \(0.50\)." width="\textwidth" />
<p class="caption">
Figure B.24: Relative loss of power of the different methods when compared to an ideal method called <em>oracle</em>, which would know ancestry chunks for each admixed individual. The relative power is averaged over the difference <span class="math inline">\(\Delta_q\)</span> of ancestry between neutral and outlier regions. Simulations use phased genotype data from <span class="math inline">\(25\)</span> <em>Populus balsamifera</em> individuals and <span class="math inline">\(25\)</span> <em>Populus trichocarpa</em> individuals and assume that admixture took place <span class="math inline">\(\lambda\)</span> generations ago. A total of <span class="math inline">\(25\)</span> admixed individuals were generated assuming <span class="math inline">\(70\%\)</span> of <em>Populus balsamifera</em> ancestry except from the 5 outlier <span class="math inline">\(500\)</span> SNP regions where <em>Populus balsamifera</em> ancestry is of <span class="math inline">\(70\%-\Delta_q\)</span> where <span class="math inline">\(\Delta_q\)</span> is equal to <span class="math inline">\(0.05\)</span>, <span class="math inline">\(0.10\)</span>, <span class="math inline">\(0.15\)</span>, <span class="math inline">\(0.20\)</span>, <span class="math inline">\(0.30\)</span>, <span class="math inline">\(0.40\)</span>, or <span class="math inline">\(0.50\)</span>.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:power-admix-max"></span>
<img src="figure/power_Dq_max.png" alt="Proportion of true outlier peaks among the five top peaks found with pcadapt and different LAI methods (EILA, Loter and RFMix) in a scenario where 2 Populus populations experienced admixture. Compared to Figure 3.10, we compute maximum of ancestry coefficients within each genomic region instead of mean of ancestry coefficients. Proportion of true outlier peaks are displayed as a function of the difference \(\Delta_q\) of ancestry between outlier and neutral regions. The three panels correspond to the three different possible values (\(\lambda=10\) or \(100\) or \(1000\)) of the number of generations since admixture." width="\textwidth" />
<p class="caption">
Figure B.25: Proportion of true outlier peaks among the five top peaks found with pcadapt and different LAI methods (EILA, Loter and RFMix) in a scenario where 2 Populus populations experienced admixture. Compared to Figure <a href="introgression-adaptative.html#fig:power-admix">3.10</a>, we compute maximum of ancestry coefficients within each genomic region instead of mean of ancestry coefficients. Proportion of true outlier peaks are displayed as a function of the difference <span class="math inline">\(\Delta_q\)</span> of ancestry between outlier and neutral regions. The three panels correspond to the three different possible values (<span class="math inline">\(\lambda=10\)</span> or <span class="math inline">\(100\)</span> or <span class="math inline">\(1000\)</span>) of the number of generations since admixture.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:si-power-admix"></span>
<img src="figure/power_Dq_curves.png" alt="Proportion of true outlier peaks as a function of the number of top peaks found with pcadapt and different LAI methods (EILA, Loter and RFMix) in a scenario where 2 Populus populations experienced admixture. The different panels correspond to the different possible values for the number of generations since admixture occurred (\(\lambda=\{10,100,1000\}\)) and to the different values of the difference of ancestry \(\Delta_q\) between neutral and outlier regions." width="\textwidth" />
<p class="caption">
Figure B.26: Proportion of true outlier peaks as a function of the number of top peaks found with pcadapt and different LAI methods (EILA, Loter and RFMix) in a scenario where 2 Populus populations experienced admixture. The different panels correspond to the different possible values for the number of generations since admixture occurred (<span class="math inline">\(\lambda=\{10,100,1000\}\)</span>) and to the different values of the difference of ancestry <span class="math inline">\(\Delta_q\)</span> between neutral and outlier regions.
</p>
</div>


<div class="figure" style="text-align: center"><span id="fig:power-introgression"></span>
<img src="figure/power_f_introgression.png" alt="Power as a function of the proportion of introgression." width="\textwidth" />
<p class="caption">
Figure B.27: Power as a function of the proportion of introgression.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:introgression"></span>
<img src="figure/introgression.png" alt="Schematic description of simulations under an introgression scenario." width="\textwidth" />
<p class="caption">
Figure B.28: Schematic description of simulations under an introgression scenario.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:admixture"></span>
<img src="figure/admixture.png" alt="Schematic description of simulations under an admixture scenario." width="\textwidth" />
<p class="caption">
Figure B.29: Schematic description of simulations under an admixture scenario.
</p>
</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="details.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="r-python.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"download": [["thesis.pdf", "PDF"], ["thesis.epub", "EPUB"], ["thesis.docx", "Word"]],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(script.src))
      script.src  = script.src.replace(/^https?:/, '');
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
