<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Méthodes statistiques en grande dimension pour l’étude de l’adaptation biologique à l’aide de larges bases de données génomiques</title>
  <meta name="description" content="Méthodes statistiques en grande dimension pour l’étude de l’adaptation biologique à l’aide de larges bases de données génomiques">
  <meta name="generator" content="bookdown 0.5 and GitBook 2.6.7">

  <meta property="og:title" content="Méthodes statistiques en grande dimension pour l’étude de l’adaptation biologique à l’aide de larges bases de données génomiques" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Méthodes statistiques en grande dimension pour l’étude de l’adaptation biologique à l’aide de larges bases de données génomiques" />
  
  
  

<meta name="author" content="Keurcien LUU">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="informations-supplementaires.html">
<link rel="next" href="bibliographie.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />









<style type="text/css">
div.sourceCode { overflow-x: auto; }
table.sourceCode, tr.sourceCode, td.lineNumbers, td.sourceCode {
  margin: 0; padding: 0; vertical-align: baseline; border: none; }
table.sourceCode { width: 100%; line-height: 100%; }
td.lineNumbers { text-align: right; padding-right: 4px; padding-left: 4px; color: #aaaaaa; border-right: 1px solid #aaaaaa; }
td.sourceCode { padding-left: 5px; }
code > span.kw { color: #007020; font-weight: bold; } /* Keyword */
code > span.dt { color: #902000; } /* DataType */
code > span.dv { color: #40a070; } /* DecVal */
code > span.bn { color: #40a070; } /* BaseN */
code > span.fl { color: #40a070; } /* Float */
code > span.ch { color: #4070a0; } /* Char */
code > span.st { color: #4070a0; } /* String */
code > span.co { color: #60a0b0; font-style: italic; } /* Comment */
code > span.ot { color: #007020; } /* Other */
code > span.al { color: #ff0000; font-weight: bold; } /* Alert */
code > span.fu { color: #06287e; } /* Function */
code > span.er { color: #ff0000; font-weight: bold; } /* Error */
code > span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
code > span.cn { color: #880000; } /* Constant */
code > span.sc { color: #4070a0; } /* SpecialChar */
code > span.vs { color: #4070a0; } /* VerbatimString */
code > span.ss { color: #bb6688; } /* SpecialString */
code > span.im { } /* Import */
code > span.va { color: #19177c; } /* Variable */
code > span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code > span.op { color: #666666; } /* Operator */
code > span.bu { } /* BuiltIn */
code > span.ex { } /* Extension */
code > span.pp { color: #bc7a00; } /* Preprocessor */
code > span.at { color: #7d9029; } /* Attribute */
code > span.do { color: #ba2121; font-style: italic; } /* Documentation */
code > span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code > span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code > span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#la-genetique-des-populations"><i class="fa fa-check"></i><b>1.1</b> La génétique des populations</a><ul>
<li class="chapter" data-level="1.1.1" data-path="introduction.html"><a href="introduction.html#levolution-comme-point-de-depart"><i class="fa fa-check"></i><b>1.1.1</b> L’évolution comme point de départ</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#a-lorigine-de-la-variabilite-genetique"><i class="fa fa-check"></i><b>1.2</b> À l’origine de la variabilité génétique</a><ul>
<li class="chapter" data-level="1.2.1" data-path="introduction.html"><a href="introduction.html#la-theorie-de-levolution"><i class="fa fa-check"></i><b>1.2.1</b> La théorie de l’évolution</a></li>
<li class="chapter" data-level="1.2.2" data-path="introduction.html"><a href="introduction.html#levolution-dune-theorie"><i class="fa fa-check"></i><b>1.2.2</b> L’évolution d’une théorie</a></li>
<li class="chapter" data-level="1.2.3" data-path="introduction.html"><a href="introduction.html#forces-evolutives"><i class="fa fa-check"></i><b>1.2.3</b> Forces évolutives</a></li>
<li class="chapter" data-level="1.2.4" data-path="introduction.html"><a href="introduction.html#adaptation-locale"><i class="fa fa-check"></i><b>1.2.4</b> Adaptation locale</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#donnees-de-polymorphismes-genetiques"><i class="fa fa-check"></i><b>1.3</b> Données de polymorphismes génétiques</a><ul>
<li class="chapter" data-level="1.3.1" data-path="introduction.html"><a href="introduction.html#des-donnees-en-grande-dimension"><i class="fa fa-check"></i><b>1.3.1</b> Des données en grande dimension</a></li>
<li class="chapter" data-level="1.3.2" data-path="introduction.html"><a href="introduction.html#les-marqueurs-genetiques"><i class="fa fa-check"></i><b>1.3.2</b> Les marqueurs génétiques</a></li>
<li class="chapter" data-level="1.3.3" data-path="introduction.html"><a href="introduction.html#encodage-des-donnees-genetiques"><i class="fa fa-check"></i><b>1.3.3</b> Encodage des données génétiques</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="introduction.html"><a href="introduction.html#les-motivations-de-la-these"><i class="fa fa-check"></i><b>1.4</b> Les motivations de la thèse</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html"><i class="fa fa-check"></i><b>2</b> Adaptation locale</a><ul>
<li class="chapter" data-level="2.1" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#letat-de-lart-pour-les-scans-genomiques"><i class="fa fa-check"></i><b>2.1</b> L’état de l’art pour les scans génomiques</a><ul>
<li class="chapter" data-level="2.1.1" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#modeles-demographiques"><i class="fa fa-check"></i><b>2.1.1</b> Modèles démographiques</a></li>
<li class="chapter" data-level="2.1.2" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#lindice-de-fixation"><i class="fa fa-check"></i><b>2.1.2</b> L’indice de fixation</a></li>
<li class="chapter" data-level="2.1.3" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#test-de-lewontin-krakauer"><i class="fa fa-check"></i><b>2.1.3</b> Test de Lewontin-Krakauer</a></li>
<li class="chapter" data-level="2.1.4" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#le-modele-f"><i class="fa fa-check"></i><b>2.1.4</b> Le modèle <span class="math inline">\(F\)</span></a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#lanalyse-en-composantes-principales-en-genetique-des-populations"><i class="fa fa-check"></i><b>2.2</b> L’Analyse en Composantes Principales en génétique des populations</a><ul>
<li class="chapter" data-level="2.2.1" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#principe-de-lanalyse-en-composantes-principales"><i class="fa fa-check"></i><b>2.2.1</b> Principe de l’Analyse en Composantes Principales</a></li>
<li class="chapter" data-level="2.2.2" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#apparentement-genetique-interindividuel"><i class="fa fa-check"></i><b>2.2.2</b> Apparentement génétique interindividuel</a></li>
<li class="chapter" data-level="2.2.3" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#applications-en-genetique-des-populations"><i class="fa fa-check"></i><b>2.2.3</b> Applications en génétique des populations</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#statistiques-de-test-basees-sur-lanalyse-en-composantes-principales"><i class="fa fa-check"></i><b>2.3</b> Statistiques de test basées sur l’Analyse en Composantes Principales</a><ul>
<li class="chapter" data-level="2.3.1" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#la-communalite"><i class="fa fa-check"></i><b>2.3.1</b> La communalité</a></li>
<li class="chapter" data-level="2.3.2" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#la-distance-robuste-de-mahalanobis"><i class="fa fa-check"></i><b>2.3.2</b> La distance robuste de Mahalanobis</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#article-1"><i class="fa fa-check"></i><b>2.4</b> Article 1</a><ul>
<li class="chapter" data-level="" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#contributions"><i class="fa fa-check"></i>Contributions</a></li>
<li class="chapter" data-level="" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#resume"><i class="fa fa-check"></i>Résumé</a></li>
<li class="chapter" data-level="" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#abstract"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#significance-statement"><i class="fa fa-check"></i>Significance Statement</a></li>
<li class="chapter" data-level="" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#introduction-1"><i class="fa fa-check"></i>Introduction</a></li>
<li class="chapter" data-level="" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#new-method"><i class="fa fa-check"></i>New Method</a></li>
<li class="chapter" data-level="" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#results"><i class="fa fa-check"></i>Results</a></li>
<li class="chapter" data-level="" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#discussion"><i class="fa fa-check"></i>Discussion</a></li>
<li class="chapter" data-level="" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#materials-and-methods"><i class="fa fa-check"></i>Materials and Methods</a></li>
<li class="chapter" data-level="" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#acknowledgments"><i class="fa fa-check"></i>Acknowledgments</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#article-2"><i class="fa fa-check"></i><b>2.5</b> Article 2</a><ul>
<li class="chapter" data-level="" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#abstract-1"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#introduction-2"><i class="fa fa-check"></i>Introduction</a></li>
<li class="chapter" data-level="" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#statistical-and-computational-approach"><i class="fa fa-check"></i>Statistical and computational approach</a></li>
<li class="chapter" data-level="" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#materials-and-methods-1"><i class="fa fa-check"></i>Materials and methods</a></li>
<li class="chapter" data-level="" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#results-1"><i class="fa fa-check"></i>Results</a></li>
<li class="chapter" data-level="" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#discussion-1"><i class="fa fa-check"></i>Discussion</a></li>
<li class="chapter" data-level="" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#acknowledgements"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#data-accessibility"><i class="fa fa-check"></i>Data accessibility</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html"><i class="fa fa-check"></i><b>3</b> Introgression adaptative</a><ul>
<li class="chapter" data-level="3.1" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#methode-basee-sur-lutilisation-de-coefficients-de-metissage-locaux."><i class="fa fa-check"></i><b>3.1</b> Méthode basée sur l’utilisation de coefficients de métissage locaux.</a><ul>
<li class="chapter" data-level="3.1.1" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#coefficients-de-metissage-locaux"><i class="fa fa-check"></i><b>3.1.1</b> Coefficients de métissage locaux</a></li>
<li class="chapter" data-level="3.1.2" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#statistique-de-test"><i class="fa fa-check"></i><b>3.1.2</b> Statistique de test</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#methodes-basees-sur-la-diversite-nucleotidique."><i class="fa fa-check"></i><b>3.2</b> Méthodes basées sur la diversité nucléotidique.</a><ul>
<li class="chapter" data-level="" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#diversite-nucleotidique-par-paire-de-sequences"><i class="fa fa-check"></i>Diversité nucléotidique par paire de séquences</a></li>
<li class="chapter" data-level="3.2.1" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#le-modele-abba-baba"><i class="fa fa-check"></i><b>3.2.1</b> Le modèle ABBA-BABA</a></li>
<li class="chapter" data-level="3.2.2" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#analyse-en-composantes-principales-locale"><i class="fa fa-check"></i><b>3.2.2</b> Analyse en Composantes Principales locale</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#simulations"><i class="fa fa-check"></i><b>3.3</b> Simulations</a><ul>
<li class="chapter" data-level="3.3.1" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#donnees-de-peupliers"><i class="fa fa-check"></i><b>3.3.1</b> Données de peupliers</a></li>
<li class="chapter" data-level="3.3.2" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#generation-aleatoire-dindividus-hybrides"><i class="fa fa-check"></i><b>3.3.2</b> Génération aléatoire d’individus hybrides</a></li>
<li class="chapter" data-level="3.3.3" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#simulations-a-partir-de-ms-et-seq-gen"><i class="fa fa-check"></i><b>3.3.3</b> Simulations à partir de ms et Seq-Gen</a></li>
<li class="chapter" data-level="3.3.4" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#resultats-de-la-comparaison-des-logiciels"><i class="fa fa-check"></i><b>3.3.4</b> Résultats de la comparaison des logiciels</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#article-3"><i class="fa fa-check"></i><b>3.4</b> Article 3</a><ul>
<li class="chapter" data-level="" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#introduction-3"><i class="fa fa-check"></i>Introduction</a></li>
<li class="chapter" data-level="" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#materials-and-methods-2"><i class="fa fa-check"></i>Materials and methods</a></li>
<li class="chapter" data-level="" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#results-2"><i class="fa fa-check"></i>Results</a></li>
<li class="chapter" data-level="" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#discussion-2"><i class="fa fa-check"></i>Discussion</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="aspect-computationnel.html"><a href="aspect-computationnel.html"><i class="fa fa-check"></i><b>4</b> Aspect computationnel</a><ul>
<li class="chapter" data-level="4.1" data-path="aspect-computationnel.html"><a href="aspect-computationnel.html#donnees-dentree"><i class="fa fa-check"></i><b>4.1</b> Données d’entrée</a></li>
<li class="chapter" data-level="4.2" data-path="aspect-computationnel.html"><a href="aspect-computationnel.html#memory-mapping"><i class="fa fa-check"></i><b>4.2</b> Memory-mapping</a></li>
<li class="chapter" data-level="4.3" data-path="aspect-computationnel.html"><a href="aspect-computationnel.html#interface-shiny"><i class="fa fa-check"></i><b>4.3</b> Interface Shiny</a></li>
<li class="chapter" data-level="4.4" data-path="aspect-computationnel.html"><a href="aspect-computationnel.html#acp-et-donnees-manquantes"><i class="fa fa-check"></i><b>4.4</b> ACP et données manquantes</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="conclusion.html"><a href="conclusion.html"><i class="fa fa-check"></i>Conclusion</a></li>
<li class="appendix"><span><b>Appendice</b></span></li>
<li class="chapter" data-level="A" data-path="details.html"><a href="details.html"><i class="fa fa-check"></i><b>A</b> Détails</a><ul>
<li class="chapter" data-level="A.0.1" data-path="details.html"><a href="details.html#rapport-entre-la-communalite-et-lindice-de-fixation"><i class="fa fa-check"></i><b>A.0.1</b> Rapport entre la communalité et l’indice de fixation</a></li>
<li class="chapter" data-level="A.0.2" data-path="details.html"><a href="details.html#lanalyse-en-composantes-principales-regularisee"><i class="fa fa-check"></i><b>A.0.2</b> L’Analyse en Composantes Principales régularisée</a></li>
</ul></li>
<li class="chapter" data-level="B" data-path="informations-supplementaires.html"><a href="informations-supplementaires.html"><i class="fa fa-check"></i><b>B</b> Informations supplémentaires</a><ul>
<li class="chapter" data-level="" data-path="informations-supplementaires.html"><a href="informations-supplementaires.html#article-1-1"><i class="fa fa-check"></i>Article 1 </a></li>
<li class="chapter" data-level="" data-path="informations-supplementaires.html"><a href="informations-supplementaires.html#article-2-1"><i class="fa fa-check"></i>Article 2</a></li>
<li class="chapter" data-level="" data-path="informations-supplementaires.html"><a href="informations-supplementaires.html#article-3-1"><i class="fa fa-check"></i>Article 3</a></li>
</ul></li>
<li class="chapter" data-level="C" data-path="r-python.html"><a href="r-python.html"><i class="fa fa-check"></i><b>C</b> R &amp; Python</a><ul>
<li class="chapter" data-level="C.1" data-path="r-python.html"><a href="r-python.html#simulations-et-modeles-demographiques"><i class="fa fa-check"></i><b>C.1</b> Simulations et modèles démographiques</a><ul>
<li class="chapter" data-level="C.1.1" data-path="r-python.html"><a href="r-python.html#modele-en-ile"><i class="fa fa-check"></i><b>C.1.1</b> Modèle en île</a></li>
<li class="chapter" data-level="C.1.2" data-path="r-python.html"><a href="r-python.html#modele-de-divergence"><i class="fa fa-check"></i><b>C.1.2</b> Modèle de divergence</a></li>
</ul></li>
<li class="chapter" data-level="C.2" data-path="r-python.html"><a href="r-python.html#details-1"><i class="fa fa-check"></i><b>C.2</b> Détails</a><ul>
<li class="chapter" data-level="C.2.1" data-path="r-python.html"><a href="r-python.html#donnees-manquantes"><i class="fa fa-check"></i><b>C.2.1</b> Données manquantes</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="bibliographie.html"><a href="bibliographie.html"><i class="fa fa-check"></i>Bibliographie</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Méthodes statistiques en grande dimension pour l’étude de l’adaptation biologique à l’aide de larges bases de données génomiques</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="r-python" class="section level1">
<h1><span class="header-section-number">C</span> R &amp; Python</h1>
<div id="simulations-et-modeles-demographiques" class="section level2">
<h2><span class="header-section-number">C.1</span> Simulations et modèles démographiques</h2>
<p>Nous mettons ici à disposition une partie des codes qui ont servi à produire les simulations et à leur exploitation.</p>
<div id="modele-en-ile" class="section level3">
<h3><span class="header-section-number">C.1.1</span> Modèle en île</h3>

<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">if (!<span class="kw">file.exists</span>(<span class="st">&quot;ms/ms&quot;</span>)) {
  <span class="kw">system</span>(<span class="st">&quot;gcc -o ms/ms ms/ms.c ms/streec.c ms/rand1.c -lm&quot;</span>)
}

### ms : list of parameters ###
nb_demes &lt;-<span class="st"> </span><span class="dv">3</span>
nCHR_per_POP &lt;-<span class="st"> </span><span class="dv">50</span>
nCHR &lt;-<span class="st"> </span>nb_demes *<span class="st"> </span>nCHR_per_POP
nIND &lt;-<span class="st"> </span>nCHR /<span class="st"> </span><span class="dv">2</span>
nb_neutral &lt;-<span class="st"> </span><span class="dv">100</span>
nb_adaptive &lt;-<span class="st"> </span><span class="dv">25</span>
mig_rate_neutral &lt;-<span class="st"> </span><span class="dv">10</span>
mig_rate_adaptive &lt;-<span class="st"> </span><span class="fl">0.1</span>
###############################

nCHR_per_POP_string &lt;-<span class="st"> </span>nCHR_per_POP
for (k in <span class="dv">1</span>:(nb_demes -<span class="st"> </span><span class="dv">1</span>)) {
  nCHR_per_POP_string &lt;-<span class="st"> </span><span class="kw">paste</span>(nCHR_per_POP_string, nCHR_per_POP)
}

mig_rate_neutral_string &lt;-<span class="st"> &quot;x&quot;</span>
mig_rate_adaptive_string &lt;-<span class="st"> &quot;x&quot;</span>
for (k in <span class="dv">1</span>:nb_demes){
  for (j in <span class="dv">1</span>:nb_demes){
    if ((k ==<span class="st"> </span>j) &amp;&amp;<span class="st"> </span>(k &gt;<span class="st"> </span><span class="dv">1</span>)){
      mig_rate_neutral_string &lt;-<span class="st"> </span><span class="kw">paste</span>(mig_rate_neutral_string, <span class="st">&quot;x&quot;</span>)
      mig_rate_adaptive_string &lt;-<span class="st"> </span><span class="kw">paste</span>(mig_rate_adaptive_string, <span class="st">&quot;x&quot;</span>)
    } else if (k !=<span class="st"> </span>j) {
      mig_rate_neutral_string &lt;-<span class="st"> </span><span class="kw">paste</span>(mig_rate_neutral_string,
                                       mig_rate_neutral)
      mig_rate_adaptive_string &lt;-<span class="st"> </span><span class="kw">paste</span>(mig_rate_adaptive_string,
                                        mig_rate_adaptive)
    }
  }
}

cmd_neutral &lt;-<span class="st"> </span><span class="kw">paste</span>(<span class="st">&quot;ms/./ms&quot;</span>,
                     nCHR,
                     nb_neutral,
                     <span class="st">&quot;-s 1 -I&quot;</span>,
                     nb_demes,
                     nCHR_per_POP_string,
                     <span class="st">&quot;-ma&quot;</span>,
                     mig_rate_neutral_string,
                     <span class="st">&quot;&gt;&quot;</span>,
                     <span class="st">&quot;data/neutral.txt&quot;</span>)

cmd_adaptive &lt;-<span class="st"> </span><span class="kw">paste</span>(<span class="st">&quot;ms/./ms&quot;</span>,
                      nCHR,
                      nb_adaptive,
                      <span class="st">&quot;-s 1 -I&quot;</span>,
                      nb_demes,
                      nCHR_per_POP_string,
                      <span class="st">&quot;-ma&quot;</span>,
                      mig_rate_adaptive_string,
                      <span class="st">&quot;&gt;&quot;</span>,
                      <span class="st">&quot;data/adaptive.txt&quot;</span>)

<span class="kw">system</span>(cmd_neutral)
<span class="kw">system</span>(cmd_adaptive)

file.neutral &lt;-<span class="st"> </span><span class="kw">scan</span>(<span class="dt">file =</span> <span class="st">&quot;data/neutral.txt&quot;</span>,
              <span class="dt">what =</span> <span class="st">&quot;character&quot;</span>,
              <span class="dt">sep =</span> <span class="st">&quot;</span><span class="ch">\n</span><span class="st">&quot;</span>,
              <span class="dt">skip =</span> <span class="dv">2</span>)

g.neutral &lt;-<span class="st"> </span><span class="ot">NULL</span>

for (locus in <span class="dv">1</span>:nb_neutral){
  res.locus1 &lt;-<span class="st"> </span>file.neutral[<span class="dv">4</span>:(nCHR +<span class="st"> </span><span class="dv">3</span>)]
  file.neutral &lt;-<span class="st"> </span>file.neutral[-(<span class="dv">1</span>:(nCHR<span class="dv">+3</span>))]
  g.neutral &lt;-<span class="st"> </span><span class="kw">cbind</span>(g.neutral, <span class="kw">as.numeric</span>(<span class="kw">as.factor</span>(res.locus1)))
}

file.adaptive &lt;-<span class="st"> </span><span class="kw">scan</span>(<span class="dt">file =</span> <span class="st">&quot;data/adaptive.txt&quot;</span>,
                      <span class="dt">what =</span> <span class="st">&quot;character&quot;</span>,
                      <span class="dt">sep =</span> <span class="st">&quot;</span><span class="ch">\n</span><span class="st">&quot;</span>,
                      <span class="dt">skip =</span> <span class="dv">2</span>)

g.adaptive &lt;-<span class="st"> </span><span class="ot">NULL</span>

for (locus in <span class="dv">1</span>:nb_adaptive){
  res.locus1 &lt;-<span class="st"> </span>file.adaptive[<span class="dv">4</span>:(nCHR +<span class="st"> </span><span class="dv">3</span>)]
  file.adaptive &lt;-<span class="st"> </span>file.adaptive[-(<span class="dv">1</span>:(nCHR<span class="dv">+3</span>))]
  g.adaptive &lt;-<span class="st"> </span><span class="kw">cbind</span>(g.adaptive, <span class="kw">as.numeric</span>(<span class="kw">as.factor</span>(res.locus1)))
}

g &lt;-<span class="st"> </span><span class="kw">cbind</span>(g.neutral, g.adaptive)

x &lt;-<span class="st"> </span>pcadapt::<span class="kw">pcadapt</span>(<span class="kw">t</span>(g), <span class="dt">K =</span> <span class="dv">2</span>)
pop &lt;-<span class="st"> </span><span class="kw">c</span>(<span class="kw">rep</span>(<span class="st">&quot;A&quot;</span>, <span class="dv">50</span>), <span class="kw">rep</span>(<span class="st">&quot;B&quot;</span>, <span class="dv">50</span>), <span class="kw">rep</span>(<span class="st">&quot;C&quot;</span>, <span class="dv">50</span>))
<span class="kw">plot</span>(x, <span class="dt">option =</span> <span class="st">&quot;scores&quot;</span>, <span class="dt">pop =</span> pop)</code></pre></div>
<div class="figure" style="text-align: center"><span id="fig:island-pca"></span>
<img src="thesis_files/figure-html/island-pca-1.png" alt="Exemple de simulation réalisée à l’aide de ms (Hudson, 2002)." width="\textwidth" />
<p class="caption">
Figure C.1: Exemple de simulation réalisée à l’aide de <em>ms</em> <span class="citation">(Hudson, <a href="#ref-hudson2002generating">2002</a>)</span>.
</p>
</div>

<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="kw">plot</span>(x, <span class="dt">option =</span> <span class="st">&quot;manhattan&quot;</span>)</code></pre></div>
<div class="figure" style="text-align: center"><span id="fig:island-manhattan"></span>
<img src="thesis_files/figure-html/island-manhattan-1.png" alt="Manhattan plot obtenu à partir de l’analyse de la simulation C.1." width="\textwidth" />
<p class="caption">
Figure C.2: Manhattan plot obtenu à partir de l’analyse de la simulation <a href="r-python.html#fig:island-pca">C.1</a>.
</p>
</div>
</div>
<div id="modele-de-divergence" class="section level3">
<h3><span class="header-section-number">C.1.2</span> Modèle de divergence</h3>
<p>Nous adaptons une version du script Python utilisé dans <span class="citation">(Roux et al., <a href="#ref-roux2012recent">2012</a>)</span>, basé sur le module de simulation simuPOP <span class="citation">(B. Peng &amp; Kimmel, <a href="#ref-peng2005simupop">2005</a>)</span>.</p>
<div class="sourceCode"><pre class="sourceCode python"><code class="sourceCode python"><span class="co">#!/usr/bin/env python</span>
<span class="im">from</span> __future__ <span class="im">import</span> division
<span class="im">import</span> simuOpt, types, os, sys, time
simuOpt.setOptions(alleleType <span class="op">=</span> <span class="st">&#39;long&#39;</span>)
<span class="im">from</span> operator <span class="im">import</span> itemgetter
<span class="im">import</span> numpy <span class="im">as</span> np
<span class="im">from</span> simuPOP <span class="im">import</span> <span class="op">*</span>
<span class="im">from</span> simuPOP.utils <span class="im">import</span> <span class="op">*</span>
<span class="im">from</span> simuPOP.sampling <span class="im">import</span> drawRandomSample

<span class="kw">def</span> simulate(Ne, Nsam, T1, T2, T3, s10, s11):
    pop <span class="op">=</span> Population(size <span class="op">=</span> Ne,
                     ploidy <span class="op">=</span> <span class="dv">2</span>,
                     loci <span class="op">=</span> [<span class="dv">1</span>],
                     infoFields <span class="op">=</span> [<span class="st">&#39;fitness&#39;</span>, <span class="st">&#39;migrate_to&#39;</span>])

    <span class="kw">def</span> getfitness10(geno):
        <span class="cf">if</span> geno[<span class="dv">0</span>] <span class="op">+</span> geno[<span class="dv">1</span>] <span class="op">==</span> <span class="dv">0</span> :
            <span class="cf">return</span> <span class="dv">1</span> <span class="op">-</span> <span class="dv">2</span> <span class="op">*</span> s10
        <span class="cf">if</span> geno[<span class="dv">0</span>] <span class="op">+</span> geno[<span class="dv">1</span>] <span class="op">==</span> <span class="dv">1</span> :
            <span class="cf">return</span> <span class="dv">1</span> <span class="op">-</span> s10
        <span class="cf">else</span> :
            <span class="cf">return</span> <span class="dv">1</span>

    <span class="kw">def</span> getfitness11(geno):
        <span class="cf">if</span> geno[<span class="dv">0</span>] <span class="op">+</span> geno[<span class="dv">1</span>] <span class="op">==</span> <span class="dv">0</span> :
            <span class="cf">return</span> <span class="dv">1</span> <span class="op">-</span> <span class="dv">2</span> <span class="op">*</span> s11
        <span class="cf">if</span> geno[<span class="dv">0</span>] <span class="op">+</span> geno[<span class="dv">1</span>] <span class="op">==</span> <span class="dv">1</span> :
            <span class="cf">return</span> <span class="dv">1</span> <span class="op">-</span> s11
        <span class="cf">else</span> :
            <span class="cf">return</span> <span class="dv">1</span>

    pop.evolve(
        initOps <span class="op">=</span> [
            InitSex(),
            InitGenotype(loci <span class="op">=</span> ALL_AVAIL,
                         freq <span class="op">=</span> [<span class="fl">0.5</span>, <span class="fl">0.5</span>],
                         begin <span class="op">=</span> <span class="dv">0</span>,
                         end <span class="op">=</span> <span class="dv">1</span>)
        ],

        preOps <span class="op">=</span> [
            <span class="co"># resize the ancestral population at the time immediatly</span>
            <span class="co"># before the split</span>
            ResizeSubPops([<span class="dv">0</span>],
                          sizes <span class="op">=</span> [Ne <span class="op">+</span> Ne],
                          at <span class="op">=</span> T1 <span class="op">-</span> <span class="dv">1</span>),

            ResizeSubPops([<span class="st">&quot;S1_1&quot;</span>],
                          sizes <span class="op">=</span> [Ne <span class="op">+</span> Ne],
                          at <span class="op">=</span> T1 <span class="op">+</span> T2 <span class="op">-</span> <span class="dv">1</span>),

            <span class="co"># split populations in 2 subpopulations</span>
            SplitSubPops(subPops <span class="op">=</span> [<span class="dv">0</span>],
                         sizes <span class="op">=</span> [Ne, Ne],
                         names <span class="op">=</span> [<span class="st">&quot;S1_0&quot;</span>, <span class="st">&quot;S1_1&quot;</span>],
                         at <span class="op">=</span> T1),

            SplitSubPops(subPops <span class="op">=</span> [<span class="st">&quot;S1_1&quot;</span>],
                         sizes <span class="op">=</span> [Ne, Ne],
                         at <span class="op">=</span> T1 <span class="op">+</span> T2,
                         names <span class="op">=</span> [<span class="st">&quot;S2_0&quot;</span>, <span class="st">&quot;S2_1&quot;</span>]),

            <span class="co"># apply selection by invoking function getfitness</span>
            PySelector(loci <span class="op">=</span> [<span class="dv">0</span>],
                       func <span class="op">=</span> getfitness11,
                       begin <span class="op">=</span> T1 <span class="op">+</span> T2,
                       subPops <span class="op">=</span> [<span class="st">&quot;S2_1&quot;</span>]),

            PySelector(loci <span class="op">=</span> [<span class="dv">0</span>],
                       func <span class="op">=</span> getfitness10,
                       begin <span class="op">=</span> T1,
                       subPops <span class="op">=</span> [<span class="st">&quot;S1_0&quot;</span>],
                       end <span class="op">=</span> T1 <span class="op">+</span> T2 <span class="op">+</span> T3 <span class="op">-</span> <span class="dv">1</span>)
        ],

        matingScheme <span class="op">=</span> RandomMating(ops <span class="op">=</span> [
                                        Recombinator(intensity <span class="op">=</span> <span class="dv">1</span>)
                                    ]),

        gen <span class="op">=</span> T1 <span class="op">+</span> T2 <span class="op">+</span> T3

    )

    sample <span class="op">=</span> drawRandomSample(pop, sizes <span class="op">=</span> [Nsam, Nsam, Nsam])

    <span class="cf">return</span> sample


Ne <span class="op">=</span> <span class="dv">1000</span>
Nsam <span class="op">=</span> <span class="dv">25</span>
T1 <span class="op">=</span> <span class="dv">10</span>
T2 <span class="op">=</span> <span class="dv">100</span>
T3 <span class="op">=</span> <span class="dv">100</span>
s <span class="op">=</span> <span class="fl">0.1</span>
nSNP <span class="op">=</span> <span class="dv">10</span>

G  <span class="op">=</span> np.zeros([<span class="dv">3</span> <span class="op">*</span> Nsam, nSNP])

<span class="cf">for</span> i <span class="op">in</span> <span class="bu">range</span>(nSNP):
    <span class="cf">if</span> i <span class="op">&lt;</span> <span class="dv">1</span>:
        s10 <span class="op">=</span> <span class="dv">2</span> <span class="op">*</span> s
        s11 <span class="op">=</span> <span class="fl">0.0</span>
    <span class="cf">elif</span> i <span class="op">&lt;</span> <span class="dv">2</span>:
        s10 <span class="op">=</span> <span class="fl">0.0</span>
        s11 <span class="op">=</span> s
    <span class="cf">else</span>:
        s10 <span class="op">=</span> <span class="fl">0.0</span>
        s11 <span class="op">=</span> <span class="fl">0.0</span>
    res <span class="op">=</span> simulate(Ne, Nsam, T1, T2, T3, s10, s11)
    <span class="cf">for</span> j <span class="op">in</span> <span class="bu">range</span>(<span class="dv">3</span>):
        Sj <span class="op">=</span> res.genotype(j)
        <span class="cf">for</span> k <span class="op">in</span> <span class="bu">range</span>(<span class="bu">int</span>(<span class="bu">len</span>(Sj) <span class="op">/</span> <span class="dv">2</span>)):
            idx <span class="op">=</span> j <span class="op">*</span> <span class="bu">int</span>(<span class="bu">len</span>(Sj) <span class="op">/</span> <span class="dv">2</span>) <span class="op">+</span> k
            G[idx][i] <span class="op">=</span> Sj[<span class="dv">2</span> <span class="op">*</span> k] <span class="op">+</span> Sj[<span class="dv">2</span> <span class="op">*</span> k <span class="op">+</span> <span class="dv">1</span>]

np.savetxt(<span class="st">&#39;data/simuPOP.pcadapt&#39;</span>, G, fmt <span class="op">=</span> <span class="st">&#39;</span><span class="sc">%i</span><span class="st">&#39;</span>)</code></pre></div>
</div>
</div>
<div id="details-1" class="section level2">
<h2><span class="header-section-number">C.2</span> Détails</h2>
<div id="donnees-manquantes" class="section level3">
<h3><span class="header-section-number">C.2.1</span> Données manquantes</h3>
<p>Les données manquantes sont prises en compte à différentes étapes de la méthode, cela va de l’estimation de matrice de covariance à la régression linéaire multiple. Nous supposons que les données manquantes sont MAR (<em>missing at random</em>) ou MCAR (<em>missing completely at random</em>) et qu’elles ne dépendent donc pas de leur valeur (elles ne sont pas MNAR).</p>
<p>Pour chaque entrée de la matrice de covariance par individus, le calcul ne tient compte que des paires d’observations présentes.</p>
<p>Si <span class="math inline">\(X_i\)</span> et <span class="math inline">\(X_j\)</span> désignent deux individus, <span class="math inline">\(\text{Cov}(X_i, X_j) = E[(X_i - \bar{X_i})(X_j - \bar{X_j})^T]\)</span> Un estimateur de <span class="math inline">\(\text{Cov}(X_i, X_j)\)</span> tenant compte des valeurs manquantes donne</p>

<!--
The bib chunk below must go last in this document according to how R Markdown renders. More info is at http://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html
-->
<p></p>
</div>
</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-hudson2002generating">
<p>Hudson, R. R. (2002). Generating samples under a wright–Fisher neutral model of genetic variation. <em>Bioinformatics</em>, <em>18</em>(2), 337–338.</p>
</div>
<div id="ref-roux2012recent">
<p>Roux, C., Pauwels, M., Ruggiero, M.-V., Charlesworth, D., Castric, V., &amp; Vekemans, X. (2012). Recent and ancient signature of balancing selection around the s-locus in arabidopsis halleri and a. lyrata. <em>Molecular Biology and Evolution</em>, <em>30</em>(2), 435–447.</p>
</div>
<div id="ref-peng2005simupop">
<p>Peng, B., &amp; Kimmel, M. (2005). SimuPOP: A forward-time population genetics simulation environment. <em>Bioinformatics</em>, <em>21</em>(18), 3686–3687.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="informations-supplementaires.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="bibliographie.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"download": [["thesis.pdf", "PDF"], ["thesis.epub", "EPUB"], ["thesis.docx", "Word"]],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(script.src))
      script.src  = script.src.replace(/^https?:/, '');
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
