<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Méthodes statistiques en grande dimension pour l’étude de l’adaptation biologique à l’aide de larges bases de données génomiques</title>
  <meta name="description" content="Méthodes statistiques en grande dimension pour l’étude de l’adaptation biologique à l’aide de larges bases de données génomiques">
  <meta name="generator" content="bookdown 0.5 and GitBook 2.6.7">

  <meta property="og:title" content="Méthodes statistiques en grande dimension pour l’étude de l’adaptation biologique à l’aide de larges bases de données génomiques" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Méthodes statistiques en grande dimension pour l’étude de l’adaptation biologique à l’aide de larges bases de données génomiques" />
  
  
  

<meta name="author" content="Keurcien LUU">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="adaptation-locale-1.html">
<link rel="next" href="introgression-adaptative.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />









<style type="text/css">
div.sourceCode { overflow-x: auto; }
table.sourceCode, tr.sourceCode, td.lineNumbers, td.sourceCode {
  margin: 0; padding: 0; vertical-align: baseline; border: none; }
table.sourceCode { width: 100%; line-height: 100%; }
td.lineNumbers { text-align: right; padding-right: 4px; padding-left: 4px; color: #aaaaaa; border-right: 1px solid #aaaaaa; }
td.sourceCode { padding-left: 5px; }
code > span.kw { color: #007020; font-weight: bold; } /* Keyword */
code > span.dt { color: #902000; } /* DataType */
code > span.dv { color: #40a070; } /* DecVal */
code > span.bn { color: #40a070; } /* BaseN */
code > span.fl { color: #40a070; } /* Float */
code > span.ch { color: #4070a0; } /* Char */
code > span.st { color: #4070a0; } /* String */
code > span.co { color: #60a0b0; font-style: italic; } /* Comment */
code > span.ot { color: #007020; } /* Other */
code > span.al { color: #ff0000; font-weight: bold; } /* Alert */
code > span.fu { color: #06287e; } /* Function */
code > span.er { color: #ff0000; font-weight: bold; } /* Error */
code > span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
code > span.cn { color: #880000; } /* Constant */
code > span.sc { color: #4070a0; } /* SpecialChar */
code > span.vs { color: #4070a0; } /* VerbatimString */
code > span.ss { color: #bb6688; } /* SpecialString */
code > span.im { } /* Import */
code > span.va { color: #19177c; } /* Variable */
code > span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code > span.op { color: #666666; } /* Operator */
code > span.bu { } /* BuiltIn */
code > span.ex { } /* Extension */
code > span.pp { color: #bc7a00; } /* Preprocessor */
code > span.at { color: #7d9029; } /* Attribute */
code > span.do { color: #ba2121; font-style: italic; } /* Documentation */
code > span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code > span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code > span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#la-genetique-des-populations"><i class="fa fa-check"></i><b>1.1</b> La génétique des populations</a><ul>
<li class="chapter" data-level="1.1.1" data-path="introduction.html"><a href="introduction.html#levolution-comme-point-de-depart"><i class="fa fa-check"></i><b>1.1.1</b> L’évolution comme point de départ</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#a-lorigine-de-la-variabilite-genetique"><i class="fa fa-check"></i><b>1.2</b> À l’origine de la variabilité génétique</a><ul>
<li class="chapter" data-level="1.2.1" data-path="introduction.html"><a href="introduction.html#la-theorie-de-levolution"><i class="fa fa-check"></i><b>1.2.1</b> La théorie de l’évolution</a></li>
<li class="chapter" data-level="1.2.2" data-path="introduction.html"><a href="introduction.html#levolution-dune-theorie"><i class="fa fa-check"></i><b>1.2.2</b> L’évolution d’une théorie</a></li>
<li class="chapter" data-level="1.2.3" data-path="introduction.html"><a href="introduction.html#forces-evolutives"><i class="fa fa-check"></i><b>1.2.3</b> Forces évolutives</a></li>
<li class="chapter" data-level="1.2.4" data-path="introduction.html"><a href="introduction.html#adaptation-locale"><i class="fa fa-check"></i><b>1.2.4</b> Adaptation locale</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#donnees-de-polymorphismes-genetiques"><i class="fa fa-check"></i><b>1.3</b> Données de polymorphismes génétiques</a><ul>
<li class="chapter" data-level="1.3.1" data-path="introduction.html"><a href="introduction.html#des-donnees-en-grande-dimension"><i class="fa fa-check"></i><b>1.3.1</b> Des données en grande dimension</a></li>
<li class="chapter" data-level="1.3.2" data-path="introduction.html"><a href="introduction.html#les-marqueurs-genetiques"><i class="fa fa-check"></i><b>1.3.2</b> Les marqueurs génétiques</a></li>
<li class="chapter" data-level="1.3.3" data-path="introduction.html"><a href="introduction.html#encodage-des-donnees-genetiques"><i class="fa fa-check"></i><b>1.3.3</b> Encodage des données génétiques</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="introduction.html"><a href="introduction.html#les-motivations-de-la-these"><i class="fa fa-check"></i><b>1.4</b> Les motivations de la thèse</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html"><i class="fa fa-check"></i><b>2</b> Adaptation locale</a><ul>
<li class="chapter" data-level="2.1" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#letat-de-lart-pour-les-scans-genomiques"><i class="fa fa-check"></i><b>2.1</b> L’état de l’art pour les scans génomiques</a><ul>
<li class="chapter" data-level="2.1.1" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#modeles-demographiques"><i class="fa fa-check"></i><b>2.1.1</b> Modèles démographiques</a></li>
<li class="chapter" data-level="2.1.2" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#lindice-de-fixation"><i class="fa fa-check"></i><b>2.1.2</b> L’indice de fixation</a></li>
<li class="chapter" data-level="2.1.3" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#test-de-lewontin-krakauer"><i class="fa fa-check"></i><b>2.1.3</b> Test de Lewontin-Krakauer</a></li>
<li class="chapter" data-level="2.1.4" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#le-modele-f"><i class="fa fa-check"></i><b>2.1.4</b> Le modèle <span class="math inline">\(F\)</span></a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#lanalyse-en-composantes-principales-en-genetique-des-populations"><i class="fa fa-check"></i><b>2.2</b> L’Analyse en Composantes Principales en génétique des populations</a><ul>
<li class="chapter" data-level="2.2.1" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#principe-de-lanalyse-en-composantes-principales"><i class="fa fa-check"></i><b>2.2.1</b> Principe de l’Analyse en Composantes Principales</a></li>
<li class="chapter" data-level="2.2.2" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#apparentement-genetique-interindividuel"><i class="fa fa-check"></i><b>2.2.2</b> Apparentement génétique interindividuel</a></li>
<li class="chapter" data-level="2.2.3" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#applications-en-genetique-des-populations"><i class="fa fa-check"></i><b>2.2.3</b> Applications en génétique des populations</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#statistiques-de-test-basees-sur-lanalyse-en-composantes-principales"><i class="fa fa-check"></i><b>2.3</b> Statistiques de test basées sur l’Analyse en Composantes Principales</a><ul>
<li class="chapter" data-level="2.3.1" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#la-communalite"><i class="fa fa-check"></i><b>2.3.1</b> La communalité</a></li>
<li class="chapter" data-level="2.3.2" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#la-distance-robuste-de-mahalanobis"><i class="fa fa-check"></i><b>2.3.2</b> La distance robuste de Mahalanobis</a></li>
<li class="chapter" data-level="" data-path="adaptation-locale-1.html"><a href="adaptation-locale-1.html#estimation-robuste-de-la-matrice-de-covariance"><i class="fa fa-check"></i>Estimation robuste de la matrice de covariance</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="section.html"><a href="section.html"><i class="fa fa-check"></i><b>3</b> </a><ul>
<li class="chapter" data-level="3.1" data-path="section.html"><a href="section.html#article-1"><i class="fa fa-check"></i><b>3.1</b> Article 1</a><ul>
<li class="chapter" data-level="" data-path="section.html"><a href="section.html#contributions"><i class="fa fa-check"></i>Contributions</a></li>
<li class="chapter" data-level="" data-path="section.html"><a href="section.html#resume"><i class="fa fa-check"></i>Résumé</a></li>
<li class="chapter" data-level="" data-path="section.html"><a href="section.html#abstract"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="section.html"><a href="section.html#significance-statement"><i class="fa fa-check"></i>Significance Statement</a></li>
<li class="chapter" data-level="" data-path="section.html"><a href="section.html#introduction-1"><i class="fa fa-check"></i>Introduction</a></li>
<li class="chapter" data-level="" data-path="section.html"><a href="section.html#new-method"><i class="fa fa-check"></i>New Method</a></li>
<li class="chapter" data-level="" data-path="section.html"><a href="section.html#results"><i class="fa fa-check"></i>Results</a></li>
<li class="chapter" data-level="" data-path="section.html"><a href="section.html#discussion"><i class="fa fa-check"></i>Discussion</a></li>
<li class="chapter" data-level="" data-path="section.html"><a href="section.html#materials-and-methods"><i class="fa fa-check"></i>Materials and Methods</a></li>
<li class="chapter" data-level="" data-path="section.html"><a href="section.html#acknowledgments"><i class="fa fa-check"></i>Acknowledgments</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="section.html"><a href="section.html#article-2"><i class="fa fa-check"></i><b>3.2</b> Article 2</a><ul>
<li class="chapter" data-level="" data-path="section.html"><a href="section.html#abstract-1"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="section.html"><a href="section.html#introduction-2"><i class="fa fa-check"></i>Introduction</a></li>
<li class="chapter" data-level="" data-path="section.html"><a href="section.html#statistical-and-computational-approach"><i class="fa fa-check"></i>Statistical and computational approach</a></li>
<li class="chapter" data-level="" data-path="section.html"><a href="section.html#materials-and-methods-1"><i class="fa fa-check"></i>Materials and methods</a></li>
<li class="chapter" data-level="" data-path="section.html"><a href="section.html#results-1"><i class="fa fa-check"></i>Results</a></li>
<li class="chapter" data-level="" data-path="section.html"><a href="section.html#discussion-1"><i class="fa fa-check"></i>Discussion</a></li>
<li class="chapter" data-level="" data-path="section.html"><a href="section.html#acknowledgements"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="section.html"><a href="section.html#data-accessibility"><i class="fa fa-check"></i>Data accessibility</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html"><i class="fa fa-check"></i><b>4</b> Introgression adaptative</a><ul>
<li class="chapter" data-level="4.1" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#methode-basee-sur-lutilisation-de-coefficients-de-metissage-locaux."><i class="fa fa-check"></i><b>4.1</b> Méthode basée sur l’utilisation de coefficients de métissage locaux.</a><ul>
<li class="chapter" data-level="4.1.1" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#coefficients-de-metissage-locaux"><i class="fa fa-check"></i><b>4.1.1</b> Coefficients de métissage locaux</a></li>
<li class="chapter" data-level="4.1.2" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#statistique-de-test"><i class="fa fa-check"></i><b>4.1.2</b> Statistique de test</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#methodes-basees-sur-la-diversite-nucleotidique."><i class="fa fa-check"></i><b>4.2</b> Méthodes basées sur la diversité nucléotidique.</a><ul>
<li class="chapter" data-level="" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#diversite-nucleotidique-par-paire-de-sequences"><i class="fa fa-check"></i>Diversité nucléotidique par paire de séquences</a></li>
<li class="chapter" data-level="4.2.1" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#le-modele-abba-baba"><i class="fa fa-check"></i><b>4.2.1</b> Le modèle ABBA-BABA</a></li>
<li class="chapter" data-level="4.2.2" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#analyse-en-composantes-principales-locale"><i class="fa fa-check"></i><b>4.2.2</b> Analyse en Composantes Principales locale</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#simulations"><i class="fa fa-check"></i><b>4.3</b> Simulations</a><ul>
<li class="chapter" data-level="4.3.1" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#donnees-de-peupliers"><i class="fa fa-check"></i><b>4.3.1</b> Données de peupliers</a></li>
<li class="chapter" data-level="4.3.2" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#generation-aleatoire-dindividus-hybrides"><i class="fa fa-check"></i><b>4.3.2</b> Génération aléatoire d’individus hybrides</a></li>
<li class="chapter" data-level="4.3.3" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#simulations-a-partir-de-ms-et-seq-gen"><i class="fa fa-check"></i><b>4.3.3</b> Simulations à partir de ms et Seq-Gen</a></li>
<li class="chapter" data-level="4.3.4" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#resultats-de-la-comparaison-des-logiciels"><i class="fa fa-check"></i><b>4.3.4</b> Résultats de la comparaison des logiciels</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#article-3"><i class="fa fa-check"></i><b>4.4</b> Article 3</a><ul>
<li class="chapter" data-level="" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#introduction-3"><i class="fa fa-check"></i>Introduction</a></li>
<li class="chapter" data-level="" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#materials-and-methods-2"><i class="fa fa-check"></i>Materials and methods</a></li>
<li class="chapter" data-level="" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#results-2"><i class="fa fa-check"></i>Results</a></li>
<li class="chapter" data-level="" data-path="introgression-adaptative.html"><a href="introgression-adaptative.html#discussion-2"><i class="fa fa-check"></i>Discussion</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="aspect-computationnel.html"><a href="aspect-computationnel.html"><i class="fa fa-check"></i><b>5</b> Aspect computationnel</a><ul>
<li class="chapter" data-level="5.1" data-path="aspect-computationnel.html"><a href="aspect-computationnel.html#donnees-dentree"><i class="fa fa-check"></i><b>5.1</b> Données d’entrée</a></li>
<li class="chapter" data-level="5.2" data-path="aspect-computationnel.html"><a href="aspect-computationnel.html#memory-mapping"><i class="fa fa-check"></i><b>5.2</b> Memory-mapping</a></li>
<li class="chapter" data-level="5.3" data-path="aspect-computationnel.html"><a href="aspect-computationnel.html#interface-shiny"><i class="fa fa-check"></i><b>5.3</b> Interface Shiny</a></li>
<li class="chapter" data-level="5.4" data-path="aspect-computationnel.html"><a href="aspect-computationnel.html#acp-et-donnees-manquantes"><i class="fa fa-check"></i><b>5.4</b> ACP et données manquantes</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="conclusion.html"><a href="conclusion.html"><i class="fa fa-check"></i>Conclusion</a></li>
<li class="appendix"><span><b>Appendice</b></span></li>
<li class="chapter" data-level="A" data-path="details.html"><a href="details.html"><i class="fa fa-check"></i><b>A</b> Détails</a><ul>
<li class="chapter" data-level="A.0.1" data-path="details.html"><a href="details.html#rapport-entre-la-communalite-et-lindice-de-fixation"><i class="fa fa-check"></i><b>A.0.1</b> Rapport entre la communalité et l’indice de fixation</a></li>
<li class="chapter" data-level="A.0.2" data-path="details.html"><a href="details.html#lanalyse-en-composantes-principales-regularisee"><i class="fa fa-check"></i><b>A.0.2</b> L’Analyse en Composantes Principales régularisée</a></li>
</ul></li>
<li class="chapter" data-level="B" data-path="informations-supplementaires.html"><a href="informations-supplementaires.html"><i class="fa fa-check"></i><b>B</b> Informations supplémentaires</a><ul>
<li class="chapter" data-level="" data-path="informations-supplementaires.html"><a href="informations-supplementaires.html#article-1-1"><i class="fa fa-check"></i>Article 1 </a></li>
<li class="chapter" data-level="" data-path="informations-supplementaires.html"><a href="informations-supplementaires.html#article-2-1"><i class="fa fa-check"></i>Article 2</a></li>
<li class="chapter" data-level="" data-path="informations-supplementaires.html"><a href="informations-supplementaires.html#article-3-1"><i class="fa fa-check"></i>Article 3</a></li>
</ul></li>
<li class="chapter" data-level="C" data-path="r-python.html"><a href="r-python.html"><i class="fa fa-check"></i><b>C</b> R &amp; Python</a><ul>
<li class="chapter" data-level="C.1" data-path="r-python.html"><a href="r-python.html#simulations-et-modeles-demographiques"><i class="fa fa-check"></i><b>C.1</b> Simulations et modèles démographiques</a><ul>
<li class="chapter" data-level="C.1.1" data-path="r-python.html"><a href="r-python.html#modele-en-ile"><i class="fa fa-check"></i><b>C.1.1</b> Modèle en île</a></li>
<li class="chapter" data-level="C.1.2" data-path="r-python.html"><a href="r-python.html#modele-de-divergence"><i class="fa fa-check"></i><b>C.1.2</b> Modèle de divergence</a></li>
</ul></li>
<li class="chapter" data-level="C.2" data-path="r-python.html"><a href="r-python.html#details-1"><i class="fa fa-check"></i><b>C.2</b> Détails</a><ul>
<li class="chapter" data-level="C.2.1" data-path="r-python.html"><a href="r-python.html#donnees-manquantes"><i class="fa fa-check"></i><b>C.2.1</b> Données manquantes</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="bibliographie.html"><a href="bibliographie.html"><i class="fa fa-check"></i>Bibliographie</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Méthodes statistiques en grande dimension pour l’étude de l’adaptation biologique à l’aide de larges bases de données génomiques</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="section" class="section level1">
<h1><span class="header-section-number">Chapitre 3</span> </h1>
<div id="resultats-principaux-1" class="section level4">
<h4><span class="header-section-number">3.0.0.1</span> Résultats principaux</h4>
<p><strong>La distance robuste de Mahalanobis est adaptée à des scénarios démographiques variés.</strong> Nous montrons que la distance robuste de Mahalanobis donne de meilleurs résultats en termes de puissance statistique, indépendamment du modèle démographique sous-jacent (à l’exception du modèle en îles où les résultats sont équivalents).</p>
<p><strong>En comparaison avec les autres méthodes de scan génomique, la distance robuste de Mahalanobis permet de mieux contrôler le taux de fausses découvertes.</strong></p>
<p><strong>La distance robuste de Mahalanobis généralise la statistique <span class="math inline">\(T_{F-LK}\)</span>.</strong></p>

</div>
<div id="article-1" class="section level2">
<h2><span class="header-section-number">3.1</span> Article 1</h2>
<div id="contributions" class="section level3 unnumbered">
<h3>Contributions</h3>
<p>Par souci de portabilité et de simplification d’utilisation, nous avons opté pour le développement d’une librairie R reprenant les algorithmes implémentés dans le logiciel <em>PCAdapt fast</em> par Nicolas Duforet-Frebourg <span class="citation">(Duforet-Frebourg, <a href="#ref-duforet2014statistiques">2014</a>)</span>. J’ai pu également adapté les travaux de Nicolas Duforet-Frebourg pour la production et l’analyse des diverses simulations étudiées dans l’article qui suit.</p>
</div>
<div id="resume" class="section level3 unnumbered">
<h3>Résumé</h3>
<p>De nombreuses méthodes analytiques ont été développées dans le but de détecter des régions génomiques mettant en évidence un évènement de sélection naturelle. Dans cet article nous abordons la possibilité de réaliser des scans génomiques pour la sélection en utilisant l’Analyse en Composantes Principales (ACP). Nous expliquons comment l’indice de différenciation génétique, communément appelé <span class="math inline">\(F_{ST}\)</span>, peut être vu comme la proportion de variance expliquée par les composantes principales. La corrélation entre les variants génétiques et les composantes principales donne un cadre conceptuel permettant la détection de variants génétiques impliqués dans les processus d’adaptation locale sans qu’il n’y ait besoin de définir de populations <em>a priori</em>. Afin de valider notre approche, nous considérons 850 individus qui ont été échantillonnées en Afrique, en Asie et en Europe. Le nombre de variants génétiques est de l’ordre de 36 millions obtenus avec une faible couverture (3x). La corrélation entre la variation génétique et chaque composante principale permet d’identifier des gènes connus pour avoir été sélectionnés positivement (EDAR, SLC24A5, SLC45A2, DARC), mais également de nouveaux gènes candidats (APPBPP2, TP1A1, RTTN, KCNMA, MYO5C) et de l’ARN non-codant. Nous identifions deux mécanismes biologiques responsables de l’adaptation polygénique et qui sont associés au système immunitaire inné (<span class="math inline">\(\beta\)</span>-défensines) et au métabolisme lipidique<a href="#fn15" class="footnoteRef" id="fnref15"><sup>15</sup></a> (oxydation des acides gras). Une analyse supplémentaire de données européennes montre que le scan génomique à partir de l’ACP permet de retrouver des exemples classiques d’adaptation locale même en l’absence de population explicitement définies. Les statistiques basées sur l’ACP, implémentées dans la librairie R <em>pcadapt</em> ainsi que dans le logiciel libre <em>PCAdapt fast</em>, exhibent des signaux connus d’adaptation chez l’Homme, ce qui s’avère prometteur pour de futurs projets de séquençage de génome entier, notamment lorsque la définition de populations est impossible.</p>
<p><img src="figure/ndf.png" width="\textwidth" style="display: block; margin: auto;" /></p>
</div>
<div id="abstract" class="section level3 unnumbered">
<h3>Abstract</h3>
<p>To characterize natural selection, various analytical methods for detecting candidate genomic regions have been developed. We propose to perform genome-wide scans of natural selection using principal component analysis (PCA). We show that the common <span class="math inline">\(F_{ST}\)</span> index of genetic differentiation between populations can be viewed as the proportion of variance explained by the principal components. Considering the correlations between genetic variants and each principal component provides a conceptual framework to detect genetic variants involved in local adaptation without any prior definition of populations. To validate the PCA-based approach, we consider the 1000 Genomes data (phase 1) considering 850 individuals coming from Africa, Asia, and Europe. The number of genetic variants is of the order of 36 millions obtained with a low-coverage sequencing depth (3x). The correlations between genetic variation and each principal component provide well-known targets for positive selection (EDAR, SLC24A5, SLC45A2, DARC), and also new candidate genes (APPBPP2, TP1A1, RTTN, KCNMA, MYO5C) and noncoding RNAs. In addition to identifying genes involved in biological adaptation, we identify two biological pathways involved in polygenic adaptation that are related to the innate immune system (beta defensins) and to lipid metabolism (fatty acid omega oxidation). An additional analysis of European data shows that a genome scan based on PCA retrieves classical examples of local adaptation even when there are no well-defined populations. PCA-based statistics, implemented in the <em>PCAdapt</em> R package and the <em>PCAdapt fast</em> open-source software, retrieve well-known signals of human adaptation, which is encouraging for future whole-genome sequencing project, especially when defining populations is difficult.</p>
</div>
<div id="significance-statement" class="section level3 unnumbered">
<h3>Significance Statement</h3>
<p>Positive natural selection or local adaptation is the driving force behind the adaption of individuals to their environment. To identify genomic regions responsible for local adaptation, we propose to consider the genetic markers that are the most related with population structure. To uncover genetic structure, we consider principal component analysis that identifies the primary axes of variation in the data. Our approach generalizes common approaches for genome scan based on measures of population differentiation. To validate our approach, we consider the human 1000 Genomes data and find well-known targets for positive selection as well as new candidate regions. We also find evidence of polygenic adaptation for two biological pathways related to the innate immune system and to lipid metabolism.</p>
</div>
<div id="introduction-1" class="section level3 unnumbered">
<h3>Introduction</h3>
<p>Because of the flood of genomic data, the ability to understand the genetic architecture of natural selection has dramatically increased. Of particular interest is the study of local positive selection which explains why individuals are adapted to their local environment. In humans, the availability of genomic data fostered the identification of loci involved in positive selection <span class="citation">(Barreiro, Laval, Quach, Patin, &amp; Quintana-Murci, <a href="#ref-barreiro2008natural">2008</a>; Grossman et al., <a href="#ref-grossman2013identifying">2013</a>; Pickrell et al., <a href="#ref-pickrell2009signals">2009</a>; Sabeti et al., <a href="#ref-sabeti2007genome">2007</a>)</span>. Local positive selection tends to increase genetic differentiation, which can be measured by difference of allele frequencies between populations <span class="citation">(Colonna et al., <a href="#ref-colonna2014human">2014</a>; Nielsen, <a href="#ref-nielsen2005molecular">2005</a>; Sabeti et al., <a href="#ref-sabeti2006positive">2006</a>)</span>. For instance, a mutation in the DARC gene that confers resistance to malaria is fixed in sub-Saharan African populations whereas it is absent elsewhere <span class="citation">(Hamblin, Thompson, &amp; Di Rienzo, <a href="#ref-hamblin2002complex">2002</a>)</span>. In addition to the variants that confer resistance to pathogens, genome scans also identify other genetic variants, and many of these are involved in human metabolic phenotypes and morphological traits <span class="citation">(Barreiro et al., <a href="#ref-barreiro2008natural">2008</a>; Hancock et al., <a href="#ref-hancock2010human">2010</a>)</span>.</p>
<p>In order to provide a list of variants potentially involved in natural selection, genome scans compute measures of genetic differentiation between populations and consider that extreme values correspond to candidate regions <span class="citation">(G. Luikart, England, Tallmon, Jordan, &amp; Taberlet, <a href="#ref-luikart2003power">2003</a>)</span>. The most widely used index of genetic differentiation is the <span class="math inline">\(F_{ST}\)</span> index which measures the amount of genetic variation that is explained by variation between populations <span class="citation">(Excoffier, Smouse, &amp; Quattro, <a href="#ref-excoffier1992analysis">1992</a>)</span>. However the <span class="math inline">\(F_{ST}\)</span> statistic requires to group individuals into populations which can be problematic when ascertainment of population structure does not show well-separated clusters of individuals (e.g., <span class="citation">Novembre et al. (<a href="#ref-novembre2008genes">2008</a>)</span>). Other statistics related to <span class="math inline">\(F_{ST}\)</span> have been derived to reduce the false discovery rate (FDR) obtained with <span class="math inline">\(F_{ST}\)</span> but they also work at the scale of populations <span class="citation">(Bonhomme et al., <a href="#ref-bonhomme2010detecting">2010</a>; Fariello, Boitard, Naya, SanCristobal, &amp; Servin, <a href="#ref-fariello2013detecting">2013</a>; Günther &amp; Coop, <a href="#ref-gunther2013robust">2013</a>)</span>. Grouping individuals into populations can be subjective, and important signals of selection may be missed with an inadequate choice of populations <span class="citation">(W.-Y. Yang, Novembre, Eskin, &amp; Halperin, <a href="#ref-yang2012model">2012</a>)</span>. We have previously developed an individual-based approach for selection scan based on a Bayesian factor model but the Markov chain Monte Carlo (MCMC) algorithm required for model fitting does not scale well to large data sets containing a million of variants or more <span class="citation">(Duforet-Frebourg et al., <a href="#ref-duforet2014genome">2014</a>)</span>.</p>
<p>We propose to detect candidates for natural selection using principal component analysis (PCA). PCA is a technique of multivariate analysis used to ascertain population structure <span class="citation">(N. Patterson, Price, &amp; Reich, <a href="#ref-patterson2006population">2006</a>)</span>. PCA decomposes the total genetic variation into <span class="math inline">\(K\)</span> axes of genetic variation called principal components. In population genomics, the principal components can correspond to evolutionary processes such as evolutionary divergence between populations <span class="citation">(G. McVean, <a href="#ref-mcvean2009genealogical">2009</a>)</span>. Using simulations of an island model and of a model of population fission followed by isolation, we show that the common <span class="math inline">\(F_{ST}\)</span> statistic corresponds to the proportion of variation explained by the first K principal components when K has been properly chosen. With this point of view, the <span class="math inline">\(F_{ST}\)</span> of a given variant is obtained by summing the squared correlations of the first <span class="math inline">\(K\)</span> principal components opening the door to new statistics for genome scans. At a genome-wide level, it is known that there is a relationship between <span class="math inline">\(F_{ST}\)</span> and PCA <span class="citation">(G. McVean, <a href="#ref-mcvean2009genealogical">2009</a>)</span>, and our simulations show that the relationship also applies at the level of a single variant.</p>
<p>The advantages of performing a genome scan based on PCA are multiple: it does not require to group individuals into populations, the computational burden is considerably reduced compared with genome scan approaches based on MCMC algorithms <span class="citation">(Foll &amp; Gaggiotti, <a href="#ref-foll2008genome">2008</a>)</span>; <span class="citation">Riebler, Held, &amp; Stephan (<a href="#ref-riebler2008bayesian">2008</a>)</span>; <span class="citation">Günther &amp; Coop (<a href="#ref-gunther2013robust">2013</a>)</span>; <span class="citation">Duforet-Frebourg et al. (<a href="#ref-duforet2014genome">2014</a>)</span>], and candidate single nucleotide polymorphisms (SNPs) can be related to different evolutionary events that correspond to the different principal components. Using simulations and the 1000 Genomes data, we show that PCA can provide useful insights for genome scans. Looking at the correlations between SNPs and principal components provides a novel conceptual framework to detect genomic regions that are candidates for local adaptation.</p>
</div>
<div id="new-method" class="section level3 unnumbered">
<h3>New Method</h3>
<div id="new-statistics-for-genome-scan" class="section level4 unnumbered">
<h4>New Statistics for Genome Scan</h4>
<p>We denote by <span class="math inline">\(Y\)</span> the <span class="math inline">\((n \times p)\)</span> centered and scaled genotype matrix where <span class="math inline">\(n\)</span> is the number of individuals and <span class="math inline">\(p\)</span> is the number of loci. The new statistics for genome scan are based on PCA. The objective of PCA is to find a new set of orthogonal variables called the principal components, which are linear combinations of (centered and standardized) allele counts, such that the projections of the data onto these axes lead to an optimal summary of the data. To present the method, we introduce the truncated singular value decomposition (SVD) that approximates the data matrix <span class="math inline">\(Y\)</span> by a matrix of smaller rank</p>
<span class="math display" id="eq:svd">\[\begin{equation}
  Y \approx U \Sigma V^T,
  \tag{3.1}
\end{equation}\]</span>
<p>where <span class="math inline">\(U\)</span> is a <span class="math inline">\((n \times K)\)</span> orthonormal matrix, <span class="math inline">\(V\)</span> is a <span class="math inline">\((p \times K)\)</span> orthonormal matrix, <span class="math inline">\(\Sigma\)</span> is a diagonal <span class="math inline">\((K \times K)\)</span> matrix and <span class="math inline">\(K\)</span> corresponds to the rank of the approximation. The solution of PCA with <span class="math inline">\(K\)</span> components can be obtained using the truncated SVD: the <span class="math inline">\(K\)</span> columns of <span class="math inline">\(V\)</span> contain the coefficients of the new orthogonal variables, the <span class="math inline">\(K\)</span> columns of <span class="math inline">\(U\)</span> contain the projections (called “scores”) of the original variables onto the principal components and capture population structure (supplementary fig. S1, Supplementary Material online), and the squares of the elements of <span class="math inline">\(\Sigma\)</span> are proportional to the proportion of variance explained by each principal component <span class="citation">(Jolliffe, <a href="#ref-jolliffe1986principal">1986</a>)</span>. We denote the diagonal elements of <span class="math inline">\(\Sigma\)</span> by <span class="math inline">\(\sqrt{\lambda_k}, \; k = 1, \dots, K\)</span> where the <span class="math inline">\(\lambda_k\)</span>’s are the ranked eigenvalues of the matrix <span class="math inline">\(YY^T\)</span>. Denoting by <span class="math inline">\(V_{jk}\)</span>, the entry of <span class="math inline">\(V\)</span> at the <span class="math inline">\(j^{th}\)</span> line and <span class="math inline">\(k^{th}\)</span> column, then the correlation <span class="math inline">\(\rho_{jk}\)</span> between the <span class="math inline">\(j^{th}\)</span> SNP and the <span class="math inline">\(k^{th}\)</span> principal component is given by <span class="math inline">\(\rho_{jk} = \sqrt{\lambda_{k}}V_{jk}/\sqrt{n-1}\)</span> <span class="citation">(Cadima &amp; Jolliffe, <a href="#ref-cadima1995loading">1995</a>)</span>. In the following, the statistics <span class="math inline">\(\rho_{jk}\)</span> are referred to as “loadings” and will be used for detecting selection. The second statistic we consider for genome scan corresponds to the proportion of variance of a SNP that is explained by the first <span class="math inline">\(K\)</span> PCs. It is called the communality in exploratory factor analysis because it is the variance of observed variables accounted for by the common factors, which correspond to the first <span class="math inline">\(K\)</span> PCs. Because the principal components are orthogonal to each other, the proportion of variance explained by the first <span class="math inline">\(K\)</span> principal components is equal to the sum of the squared correlations with the first <span class="math inline">\(K\)</span> principal components. Denoting by <span class="math inline">\(h_j^2\)</span> the communality of the <span class="math inline">\(j^{th}\)</span> SNP, we have</p>
<span class="math display" id="eq:communality">\[\begin{equation}
  h_j^2 = \sum_{k=1}^K \rho_{jk}^2.
  \tag{3.2}
\end{equation}\]</span>
<p>The last statistic we consider for genome scans sums the squared of normalized loadings. It is defined as <span class="math inline">\({h^{\prime}_j}^2 = \sum_{k=1}^K V_{jk}^2\)</span>. Compared to the communality <span class="math inline">\(h^2\)</span>, the statistic <span class="math inline">\({h^{\prime}_j}^2\)</span> should theoretically give the same importance to each PC because the normalized loadings are on the same scale as we have <span class="math inline">\(\sum_{j=1}^K V_{jk}^2 = 1\)</span>, for <span class="math inline">\(k = 1, \dots, K\)</span>.</p>
</div>
<div id="numerical-computations" class="section level4 unnumbered">
<h4>Numerical Computations</h4>
<p>The method of selection scan should be able to handle a large number <span class="math inline">\(p\)</span> of genetic variants. In order to compute truncated SVD with large values of <span class="math inline">\(p\)</span>, we compute the <span class="math inline">\(n \times n\)</span> covariance matrix <span class="math inline">\(\Omega = YY^T/(p-1)\)</span>. The covariance matrix <span class="math inline">\(\Omega\)</span> is typically of much smaller dimension than the <span class="math inline">\(p \times p\)</span> covariance matrix. Considering the <span class="math inline">\(n \times n\)</span> covariance matrix <span class="math inline">\(\Omega\)</span> speeds up matrix operations. Computation of the covariance matrix is the most costly operation and it requires a number of arithmetic operations proportional to <span class="math inline">\(p n^2\)</span>. After computing the covariance matrix <span class="math inline">\(\Omega\)</span>, we compute its first <span class="math inline">\(K\)</span> eigenvalues and eigenvectors to find <span class="math inline">\(\Sigma^2/(p-1)\)</span> and <span class="math inline">\(U\)</span>. Eigenanalysis is performed with the <em>dsyevr</em> routine of the linear algebra package LAPACK <span class="citation">(Anderson et al., <a href="#ref-anderson1999lapack">1999</a>)</span>. The matrix <span class="math inline">\(V\)</span>, which captures the relationship between each SNPs and population structure, is obtained by the matrix operation <span class="math inline">\(V^T = \Sigma^{-1} U^T Y\)</span>. The software <em>PCAdapt fast</em>, process data as a stream and never store in order to have a very low memory access whatever the size of the data.</p>
</div>
</div>
<div id="results" class="section level3 unnumbered">
<h3>Results</h3>
<div id="island-model" class="section level4 unnumbered">
<h4>Island Model</h4>
<p>To investigate the relationship between communality <span class="math inline">\(h^2\)</span> and <span class="math inline">\(F_{ST}\)</span>, we consider an island model with three islands. We use <span class="math inline">\(K = 2\)</span> when performing PCA because there are three islands. We choose a value of the migration rate that generates a mean <span class="math inline">\(F_{ST}\)</span> value (across the 1,400 neutral SNPs) of 4%. We consider five different simulations with varying strengths of selection for the 100 adaptive SNPs. In all simulations, the <span class="math inline">\(R^2\)</span> correlation coefficient between <span class="math inline">\(h^2\)</span> and <span class="math inline">\(F_{ST}\)</span> is larger than 98%. Considering as candidate SNPs the 1% of the SNPs with largest values of <span class="math inline">\(F_{ST}\)</span> or of <span class="math inline">\(h^2\)</span>, we find that the overlap coefficient between the two sets of SNPs is comprised between 88% and 99%. When varying the strength of selection for adaptive SNPs, we find that the relative difference of FDRs obtained with <span class="math inline">\(F_{ST}\)</span> (top 1%) and with <span class="math inline">\(h^2\)</span> (top 1%) is smaller than 5%. The similar values of FDR obtained with <span class="math inline">\(h^2\)</span> and with <span class="math inline">\(F_{ST}\)</span> decrease for increasing strength of selection (supplementary fig. S2, Supplementary Material online).</p>
</div>
<div id="divergence-model" class="section level4 unnumbered">
<h4>Divergence Model</h4>
<p>To compare the performance of different PCA-based summary statistics, we simulate genetic variation in models of population divergence. The divergence models assume that there are three populations, <span class="math inline">\(A\)</span>, <span class="math inline">\(B_1\)</span> and <span class="math inline">\(B_2\)</span> with <span class="math inline">\(B_1\)</span> and <span class="math inline">\(B_2\)</span> being the most related populations (figs. <a href="section.html#fig:ndf-fig1">3.1</a> and <a href="section.html#fig:ndf-fig2">3.2</a>). The first simulation scheme assumes that local adaptation took place in the lineages corresponding to the environments of populations <span class="math inline">\(A\)</span> and <span class="math inline">\(B_1\)</span> (fig. <a href="section.html#fig:ndf-fig1">3.1</a>). The SNPs, which are assumed to be independent, are divided into three groups: 9,500 SNPs evolve neutrally, 250 SNPs confer a selective advantage in the environment of <span class="math inline">\(A\)</span>, and 250 other SNPs confer a selective advantage in the environment of <span class="math inline">\(B_1\)</span>. Genetic differentiation, measured by pairwise <span class="math inline">\(F_{ST}\)</span>, is equal to 14% when comparing population <span class="math inline">\(A\)</span> to the other ones and is equal to 5% when comparing populations <span class="math inline">\(B_1\)</span> and <span class="math inline">\(B_2\)</span>. Performing PCA with <span class="math inline">\(K = 2\)</span> shows that the first component separates population <span class="math inline">\(A\)</span> from <span class="math inline">\(B_1\)</span> and <span class="math inline">\(B_2\)</span> whereas the second component separates <span class="math inline">\(B_1\)</span> from <span class="math inline">\(B_2\)</span> (supplementary fig. S1, Supplementary Material online). The choice of <span class="math inline">\(K = 2\)</span> is evident when looking at the scree plot because the eigenvalues, which are proportional to the proportion of variance explained by each PC, drop beyond <span class="math inline">\(K = 2\)</span> and stay almost constant as <span class="math inline">\(K\)</span> further increases (supplementary fig. S3, Supplementary Material online).</p>

<div class="figure" style="text-align: center"><span id="fig:ndf-fig1"></span>
<img src="figure/ndf-fig1.png" alt="Repartition of the 1% top-ranked SNPs for each PCA-based statistic under a divergence model with two types of adaptive constraints. Thicker and colored lineages correspond to lineages where adaptation took place. The squared loadings with PC1 \(\rho_{j1}^2\) pick a large proportion of SNPs involved in selection in population \(A\) whereas the squared loadings with PC2 \(\rho_{j2}^2\) pick SNPs involved in selection in population \(B_1\). This difference is reflected in the different repartition of the top-ranked SNPs for the communality \(h^2\) and the statistic \({h^{\prime}}^2\)." width="\textwidth" />
<p class="caption">
Figure 3.1: Repartition of the 1% top-ranked SNPs for each PCA-based statistic under a divergence model with two types of adaptive constraints. Thicker and colored lineages correspond to lineages where adaptation took place. The squared loadings with PC1 <span class="math inline">\(\rho_{j1}^2\)</span> pick a large proportion of SNPs involved in selection in population <span class="math inline">\(A\)</span> whereas the squared loadings with PC2 <span class="math inline">\(\rho_{j2}^2\)</span> pick SNPs involved in selection in population <span class="math inline">\(B_1\)</span>. This difference is reflected in the different repartition of the top-ranked SNPs for the communality <span class="math inline">\(h^2\)</span> and the statistic <span class="math inline">\({h^{\prime}}^2\)</span>.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:ndf-fig2"></span>
<img src="figure/ndf-fig2.png" alt="Repartition of the 1% top-ranked SNPs of each PCA-based statistic under a divergence model with four types of adaptive constraints. Thicker and colored lineages correspond to lineages where adaptation occurred. The different types of SNPs picked by the squared loadings \(\rho_{j1}^2\) and \(\rho_{j2}^2\) are also found when comparing the communality \(h^2\) and the statistic \({h^{\prime}}^2\)." width="\textwidth" />
<p class="caption">
Figure 3.2: Repartition of the 1% top-ranked SNPs of each PCA-based statistic under a divergence model with four types of adaptive constraints. Thicker and colored lineages correspond to lineages where adaptation occurred. The different types of SNPs picked by the squared loadings <span class="math inline">\(\rho_{j1}^2\)</span> and <span class="math inline">\(\rho_{j2}^2\)</span> are also found when comparing the communality <span class="math inline">\(h^2\)</span> and the statistic <span class="math inline">\({h^{\prime}}^2\)</span>.
</p>
</div>
<p>We investigate the relationship between the communality statistic <span class="math inline">\(h^2\)</span>, which measures the proportion of variance explained by the first two PCs, and the <span class="math inline">\(F_{ST}\)</span> statistic. We find a squared Pearson correlation coefficient between the two statistics larger than 98.8% in the simulations corresponding to figures <a href="section.html#fig:ndf-fig1">3.1</a> and <a href="section.html#fig:ndf-fig2">3.2</a> (supplementary fig. S4, Supplementary Material online). For these two simulations, we look at the SNPs in the top 1% (respectively, 5%) of the ranked lists based on <span class="math inline">\(h^2\)</span> and <span class="math inline">\(F_{ST}\)</span>, and we find an overlap coefficient always larger than 93% for the lists provided by the two different statistics (respectively, 95%). Providing a ranking of the SNPs almost similar to the ranking provided by <span class="math inline">\(F_{ST}\)</span> is therefore possible without considering that individuals originate from predefined populations.</p>
<p>We then compare the performance of the different statistics based on PCA by investigating if the top-ranked SNPs (top 1%) manage to pick SNPs involved in local adaptation (fig. <a href="section.html#fig:ndf-fig1">3.1</a>). The squared loadings <span class="math inline">\(\rho_{j1}^2\)</span> with the first PC pick SNPs involved in selection in population <span class="math inline">\(A\)</span> (39% of the top 1%), a few SNPs involved in selection in <span class="math inline">\(B_1\)</span> (9%), and many false positive SNPs (FDR of 53%). The squared loadings with the second PC <span class="math inline">\(\rho_{j2}^2\)</span> pick less false positives (FDR of 12%) and most SNPs are involved in selection in <span class="math inline">\(B_1\)</span> (88%) with just a few involved in selection in <span class="math inline">\(A\)</span> (1%). When adaptation took place in two different evolutionary lineages of a divergence tree between populations, a genome scan based on PCA has the nice property that outlier loci correlated with PC1 or with PC2 correspond to adaptive constraints that occurred in different parts of the tree.</p>
<p>Because the communality <span class="math inline">\(h^2\)</span> gives more importance to the first PC, it picks preferentially the SNPs that are the most correlated with PC1. There is a large overlap of 72% between the 1% top-ranked lists provided by <span class="math inline">\(h^2\)</span> and <span class="math inline">\(\rho_{j1}^2\)</span>. Therefore, the communality statistic <span class="math inline">\(h^2\)</span> is more sensitive to ancient adaptation events that occurred in the environment of population <span class="math inline">\(A\)</span>. In contrast, the alternative statistic <span class="math inline">\({h^{\prime}}^2\)</span> is more sensitive to recent adaptation events that occurred in the environment of population <span class="math inline">\(B_1\)</span>. When considering the top-ranked 1% of the SNPs, <span class="math inline">\({h^{\prime}}^2\)</span> captures only one SNP involved in selection in <span class="math inline">\(A\)</span> (1% of the top 1%) and 88 SNPs related to adaptation in <span class="math inline">\(B_1\)</span> (88% of the top 1%). The overlap between the 1% top-ranked lists provided by <span class="math inline">\({h^{\prime}}^2\)</span> and by <span class="math inline">\(\rho_{j2}^2\)</span> is of 86%.</p>
<p>The <span class="math inline">\({h^{\prime}}^2\)</span> statistic is mostly influenced by the second principal component because the distribution of squared loadings corresponding to the second PC has a heavier tail, and this result holds for the two divergence models and for the 1000 Genomes data (supplementary fig. S5, Supplementary Material online). To summarize, the <span class="math inline">\(h^2\)</span> and <span class="math inline">\({h^{\prime}}^2\)</span> statistics give too much importance to PC1 and PC2, respectively, and they fail to capture in an equal manner both types of adaptive events occurring in the environment of populations <span class="math inline">\(A\)</span> and <span class="math inline">\(B_1\)</span>.</p>
<p>We also investigate a more complex simulation in which adaptation occurs in the four branches of the divergence tree (fig. <a href="section.html#fig:ndf-fig2">3.2</a>). Among the 10,000 simulated SNPs, we assume that there are four sets of 125 adaptive SNPs with each set being related to adaptation in one of the four branches of the divergence tree. Compared with the simulation of figure <a href="section.html#fig:ndf-fig1">3.1</a>, we find the same pattern of population structure (supplementary fig. S1, Supplementary Material online). The squared loadings <span class="math inline">\(\rho_{j1}^2\)</span> with the first PC mostly pick SNPs involved in selection in the branch that predates the split between <span class="math inline">\(B_1\)</span> and <span class="math inline">\(B_2\)</span> (51% of the top 1%), SNPs involved in selection in the environment of population <span class="math inline">\(A\)</span> (9%), and false positive SNPs (FDR of 38%). Except for false positives (FDR of 14%), the squared loadings <span class="math inline">\(\rho_{j2}^2\)</span> with the second PC rather pick SNPs involved in selection in <span class="math inline">\(B_1\)</span> and <span class="math inline">\(B_2\)</span> (42% for <span class="math inline">\(B_1\)</span> and 44% for <span class="math inline">\(B_2\)</span>). Once again, there is a large overlap between the SNPs picked by the communality <span class="math inline">\(h^2\)</span> and by <span class="math inline">\(\rho_1^2\)</span> (92% of overlap) and between the SNPs picked by <span class="math inline">\({h^{\prime}}^2\)</span> and <span class="math inline">\(\rho_2^2\)</span> (93% of overlap). Because the first PC discriminates population <span class="math inline">\(A\)</span> from <span class="math inline">\(B_1\)</span> and <span class="math inline">\(B_2\)</span> (supplementary fig. S1, Supplementary Material online), the SNPs most correlated with PC1 correspond to SNPs related to adaptation in the (red and green) branches that separate <span class="math inline">\(A\)</span> from populations <span class="math inline">\(B_1\)</span> and <span class="math inline">\(B_2\)</span>. In contrast, the SNPs that are most correlated to PC2 correspond to SNPs related to adaptation in the two (blue and yellow) branches that separate population <span class="math inline">\(B_1\)</span> from <span class="math inline">\(B_2\)</span> (fig. <a href="section.html#fig:ndf-fig2">3.2</a>).</p>
<p>We additionally evaluate to what extent the results are robust with respect to some parameter settings. When considering the 5% of the SNPs with most extreme values of the statistics instead of the top 1%, we also find that the summary statistics pick SNPs related to different evolutionary events (supplementary fig. S6, Supplementary Material online). The main difference being that the FDR increases considerably when considering the top 5% instead of the top 1% (supplementary fig. S6, Supplementary Material online). We also consider variation of the selection coefficient ranging from <span class="math inline">\(s = 1.01\)</span> to <span class="math inline">\(s = 1.1\)</span> (<span class="math inline">\(s = 1.025\)</span> corresponds to the simulations of figs. <a href="section.html#fig:ndf-fig1">3.1</a> and <a href="section.html#fig:ndf-fig2">3.2</a>). As expected, the FDR of the different statistics based on PCA is considerably reduced when the selection coefficient increases (supplementary fig. S7, Supplementary Material online).</p>
<p>In the divergence model of figure <a href="section.html#fig:ndf-fig1">3.1</a>, we also compare the FDRs obtained with the statistics <span class="math inline">\(h^2\)</span>, <span class="math inline">\({h^{\prime}}^2\)</span>, and with a Bayesian factor model implemented in the software <em>PCAdapt</em> <span class="citation">(Duforet-Frebourg et al., <a href="#ref-duforet2014genome">2014</a>)</span>. For the optimal choice of <span class="math inline">\(K = 2\)</span>, the statistic <span class="math inline">\({h^{\prime}}^2\)</span> and the Bayesian factor model provide the smallest FDR (supplementary fig. S8, Supplementary Material online). However, when varying the value of <span class="math inline">\(K\)</span> from <span class="math inline">\(K = 1\)</span> to <span class="math inline">\(K = 6\)</span>, we find that the communality <span class="math inline">\(h^2\)</span> and the Bayesian approach are robust to overspecification of <span class="math inline">\(K\)</span> (<span class="math inline">\(K &gt; 3\)</span>) whereas the FDR obtained with <span class="math inline">\({h^{\prime}}^2\)</span> increases importantly as <span class="math inline">\(K\)</span> increases beyond <span class="math inline">\(K = 2\)</span> (supplementary fig. S8, Supplementary Material online).</p>
<p>We also consider a more general isolation-with-migration model. In the divergence model where adaptation occurs in two different lineages of the population tree (fig. <a href="section.html#fig:ndf-fig1">3.1</a>), we add constant migration between all pairs of populations. We assume that migration occurred after the split between <span class="math inline">\(B_1\)</span> and <span class="math inline">\(B_2\)</span>. We consider different values of migration rates generating a mean <span class="math inline">\(F_{ST}\)</span> of 7.5% for the smallest migration rate to a mean <span class="math inline">\(F_{ST}\)</span> of 0% for the largest migration rate. We find that the <span class="math inline">\(R^2\)</span> correlation between <span class="math inline">\(F_{ST}\)</span> and <span class="math inline">\(h^2\)</span> decreases as a function of the migration rate (supplementary fig. S9, Supplementary Material online). For <span class="math inline">\(F_{ST}\)</span> values larger than 0.5%, <span class="math inline">\(R^2\)</span> is larger than 97%. The squared correlation <span class="math inline">\(R^2\)</span> decreases to 47% for the largest migration rate. Beyond a certain level of migration rate, population structure, as ascertained by principal components, is no more described by well-separated clusters of individuals (supplementary fig. S10, Supplementary Material online) but by a more clinal or continuous pattern (supplementary fig. S10, Supplementary Material online) explaining the difference between <span class="math inline">\(F_{ST}\)</span> and <span class="math inline">\(h^2\)</span>. However, the FDRs obtained with the different statistics based on PCA and with <span class="math inline">\(F_{ST}\)</span> evolve similarly as a function of the migration rate. For both types of approaches, the FDR increases for larger migration with almost no true discovery (only one true discovery in the top 1% lists) when considering the largest migration rate.</p>
<p>The main results obtained under the divergence models can be described as follows. The principal components correspond to different evolutionary lineages of the divergence tree. The communality statistic <span class="math inline">\(h^2\)</span> provides similar list of candidate SNPs than <span class="math inline">\(F_{ST}\)</span> and it is mostly influenced by the first principal component which can be problematic if other PCs also convey adaptive events. To counteract this limitation, which can potentially lead to the loss of important signals of selection, we show that looking at the squared loadings with each of the principal components provide adaptive SNPs that are related to different evolutionary events. When adding migration rates between lineages, we find that the main results are unchanged up to a certain level of migration rate. Above this level of migration rate, the relationship between <span class="math inline">\(F_{ST}\)</span> and <span class="math inline">\(h^2\)</span> does not hold anymore and genome scans based on either PCA or <span class="math inline">\(F_{ST}\)</span> produce a majority of false positives.</p>
</div>
<div id="genomes-data" class="section level4 unnumbered">
<h4>1000 Genomes Data</h4>
<p>Since we are interested in selective pressures that occurred during the human diaspora out of Africa, we decide to exclude individuals whose genetic makeup is the result of recent admixture events (African Americans, Columbians, Puerto Ricans, and Mexicans). The first three principal components capture population structure whereas the following components separate individuals within populations (fig. <a href="section.html#fig:ndf-fig3">3.3</a> and supplementary fig. S11, Supplementary Material online). The first and second PCs ascertain population structure between Africa, Asia, and Europe (fig. <a href="section.html#fig:ndf-fig3">3.3</a>) and the third principal component separates the Yoruba from the Luhya population (supplementary fig. S11, Supplementary Material online). The decay of eigenvalues suggests to use <span class="math inline">\(K = 2\)</span> because the eigenvalues drop between <span class="math inline">\(K = 2\)</span> and <span class="math inline">\(K = 3\)</span> where a plateau of eigenvalues is reached (supplementary fig. S3, Supplementary Material online).</p>

<div class="figure" style="text-align: center"><span id="fig:ndf-fig3"></span>
<img src="figure/ndf-fig3.png" alt="PCA with \(K = 2\) applied to the 1000 Genomes data. The sampled populations are the following: British in England and Scotland (GBR), Utah residents with Northern and Western European ancestry (CEU), Finnish in Finland (FIN), Iberian populations in Spain (IBS), Toscani in Italy (TSI), Han Chinese in Bejing (CHB), Southern Han Chinese (CHS), Japanese in Tokyo (JPT), Luhya in Kenya (LWK), Yoruba in Nigeria (YRI)." width="\textwidth" />
<p class="caption">
Figure 3.3: PCA with <span class="math inline">\(K = 2\)</span> applied to the 1000 Genomes data. The sampled populations are the following: British in England and Scotland (GBR), Utah residents with Northern and Western European ancestry (CEU), Finnish in Finland (FIN), Iberian populations in Spain (IBS), Toscani in Italy (TSI), Han Chinese in Bejing (CHB), Southern Han Chinese (CHS), Japanese in Tokyo (JPT), Luhya in Kenya (LWK), Yoruba in Nigeria (YRI).
</p>
</div>
<p>When performing a genome scan with PCA, there are different choices of statistics. The first choice is the <span class="math inline">\(h^2\)</span> communality statistic. Using the three continents as labels, there is a squared correlation between <span class="math inline">\(h^2\)</span> and <span class="math inline">\(F_{ST}\)</span> of <span class="math inline">\(R^2 = 0.989\)</span>. To investigate if <span class="math inline">\(h^2\)</span> is mostly influenced by the first PC, we determine if the outliers for the <span class="math inline">\(h^2\)</span> statistics are related with PC1 or with PC2. Among the top 0.1% of SNPs with the largest values of <span class="math inline">\(h^2\)</span>, we find that 74% are in the top 0.1% of the squared loadings <span class="math inline">\(\rho_{j1}^2\)</span> corresponding to PC1 and 20% are in the top 0.1% of the squared loadings <span class="math inline">\(\rho_{j2}^2\)</span> corresponding to PC2. The second possible choice of summary statistics is the <span class="math inline">\({h^{\prime}}^2\)</span> statistic. Investigating the repartition of the 0.1% outliers for <span class="math inline">\({h^{\prime}}\)</span>, we find that 0.005% are in the top 0.1% of the squared loadings <span class="math inline">\(\rho_{j1}^2\)</span> corresponding to PC1 and 85% are in the top 0.1% of the squared loadings <span class="math inline">\(\rho_{j2}^2\)</span> corresponding to PC2. The <span class="math inline">\({h^{\prime}}^2\)</span> statistic is mostly influenced by the second PC because the distribution of the <span class="math inline">\(V_{j2}^2\)</span> (normalized squared loadings) has a longer tail than the corresponding distribution for PC1 (supplementary fig. S5, Supplementary Material online). Because the <span class="math inline">\(h^2\)</span> statistic is mostly influenced by PC1 and <span class="math inline">\({h^{\prime}}^2\)</span> is mostly influenced by PC2, confirming the results obtained under the divergence models, we rather decide to perform two separate genome scans based on the squared loadings <span class="math inline">\(\rho_{j1}^2\)</span> and <span class="math inline">\(\rho_{j2}^2\)</span>.</p>
<p>The two Manhattan plots based on the squared loadings for PC1 and PC2 are displayed in figures <a href="section.html#fig:ndf-fig4">3.4</a> and <a href="section.html#fig:ndf-fig5">3.5</a> (supplementary table S1, Supplementary Material online, contains the loadings for all variants). Because of linkage disequilibrium (LD), Manhattan plots generally produce clustered outliers. To investigate if the top 0.1% outliers are clustered in the genome, we count–for various window sizes–the proportion of contiguous windows containing at least one outlier. We find that outlier SNPs correlated with PC1 or with PC2 are more clustered than expected if they would have been uniformly distributed among the 36,536,154 variants (supplementary fig. S12, Supplementary Material online). Additionally, the clustering is larger for the outliers related to the second PC as they cluster in fewer windows (supplementary fig. S12, Supplementary Material online). As the genome scan for PC2 captures more recent adaptive events, it reveals larger genomic windows that experienced fewer recombination events.</p>

<div class="figure" style="text-align: center"><span id="fig:ndf-fig4"></span>
<img src="figure/ndf-fig4.png" alt="Manhattan plot for the 1000 Genomes data of the squared loadings \(\rho_{j1}^2\) with the first principal component. For sake of presentation, only the top-ranked SNPs (top 0.1%) are displayed and the 100 top-ranked SNPs are colored in red." width="\textwidth" />
<p class="caption">
Figure 3.4: Manhattan plot for the 1000 Genomes data of the squared loadings <span class="math inline">\(\rho_{j1}^2\)</span> with the first principal component. For sake of presentation, only the top-ranked SNPs (top 0.1%) are displayed and the 100 top-ranked SNPs are colored in red.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:ndf-fig5"></span>
<img src="figure/ndf-fig5.png" alt="Manhattan plot for the 1000 Genomes data of the squared loadings \(\rho_{j2}^2\) with the second principal component. For sake of presentation, only the top-ranked SNPs (top 0.1%) are displayed and the 100 top-ranked SNPs are colored in red." width="\textwidth" />
<p class="caption">
Figure 3.5: Manhattan plot for the 1000 Genomes data of the squared loadings <span class="math inline">\(\rho_{j2}^2\)</span> with the second principal component. For sake of presentation, only the top-ranked SNPs (top 0.1%) are displayed and the 100 top-ranked SNPs are colored in red.
</p>
</div>
<p>The 1000 Genome data contain many low-frequency SNPs; 82% of the SNPs have a minor allele frequency smaller than 5%. However, these low-frequency variants are not found among outlier SNPs. There are no SNP with a minor allele frequency smaller than 5% among the 0.1% of the SNPs most correlated with PC1 or with PC2.</p>
<p>The 100 SNPs that are the most correlated with the first PC are located in 24 genomic regions (supplementary table S2, Supplementary Material online). Most of the regions contain just one or a few SNPs except a peak in the gene APPBP2 that contains 33 out of the 100 top SNPs, a peak encompassing the RTTN and CD226 genes containing 17 SNPS and a peak in the ATP1A1 gene containing seven SNPs (fig. <a href="section.html#fig:ndf-fig4">3.4</a>). Confirming a larger clustering for PC2 outliers, the 100 SNPs that are the most correlated with PC2 cluster in fewer genomic regions (supplementary table S3, Supplementary Material online). They are located in 14 genomic regions including a region overlapping with EDAR contains 44 top hits, two regions containing eight SNPs and located in the pigmentation genes SLC24A5 and SLC45A2, and two regions with seven top hit SNPs, one in the gene KCNMA1 and another one encompassing the RGLA/MYO5C genes (fig. <a href="section.html#fig:ndf-fig5">3.5</a>).</p>
<p>We perform Gene Ontology (GO) enrichment analyses using Gowinda for the SNPs that are the most correlated with PC1 and PC2. For PC1, we find, among others, enrichment (<span class="math inline">\(\text{FDR} \leq 5 \%\)</span>) for ontologies related to the regulation of arterial blood pressure, the endocrine system and the immunity response (interleukin production, response to viruses) (supplementary table S4, Supplementary Material online). For PC2, we find enrichment (<span class="math inline">\(\text{FDR} \leq 5 \%\)</span>) related to olfactory receptors, keratinocyte and epidermal cell differentiation, and ethanol metabolism (supplementary table S5, Supplementary Material online). We also search for polygenic adaptation by looking for biological pathways enriched with outlier genes <span class="citation">(Daub et al., <a href="#ref-daub2013evidence">2013</a>)</span>. For PC1, we find one enriched (<span class="math inline">\(\text{FDR} \leq 5 \%\)</span>) pathway consisting of the beta defensin pathway (supplementary table S6, Supplementary Material online). The beta defensin pathway contains mainly genes involved in the innate immune system consisting of 36 defensin genes and of two Toll-Like receptors (TLR1 and TLR2). There are additionally two chemokine receptors (CCR2 and CCR6) involved in the beta defensin pathway. For PC2, we also find one enriched pathway consisting of fatty acid omega oxidation (<span class="math inline">\(\text{FDR} \leq 5 \%\)</span>, supplementary table S7, Supplementary Material online). This pathway consists of genes involved in alcohol oxidation (CYP, ALD, and ALDH genes). Performing a less stringent enrichment analysis which can find pathways containing overlapping genes, we find more enriched pathways: the beta defensin and the defensin pathways for PC1 and ethanol oxidation, glycolysis/gluconeogenesis and fatty acid omega oxidation for PC2 (supplementary table S8, Supplementary Material online).</p>
<p>To further validate the proposed list of candidate SNPs involved in local adaptation, we test for an enrichment of genic or nonsynonymous SNP among the SNPs that are the most correlated with the PC. We measure the enrichment among outliers by computing odds ratio <span class="citation">(Fagny et al., <a href="#ref-fagny2014exploring">2014</a>; Kudaravalli, Veyrieras, Stranger, Dermitzakis, &amp; Pritchard, <a href="#ref-kudaravalli2008gene">2008</a>)</span>. For PC1, we do not find significant enrichments (table <a href="section.html#tab:ndf-table1">3.1</a>) except when measuring the enrichment of genic regions compared with nongenic regions (OR = 10.18 for the 100 most correlated SNPs, <span class="math inline">\(P \leq 5 \%\)</span> using a permutation procedure). For PC2, we find an enrichment of genic regions among outliers as well as an enrichment of nonsynonymous SNPs (table <a href="section.html#tab:ndf-table1">3.1</a>). By contrast with the enrichment of genic regions for SNPs extremely correlated with the first PC, the enrichment for the variants extremely correlated with PC2 outliers is significant when using different thresholds to define outliers (table <a href="section.html#tab:ndf-table1">3.1</a>).</p>

<table>
<caption><span id="tab:ndf-table1">Table 3.1: </span>Enrichment Measured with Odds Ratio (OR) of the Variants Most Correlated with the Principal Components Obtained from the 1000 Genomes Data. Enrichment significant at the 1% (resp. 5%) level are indicated with ** (resp. *).</caption>
<thead>
<tr class="header">
<th></th>
<th align="left">Top 0.1%</th>
<th align="left">Top 0.01%</th>
<th align="left">Top 0.005%</th>
<th align="left">Top 100 SNPs</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>pc1–genic/nogenic</td>
<td align="left">1.60*</td>
<td align="left">1.24</td>
<td align="left">1.09</td>
<td align="left">1.93</td>
</tr>
<tr class="even">
<td>pc1–nonsyn/all</td>
<td align="left">1.70</td>
<td align="left">1.18</td>
<td align="left">2.42</td>
<td align="left">10.07*</td>
</tr>
<tr class="odd">
<td>pc1–UTR/all</td>
<td align="left">1.37</td>
<td align="left">0.80</td>
<td align="left">1.65</td>
<td align="left">3.44</td>
</tr>
<tr class="even">
<td>pc2–genic/nogenic</td>
<td align="left">1.51*</td>
<td align="left">2.27</td>
<td align="left">4.73**</td>
<td align="left">4.44*</td>
</tr>
<tr class="odd">
<td>pc2–nonsyn/all</td>
<td align="left">1.72</td>
<td align="left">4.66*</td>
<td align="left">7.40</td>
<td align="left">12.18*</td>
</tr>
<tr class="even">
<td>pc2–UTR/all</td>
<td align="left">1.68</td>
<td align="left">4.01*</td>
<td align="left">3.36</td>
<td align="left">2.73</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="discussion" class="section level3 unnumbered">
<h3>Discussion</h3>
<p>The promise of a fine characterization of natural selection in humans fostered the development of new analytical methods for detecting candidate genomic regions <span class="citation">(Vitti, Grossman, &amp; Sabeti, <a href="#ref-vitti2013detecting">2013</a>)</span>. Population-differentiation based methods such as genome scans based on <span class="math inline">\(F_{ST}\)</span> look for marked differences in allele frequencies between population <span class="citation">(Holsinger &amp; Weir, <a href="#ref-holsinger2009genetics">2009</a>)</span>. Here, we show that the communality statistic <span class="math inline">\(h^2\)</span>, which measures the proportion of variance of a SNP that is explained by the first <span class="math inline">\(K\)</span> principal components, provides a similar list of outliers than the <span class="math inline">\(F_{ST}\)</span> statistic when there are <span class="math inline">\(K + 1\)</span> populations. In addition, the communality statistic <span class="math inline">\(h^2\)</span> based on PCA can be viewed as an extension of <span class="math inline">\(F_{ST}\)</span> because it does not require to define populations in advance and can even be applied in the absence of well-defined populations.</p>
<p>To provide an example of genome scans based on PCA when there are no clusters of populations, we additionally consider the POPRES data consisting of 447,245 SNPSs typed for 1,385 European individuals <span class="citation">(Nelson et al., <a href="#ref-nelson2008population">2008</a>)</span>. The scree plot indicates that there are <span class="math inline">\(K = 2\)</span> relevant clusters (supplementary fig. S3, Supplementary Material online). The first principal component corresponds to a Southeast–Northwest gradient and the second one discriminates individuals from Southern Europe along a East–West gradient <span class="citation">(Jay, Sjödin, Jakobsson, &amp; Blum, <a href="#ref-jay2012anisotropic">2012</a>; Novembre et al., <a href="#ref-novembre2008genes">2008</a>)</span> (fig. <a href="section.html#fig:ndf-fig6">3.6</a>). Considering the 100 SNPs most correlated with the first PC, we find that 75 SNPs are in the lactase region, 18 SNPs are in the HLA region, 5 SNPs are in the ADH1C gene, 1 SNP is in HERC2, and another is close to the LOC283177 gene (fig. <a href="section.html#fig:ndf-fig7">3.7</a>). When considering the 100 SNPs most correlated with the second PC, we find less clustering than for PC1 with more peaks (supplementary fig. S13, Supplementary Material online). The regions that contain the largest number of SNPs in the top 100 SNPs are the HLA region (41 SNPs) and a region close to the NEK10 gene (10 SNPs), which is a gene potentially involved in breast cancer <span class="citation">(Ahmed et al., <a href="#ref-ahmed2009newly">2009</a>)</span>. The genome scan retrieves well-known signals of adaption in humans that are related to lactase persistence (LCT) <span class="citation">(Bersaglieri et al., <a href="#ref-bersaglieri2004genetic">2004</a>)</span>, immunity (HLA), alcohol metabolism (ADH1C) <span class="citation">(Han et al., <a href="#ref-han2007evidence">2007</a>)</span>, and pigmentation (HERC2) <span class="citation">(Wilde et al., <a href="#ref-wilde2014direct">2014</a>)</span>. The analysis of the POPRES data shows that genome scan based on PCA can be applied when there is a clinal or continuous pattern of population structure without well-defined clusters of individuals.</p>

<div class="figure" style="text-align: center"><span id="fig:ndf-fig6"></span>
<img src="figure/ndf-fig6.png" alt="PCA with \(K = 2\) applied to the POPRES data." width="\textwidth" />
<p class="caption">
Figure 3.6: PCA with <span class="math inline">\(K = 2\)</span> applied to the POPRES data.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:ndf-fig7"></span>
<img src="figure/ndf-fig7.png" alt="Manhattan plot for the POPRES data of the squared loadings \(\rho_{j1}^2\) with the first principal component. For sake of presentation, only the top-ranked SNPs (top 5%) are displayed and the 100 top-ranked SNPs are colored in red." width="\textwidth" />
<p class="caption">
Figure 3.7: Manhattan plot for the POPRES data of the squared loadings <span class="math inline">\(\rho_{j1}^2\)</span> with the first principal component. For sake of presentation, only the top-ranked SNPs (top 5%) are displayed and the 100 top-ranked SNPs are colored in red.
</p>
</div>
<p>When there are clusters of populations, we have shown with simulations that genome scans based on <span class="math inline">\(F_{ST}\)</span> can be reproduced with PCA. Genome scans based on PCA have the additional advantage that a particular axis of genetic variation, which is related to adaptation, can be pinpointed. Bearing some similarities with PCA, performing a spectral decomposition of the kinship matrix has been proposed to pinpoint populations where adaptation took place <span class="citation">(Fariello et al., <a href="#ref-fariello2013detecting">2013</a>)</span>. However, despite of some advantages, the statistical problems related to genome scans with <span class="math inline">\(F_{ST}\)</span> remain. The drawbacks of <span class="math inline">\(F_{ST}\)</span> arise when there is hierarchical population structure or range expansion because <span class="math inline">\(F_{ST}\)</span> does not account for correlations of allele frequencies among subpopulations <span class="citation">(Bierne, Roze, &amp; Welch, <a href="#ref-bierne2013pervasive">2013</a>; Lotterhos &amp; Whitlock, <a href="#ref-lotterhos2014evaluation">2014</a>)</span>. An alternative presentation of the issues arising with <span class="math inline">\(F_{ST}\)</span> is that it implicitly assumes either a model of instantaneous divergence between populations or an island-model <span class="citation">(Bonhomme et al., <a href="#ref-bonhomme2010detecting">2010</a>)</span>. Deviations from these models severely impact FDRs <span class="citation">(Duforet-Frebourg et al., <a href="#ref-duforet2014genome">2014</a>)</span>. Viewing <span class="math inline">\(F_{ST}\)</span> from the point of view of PCA provides a new explanation about why <span class="math inline">\(F_{ST}\)</span> does not provide an optimal ranking of SNPs for detecting selection. The statistic <span class="math inline">\(F_{ST}\)</span> or the proposed <span class="math inline">\(h^2\)</span> communality statistic are mostly influenced by the first principal component and the relative importance of the first PC increases with the difference between the first and second eigenvalues of the covariance matrix of the data. Because the first PC can represent ancient adaptive events, especially under population divergence models <span class="citation">(G. McVean, <a href="#ref-mcvean2009genealogical">2009</a>)</span>, it explains why <span class="math inline">\(F_{ST}\)</span> and the communality <span class="math inline">\(h^2\)</span> are biased toward ancient evolutionary events. Following recent developments of <span class="math inline">\(F_{ST}\)</span>-related statistics that account for hierarchical population structure <span class="citation">(Bonhomme et al., <a href="#ref-bonhomme2010detecting">2010</a>; Foll, Gaggiotti, Daub, Vatsiou, &amp; Excoffier, <a href="#ref-foll2014widespread">2014</a>; Günther &amp; Coop, <a href="#ref-gunther2013robust">2013</a>)</span>, we proposed an alternative statistic <span class="math inline">\({h^{\prime}_j}^2\)</span>, which should give equal weights to the different PCs. However, analyzing simulations and the 1000 Genomes data show that <span class="math inline">\({h^{\prime}_j}^2\)</span> do not properly account for hierarchical population structure because outliers identified by <span class="math inline">\({h^{\prime}_j}^2\)</span> are almost always related to the last PC kept in the analysis. To avoid to bias data analysis in favor of one principal component, it is possible to perform a genome scan for each principal component.</p>
<p>In addition to ranking the SNPs when performing a genome scan, a threshold should be chosen to extract a list of outlier SNPs. We do not have addressed the question of how to choose the threshold and rather used empirical threshold such as the 99% quantile of the distribution of the test statistic (top 1%). If interested in controlling the FDR, we can assume that the loadings <span class="math inline">\(\rho_{kj}\)</span> are Gaussian with zero mean <span class="citation">(Galinsky et al., <a href="#ref-galinsky2016fast">2016</a>)</span>. Because of the constraints imposed on the loadings when performing PCA, the variance of the <span class="math inline">\(\rho_{kj}\)</span>’s is equal to the proportion of variance explained by the <span class="math inline">\(k^{th}\)</span> PC, which is given by <span class="math inline">\(\lambda_k / (p \times (n - 1))\)</span> where <span class="math inline">\(\lambda_k\)</span> is the <span class="math inline">\(k^{th}\)</span> eigenvalue of the matrix <span class="math inline">\(YY^T\)</span>. Assuming a Gaussian distribution for the loadings, the communality can then be approximated by a weighted sum of chi-square distribution. Approximating a weighted sum of chi-square distribution with a chi-square distribution, we have <span class="citation">(Yuan &amp; Bentler, <a href="#ref-yuan2010two">2010</a>)</span></p>
<span class="math display" id="eq:chi-square">\[\begin{equation}
  h^2 \times K / c \leadsto \chi^2_K,
  \tag{3.3}
\end{equation}\]</span>
<p>where <span class="math inline">\(c = \sum_{k=1}^K \lambda_k / (p \times (n - 1))\)</span> is the proportion of variance explained by the first <span class="math inline">\(K\)</span> PCs. The chi-square approximation of equation <a href="section.html#eq:chi-square">(3.3)</a> bears similarity with the approximation of <span class="citation">Lewontin &amp; Krakauer (<a href="#ref-lewontin1973distribution">1973</a>)</span> that states that <span class="math inline">\(F_{ST} \times (n_{\text{pops}}-1)/\bar{F}_{ST}\)</span> follows a chi-square approximation with <span class="math inline">\((n_{\text{pops}}-1)\)</span> degrees of freedom where <span class="math inline">\(\bar{F}_{ST}\)</span> is the mean <span class="math inline">\(F_{ST}\)</span> over loci and <span class="math inline">\((n_{\text{pops}}-1)\)</span> is the number of populations. In the simulations of an island model and of a divergence model, quantile-to-quantile plots indicate a good fit to the theoretical chi-square distribution of expression <a href="section.html#eq:chi-square">(3.3)</a> (supplementary fig. S14, Supplementary Material online). When using the chi-square approximation to compute <span class="math inline">\(P\)</span>-values, we evaluate if FDR can be controlled using Benjamini–Hochberg correction (Benjamini and Hochberg 1995). We find that the actual proportion of false discoveries corresponds to the target FDR for the island model but the procedure is too conservative for the divergence model (supplementary fig. S15, Supplementary Material online). For instance, when controlling FDR at a level of 25%, the actual proportion of false discoveries is of 15%. A recent test based on <span class="math inline">\(F_{ST}\)</span> and a chi-square approximation was also found to be conservative <span class="citation">(Whitlock &amp; Lotterhos, <a href="#ref-whitlock2015reliable">2015</a>)</span>. Analysing the phase 1 release of the 1000 Genomes data demonstrates the suitability of a genome scan based on PCA to detect signals of positive selection. We search for variants extremely correlated with the first PC, which corresponds to differentiation between Africa and Eurasia and with the second PC, which corresponds to differentiation between Europe and Asia. For variants most correlated with the second PC, there is a significant enrichment of genic and nonsynonymous SNPs whereas the enrichment is less detectable for variants related to the first PC. The enrichment analysis confirms that positive selection may favor local adaptation of human population by increasing differentiation in genic regions especially in nonsynonymous variants <span class="citation">(Barreiro et al., <a href="#ref-barreiro2008natural">2008</a>)</span>. Consistent with LD, we find that candidate variants are clustered along the genome with a larger clustering for variants correlated with the Europe–Asia axis of differentiation (PC2). The difference of clustering illustrates that statistical methods based on LD for detecting selection will perform differently depending on the time frame under which adaptation had the opportunity to occur <span class="citation">(Sabeti et al., <a href="#ref-sabeti2006positive">2006</a>)</span>. The fact that population divergence, and its concomitant adaptive events, between Europe and Asia is more recent that the out-of-Africa event is a putative explanation of the difference of clustering between PC1 and PC2 outliers. Explaining the difference of enrichment between PC1 and PC2 outliers is more difficult. The weaker enrichment for PC1 outliers can be attributed either to a larger number of false discoveries or to a larger importance of other forms of natural selection such as background selection <span class="citation">(Hernandez et al., <a href="#ref-hernandez2011classic">2011</a>)</span>.</p>
<p>When looking at the 100 SNPs most correlated with PC1 or PC2, we find genes for which selection in humans was already documented (9/24 for PC1 and 5/14 for PC2, supplementary table S9, Supplementary Material online). Known targets for selection include genes involved in pigmentation (MATP, OCA2 for PC1 and SLC45A2, SLC24A5, and MYO5C for PC2), in the regulation of sweating (EDAR for PC2), and in adaptation to pathogens (DARC, SLC39A4, and VAV2 for PC1). A 100 kb region in the vicinity of the APPBPP2 gene contains one-third of the 100 SNPs most correlated with PC1. This APPBPP2 region is a known candidate for selection and has been identified by looking for miRNA binding sites with extreme population differentiation <span class="citation">(J. Li et al., <a href="#ref-li2012evidence">2012</a>)</span>. APPBPP2 is a nervous system gene that has been associated with Alzheimer disease, and it may have experienced a selective sweep <span class="citation">(Williamson et al., <a href="#ref-williamson2007localizing">2007</a>)</span>. For some SNPs in APPBPP2, the differences of allele frequencies between Eurasiatic population and sub-Saharan populations from Africa are of the order of 90% (<a href="http://popgen.uchicago.edu/ggv/" class="uri">http://popgen.uchicago.edu/ggv/</a>, last accessed December 2015) calling for a further functional analysis. Moreover, looking at the 100 SNPs most correlated with PC1 and PC2 confirms the importance of noncoding RNA (FAM230B, D21S2088E, LOC100133461, LINC00290, LINC01347, LINC00681), such as miRNA (MIR429), as a substrate for human adaptation <span class="citation">(Grossman et al., <a href="#ref-grossman2013identifying">2013</a>; J. Li et al., <a href="#ref-li2012evidence">2012</a>)</span>. Among the other regions with a large number of candidate SNPs, we also found the RTTN/CD226 regions, which contain many SNPs correlated with PC1. In different selection scans, the RTTN genes has been detected <span class="citation">(Barreiro et al., <a href="#ref-barreiro2008natural">2008</a>; Carlson et al., <a href="#ref-carlson2005genomic">2005</a>)</span>, and it is involved in the development of the human skeletal system <span class="citation">(Wu &amp; Zhang, <a href="#ref-wu2010positive">2010</a>)</span>. An other region with many SNPs correlated with PC1 contains the ATP1A1 gene involved in osmoregulation and associated with hypertension <span class="citation">(Gurdasani et al., <a href="#ref-gurdasani2015african">2015</a>)</span>. The regions containing the largest number of SNPs correlated with PC2 are well-documented instances of adaptation in humans and includes the EDAR, SLC24A5, and SLC45A2 genes. The KCNMA1 gene contains seven SNPs correlated with PC2 and is involved in breast cancer and obesity <span class="citation">(Jiao et al., <a href="#ref-jiao2011genome">2011</a>; Oeggerli et al., <a href="#ref-oeggerli2012role">2012</a>)</span>. As for KCNMA1, the MYO5C has already been reported in selection scans although no mechanism of biological adaption has been proposed yet <span class="citation">(H. Chen, Patterson, &amp; Reich, <a href="#ref-chen2010population">2010</a>; Fumagalli et al., <a href="#ref-fumagalli2010genome">2010</a>)</span>. To summarize, the list of most correlated SNPs with the PCs identifies well-known genes related to biological adaptation in humans (EDAR, SLC24A5, SLC45A2, DARC), but also provides candidate genes that deserve further studies such as the APPBPP2, TP1A1, RTTN, KCNMA1, and MYO5C genes, as well as the ncRNAs listed above.</p>
<p>We also show that a scan based on PCA can also be used to detect more subtle footprints of positive selection. We conduct an enrichment analysis that detects polygenic adaptation at the level of biological pathways <span class="citation">(Daub et al., <a href="#ref-daub2013evidence">2013</a>)</span>. We find that genes in the beta-defensin pathway are enriched in SNPs correlated with PC1. The beta-defensin genes are key components of the innate immune system and have evolved through positive selection in the catarrhine primate lineages <span class="citation">(Hollox &amp; Armour, <a href="#ref-hollox2008directional">2008</a>)</span>. As for the HLA complex, some beta-defensin genes (DEFB1, DEFB127) show evidence of long-term balancing selection with major haplotypic clades coexisting since millions of years <span class="citation">(Cagliani et al., <a href="#ref-cagliani2008signature">2008</a>; Hollox &amp; Armour, <a href="#ref-hollox2008directional">2008</a>)</span>. We also find that genes in the omega fatty acid oxidation pathways are enriched in SNPs correlated with PC2. This pathway was also found when investigating polygenic adaptation to altitude in humans <span class="citation">(Foll et al., <a href="#ref-foll2014widespread">2014</a>)</span>. The proposed explanation was that omega oxidation becomes a more important metabolic pathway when beta oxidation is defective, which can occur in case of hypoxia <span class="citation">(Foll et al., <a href="#ref-foll2014widespread">2014</a>)</span>. However, this explanation is not valid in the context of the 1000 Genomes data when there are no populations living in hypoxic environments. Proposing phenotypes on which selection operates is complicated by the fact that the omega fatty acid oxidation pathway strongly overlaps with two other pathways: ethanol oxidation and glycolysis. Evidence of selection on the alcohol dehydrogenase locus have already been provided <span class="citation">(Han et al., <a href="#ref-han2007evidence">2007</a>)</span> with some authors proposing that a lower risk for alcoholism might have been beneficial after rice domestication in Asia <span class="citation">(Y. Peng et al., <a href="#ref-peng2010adh1b">2010</a>)</span>. This hypothesis is speculative and we lack a confirmed biological mechanism explaining the enrichment of the fatty acid oxidation pathway. More generally, the enrichment of the beta-defensin and of the omega fatty acid oxidation pathways confirms the importance of pathogenic pressure and of metabolism in human adaptation to different environments <span class="citation">(Barreiro &amp; Quintana-Murci, <a href="#ref-barreiro2010evolutionary">2010</a>; Daub et al., <a href="#ref-daub2013evidence">2013</a>; Fumagalli et al., <a href="#ref-fumagalli2011signatures">2011</a>; Hancock et al., <a href="#ref-hancock2008adaptations">2008</a>)</span>.</p>
<p>In conclusion, we propose a new approach to scan genomes for local adaptation that works with individual genotype data. Because the method is efficiently implemented in the software PCAdapt fast, analyzing 36,536,154 SNPs took only 502 min using a single core of an Intel(R) Xeon(R) (E5-2650, 2.00GHz, 64 bits). Even with low-coverage sequence data (3x), PCA-based statistics retrieve well-known examples of biological adaptation which is encouraging for future whole-genome sequencing project, especially for nonmodel species, aiming at sampling many individuals with limited cost.</p>
</div>
<div id="materials-and-methods" class="section level3 unnumbered">
<h3>Materials and Methods</h3>
<div id="simulations-of-an-island-model" class="section level4 unnumbered">
<h4>Simulations of an Island Model</h4>
<p>Simulations were performed with <em>ms</em> <span class="citation">(Hudson, <a href="#ref-hudson2002generating">2002</a>)</span>. We assume that there are three islands with 100 sampled individuals in each of them. There is a total of 1,400 neutral SNPs, and 100 adaptive SNPs. SNPs are assumed to be unlinked. To mimic adaptation, we consider that adaptive SNP have a migration rate smaller than the migration rate of neutral SNPs (<span class="math inline">\(4N_0m = 4\)</span> for neutral SNPs) (Bazin et al. 2010). The strength of selection is equal to the ratio of the migration rates of neutral and adaptive SNPs. Adaptation is assumed to occur in one population only. The <em>ms</em> command lines for neutral and adaptive SNPs are given below (assuming an effective migration rate of <span class="math inline">\(4N_0m = 0.1\)</span> for adaptive SNPs).</p>
<p>The values of migrations rates we consider for adaptive SNPs are <span class="math inline">\(4N_0m = 0.04, 0.1, 0.4, 1, 2\)</span>.</p>
</div>
<div id="simulations-of-divergence-models" class="section level4 unnumbered">
<h4>Simulations of Divergence Models</h4>
<p>We assume that each population has a constant effective population size of <span class="math inline">\(N_0 = 1000\)</span> diploid individuals, with 50 individuals sampled in each population. The genotypes consist of 10,000 independent SNPs. The simulations were performed in two steps. In the first step, we used the software <em>ms</em> to simulate genetic diversity <span class="citation">(Hudson, <a href="#ref-hudson2002generating">2002</a>)</span> in the ancestral population. We kept only variants with a minor allele frequency larger than 5% at the end of the first step. The second step was performed with <em>simuPOP</em> <span class="citation">(B. Peng &amp; Kimmel, <a href="#ref-peng2005simupop">2005</a>)</span> and simulations were started using the allele frequencies generated with <em>ms</em> in the ancestral population. Looking forward in time, we consider that there are 100 generations between the initial split and the following split between the two <span class="math inline">\(B\)</span> subpopulations, and 200 generations following the split between the two <span class="math inline">\(B\)</span> subpopulations. We assume no migration between populations. In the simulation of figure <a href="section.html#fig:ndf-fig1">3.1</a>, we assume that 250 SNPs confer a selective advantage in the branch leading to population <span class="math inline">\(A\)</span> and 250 other SNPs confer a selective advantage in the branch leading to population <span class="math inline">\(B_1\)</span>. We consider an additive model for selection with a selection coefficient of <span class="math inline">\(s = 1.025\)</span> for heterozygotes. For the simulation of figure <a href="section.html#fig:ndf-fig2">3.2</a>, we assume that there are four nonoverlapping sets of 125 adaptive SNPs with each set being related to adaptation in one of the four branches of the divergence tree. A SNP can confer a selective advantage in a single branch only.</p>
<p>When including migration, we consider that there are 200 generations between the initial split and the following split between the two <span class="math inline">\(B\)</span> subpopulations, and 100 generations following the split between the two <span class="math inline">\(B\)</span> subpopulations. We consider migration rates ranging from 0.2% to 5% per generation. Migration is assumed to occur only after the split between <span class="math inline">\(B_1\)</span> and <span class="math inline">\(B_2\)</span>. The migration rate is the same for the three pairs of populations. To estimate the <span class="math inline">\(F_{ST}\)</span> statistic, we consider the estimator of <span class="citation">Weir &amp; Cockerham (<a href="#ref-weir1984estimating">1984</a>)</span>.</p>
</div>
<div id="genomes-data-1" class="section level4 unnumbered">
<h4>1000 Genomes Data</h4>
<p>We downloaded the 1000 Genomes data (phase 1 v3) <span class="citation">(Consortium &amp; others, <a href="#ref-10002012integrated">2012</a>)</span>. We kept low-coverage genome data and excluded exomes and triome data to minimize variation in read depth. Filtering the data resulted in a total of 36,536,154 SNPs that have been typed on 1,092 individuals. Because the analysis focuses on biological adaptation that took place during the human diaspora out of Africa, we removed recently admixed populations (Mexican, Columbian, PortoRican, and AfroAmerican individuals from the Southwest of the United States). The resulting data set contains 850 individuals coming from Asia (two Han Chinese and one Japanese populations), Africa (Yoruba and Luhya), and Europe (Finish, British in England and Scotland, Iberian, Toscan, and Utah residents with Northern and Western European ancestry).</p>
</div>
<div id="enrichment-analyses" class="section level4 unnumbered">
<h4>Enrichment Analyses</h4>
<p>We used Gowinda <span class="citation">(Kofler &amp; Schlötterer, <a href="#ref-kofler2012gowinda">2012</a>)</span> to test for enrichment of GO. A gene is considered as a candidate if there is at least one of the most correlated SNPs (top 1%) that is mapped to the gene (within an interval of 50 kb upstream and downstream of the gene). Enrichment was computed as the proportion of genes containing at least one outlier SNPs among the genes of the given GO category that are present in the data set. In order to sample a null distribution for enrichment, <em>Gowinda</em> performs resampling without replacement of the SNPs. We used the <code>-gene</code> option of <em>Gowinda</em> that assumes complete linkage within genes.</p>
<p>We performed a second enrichment analysis to determine if outlier SNPs are enriched for genic regions. We computed odds ratio <span class="citation">(Kudaravalli et al., <a href="#ref-kudaravalli2008gene">2008</a>)</span></p>
<span class="math display" id="eq:odds-ratio">\[\begin{equation}
  \text{OR} = \frac{\text{Pr(genic|outlier)}}{\text{Pr(not genic|outlier)}} \frac{\text{Pr(not genic|not outlier)}}{\text{Pr(genic|not outlier)}}
  \tag{3.4}
\end{equation}\]</span>
<p>We implemented a permutation procedure to test if an odds ratio is significantly larger than 1 <span class="citation">(Fagny et al., <a href="#ref-fagny2014exploring">2014</a>)</span>. The same procedure was applied when testing for enrichment of UTR regions (untranslated regions) and of nonsynonymous SNPs.</p>
</div>
<div id="polygenic-adaptation" class="section level4 unnumbered">
<h4>Polygenic Adaptation</h4>
<p>To test for polygenic adaptation, we determined whether genes in a given biological pathway show a shift in the distribution of the loadings <span class="citation">(Daub et al., <a href="#ref-daub2013evidence">2013</a>)</span>. We computed the SUMSTAT statistic for testing if there is an excess of selection signal in each pathway <span class="citation">(Daub et al., <a href="#ref-daub2013evidence">2013</a>)</span>. We applied the same pruning method to take into account redundancy of genes within pathways. The test statistic is the squared loading standardized into a <span class="math inline">\(z\)</span>-score <span class="citation">(Daub et al., <a href="#ref-daub2013evidence">2013</a>)</span>. SUMSTAT is computed for each gene as the sum of test statistic of each SNP belonging to the gene. Intergenic SNPs are assigned to a gene provided they are situated 50kb up- or downstream. We downloaded 63,693 known genes from the UCSC website and we mapped SNPs to a gene if a SNP is located within a gene transcript or within 50kb of a gene. A total of 18,267 genes were mapped with this approach. We downloaded 2,681 gene sets from the NCBI Biosystems database. After discarding genes that were not part of the aforementioned gene list, removing gene sets with less than 10 genes and pooling nearly identical gene sets, we kept 1,532 sets for which we test if there was a shift of the distribution of loadings.</p>
</div>
</div>
<div id="acknowledgments" class="section level3 unnumbered">
<h3>Acknowledgments</h3>
<p>This work has been supported by the LabEx PERSYVAL-Lab (ANR-11-LABX-0025-01) and the ANR AGRHUM project (ANR-14-CE02-0003-01). POPRES data were obtained from dbGaP (accession number phs000145.v1.p1).</p>

</div>
</div>
<div id="article-2" class="section level2">
<h2><span class="header-section-number">3.2</span> Article 2</h2>
<p><img src="figure/molecol.png" width="\textwidth" style="display: block; margin: auto;" /></p>
<div id="abstract-1" class="section level3 unnumbered">
<h3>Abstract</h3>
<p>The R package <em>pcadapt</em> performs genome scans to detect genes under selection based on population genomic data. It assumes that candidate markers are outliers with respect to how they are related to population structure. Because population structure is ascertained with principal component analysis, the package is fast and works with large-scale data. It can handle missing data and pooled sequencing data. By contrast to population-based approaches, the package handle admixed individuals and does not require grouping individuals into populations. Since its first release, <em>pcadapt</em> has evolved in terms of both statistical approach and software implementation. We present results obtained with robust Mahalanobis distance, which is a new statistic for genome scans available in the 2.0 and later versions of the package. When hierarchical population structure occurs, Mahalanobis distance is more powerful than the communality statistic that was implemented in the first version of the package. Using simulated data, we compare <em>pcadapt</em> to other computer programs for genome scans (<em>BayeScan</em>, <em>hapflk</em>, <em>OutFLANK</em>, <em>sNMF</em>). We find that the proportion of false discoveries is around a nominal false discovery rate set at 10% with the exception of <em>BayeScan</em> that generates 40% of false discoveries. We also find that the power of <em>BayeScan</em> is severely impacted by the presence of admixed individuals whereas <em>pcadapt</em> is not impacted. Last, we find that <em>pcadapt</em> and <em>hapflk</em> are the most powerful in scenarios of population divergence and range expansion. Because <em>pcadapt</em> handles next-generation sequencing data, it is a valuable tool for data analysis in molecular ecology.</p>
</div>
<div id="introduction-2" class="section level3 unnumbered">
<h3>Introduction</h3>
<p>Looking for variants with unexpectedly large differences of allele frequencies between populations is a common approach to detect signals of natural selection <span class="citation">(Lewontin &amp; Krakauer, <a href="#ref-lewontin1973distribution">1973</a>)</span>. When variants confer a selective advantage in the local environment, allele frequency changes are triggered by natural selection leading to unexpectedly large differences of allele frequencies between populations. To detect variants with large differences of allele frequencies, numerous test statistics have been proposed, which are usually based on chi-square approximations of <span class="math inline">\(F_{ST}\)</span>-related test statistics <span class="citation">(François, Martins, Caye, &amp; Schoville, <a href="#ref-franccois2016controlling">2016</a>)</span>.</p>
<p>Statistical approaches for detecting selection should address several challenges. The first challenge is to account for hierarchical population structure that arises when genetic differentiation between populations is not identical between all pairs of populations. Statistical tests based on <span class="math inline">\(F_{ST}\)</span> that do not account for hierarchical structure, when it occurs, generate a large excess of false-positive loci <span class="citation">(Bierne et al., <a href="#ref-bierne2013pervasive">2013</a>; Excoffier, Hofer, &amp; Foll, <a href="#ref-excoffier2009detecting">2009</a>)</span>.</p>
<p>A second challenge arises because approaches based on <span class="math inline">\(F_{ST}\)</span>-related measures require to group individuals into populations, although defining populations is a difficult task <span class="citation">(Waples &amp; Gaggiotti, <a href="#ref-waples2006invited">2006</a>)</span>. Individual sampling may not be population based but based on more continuous sampling schemes <span class="citation">(Lotterhos &amp; Whitlock, <a href="#ref-lotterhos2015relative">2015</a>)</span>. Additionally assigning an admixed individual to a single population involves some arbitrariness because different regions of its genome might come from different populations <span class="citation">(Pritchard, Stephens, &amp; Donnelly, <a href="#ref-pritchard2000inference">2000</a>)</span>. Several individual-based methods of genome scans have already been proposed to address this challenge and they are based on related techniques of multivariate analysis including principal component analysis (PCA), factor models and non-negative matrix factorization <span class="citation">(G. Chen et al., <a href="#ref-chen2016eigengwas">2016</a>; Duforet-Frebourg et al., <a href="#ref-duforet2014genome">2014</a>, <a href="#ref-duforet2015detecting">2015</a>; Galinsky et al., <a href="#ref-galinsky2016fast">2016</a>; Hao, Song, &amp; Storey, <a href="#ref-hao2015probabilistic">2015</a>; Martins, Caye, Luu, Blum, &amp; Francois, <a href="#ref-martins2016identifying">2016</a>)</span>.</p>
<p>The last challenge arises from the nature of multilocus data sets generated from next-generation sequencing platforms. Because data sets are massive with a large number of molecular markers, Monte Carlo methods usually implemented in Bayesian statistics may be prohibitively slow <span class="citation">(Lange, Papp, Sinsheimer, &amp; Sobel, <a href="#ref-lange2014next">2014</a>)</span>. Additionally, next-generation sequencing data may contain a substantial proportion of missing data that should be accounted for <span class="citation">(B. Arnold, Corbett-Detig, Hartl, &amp; Bomblies, <a href="#ref-arnold2013radseq">2013</a>; Gautier et al., <a href="#ref-gautier2013effect">2013</a>)</span>.</p>
<p>To address the aforementioned challenges, we have developed the computer program <em>pcadapt</em> and the R package <em>pcadapt</em>. The computer program <em>pcadapt</em> is now deprecated and the R package only is maintained. <em>pcadapt</em> assumes that markers excessively related to population structure are candidates for local adaptation. Since its first release, <em>pcadapt</em> has substantially evolved in terms of both statistical approach and implementation (Table <a href="section.html#tab:table1">3.2</a>).</p>

<table>
<caption><span id="tab:table1">Table 3.2: </span>Summary of the different statistical methods and implementations of <em>pcadapt</em>. Pop. structure stands for population structure and dist. stands for distance</caption>
<thead>
<tr class="header">
<th align="left">Test statistic</th>
<th align="left">Pop. structure</th>
<th align="left">Language</th>
<th align="left">Command line</th>
<th align="left">Versions of the R package</th>
<th align="left">References</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Bayes factor</td>
<td align="left">Factor model</td>
<td align="left">C</td>
<td align="left">PCAdapt</td>
<td align="left">NA</td>
<td align="left">Duforet-Frebourg et al. (2014)</td>
</tr>
<tr class="even">
<td align="left">Communality</td>
<td align="left">PCA</td>
<td align="left">C and R</td>
<td align="left">PCAdapt fast</td>
<td align="left">1. x</td>
<td align="left">Duforet-Frebourg et al. (2016)</td>
</tr>
<tr class="odd">
<td align="left">Mahalanobis dist.</td>
<td align="left">PCA</td>
<td align="left">R</td>
<td align="left">NA</td>
<td align="left">2. x and 3. x</td>
<td align="left">This study</td>
</tr>
</tbody>
</table>
<p>The first release of <em>pcadapt</em> was a command line computer program written in C. It implemented a Monte Carlo approach based on a Bayesian factor model <span class="citation">(Duforet-Frebourg et al., <a href="#ref-duforet2014genome">2014</a>)</span>. The test statistic for outlier detection was a Bayes factor. Because Monte Carlo methods can be computationally prohibitive with massive NGS data, we then developed an alternative approach based on PCA. The first statistic based on PCA was the communality statistic, which measures the percentage of variation of a single-nucleotide polymorphism (SNP) explained by the first <span class="math inline">\(K\)</span> principal components <span class="citation">(Duforet-Frebourg et al., <a href="#ref-duforet2015detecting">2015</a>)</span>. It was initially implemented with a command line computer program (the <em>pcadapt fast</em> command) before being implemented in the <em>pcadapt</em> R package. We do not maintain C versions of <em>pcadapt</em> anymore. The whole analysis that goes from reading genotype files to detecting outlier SNPs can now be performed in R <span class="citation">(Team, <a href="#ref-team2015r">2015</a>)</span>.</p>
<p>The 2.0 and following versions of the R package implement a more powerful statistic for genome scans. The test statistic is a robust Mahalanobis distance. A vector containing <span class="math inline">\(K\)</span> <span class="math inline">\(z\)</span>-scores measures to what extent a SNP is related to the first <span class="math inline">\(K\)</span> principal components. The Mahalanobis distance is then computed for each SNP to detect outliers for which the vector of <span class="math inline">\(z\)</span>-scores does not follow the distribution of the main bulk of points. The term robust refers to the fact that the estimators of the mean and of the covariance matrix of <span class="math inline">\(z\)</span>, which are required to compute the Mahalanobis distances, are not sensitive to the presence of outliers in the data set <span class="citation">(Maronna &amp; Zamar, <a href="#ref-maronna2002robust">2002</a>)</span>. In the following, we provide a comparison of statistical power that shows that Mahalanobis distance provides more powerful genome scans compared with the communality statistic and with the Bayes factor that were implemented in previous versions of <em>pcadapt</em>.</p>
<p>In addition to comparing the different test statistics that were implemented in <em>pcadapt</em>, we compare statistic performance obtained with the 3.0 version of <em>pcadapt</em> and with other computer programs for genome scans. We use simulated data to compare computer programs in terms of false discovery rate (FDR) and statistical power. We consider data simulated under different demographic models including island model, divergence model and range expansion. To perform comparisons, we include programs that require to group individuals into populations: <em>BayeScan</em> <span class="citation">(Foll &amp; Gaggiotti, <a href="#ref-foll2008genome">2008</a>)</span>, the <span class="math inline">\(F_{LK}\)</span> statistic as implemented in the <em>hapflk</em> computer program <span class="citation">(Bonhomme et al., <a href="#ref-bonhomme2010detecting">2010</a>)</span>, and <em>OutFLANK</em> that provides a robust estimation of the null distribution of a <span class="math inline">\(F_{ST}\)</span> test statistic <span class="citation">(Whitlock &amp; Lotterhos, <a href="#ref-whitlock2015reliable">2015</a>)</span>. We additionally consider the <em>sNMF</em> computer program that implements another individual-based test statistic for genome scans <span class="citation">(Frichot, Mathieu, Trouillon, Bouchard, &amp; François, <a href="#ref-frichot2014fast">2014</a>; Martins et al., <a href="#ref-martins2016identifying">2016</a>)</span>.</p>
</div>
<div id="statistical-and-computational-approach" class="section level3 unnumbered">
<h3>Statistical and computational approach</h3>
<div id="input-data" class="section level4 unnumbered">
<h4>Input data</h4>
<p>The R package can handle different data formats for the genotype data matrix. In the version 3.0 that is currently available on CRAN, the package can handle genotype data files in the <em>vcf</em>, <em>ped</em> and <em>lfmm</em> formats. In addition, the package can also handle a <em>pcadapt</em> format, which is a text file where each line contains the allele counts of all individuals at a given locus. When reading a genotype data matrix with the <em>read.pcadapt</em> function, a <em>.pcadapt</em> file is generated, which contains the genotype data in the <em>pcadapt</em> format.</p>
</div>
<div id="choosing-the-number-of-principal-components" class="section level4 unnumbered">
<h4>Choosing the number of principal components</h4>
<p>In the following, we denote by <span class="math inline">\(n\)</span> the number of individuals, by <span class="math inline">\(p\)</span> the number of genetic markers and by <span class="math inline">\(G\)</span> the genotype matrix that is composed of <span class="math inline">\(n\)</span> lines and <span class="math inline">\(p\)</span> columns. The genotypic information at locus <span class="math inline">\(j\)</span> for individual <span class="math inline">\(i\)</span> is encoded by the allele count <span class="math inline">\(G_{ij}\)</span>, <span class="math inline">\(1 \leq i \leq n\)</span> and <span class="math inline">\(1 \leq j \leq p\)</span>, which is a value in 0,1 for haploid species and in 0,1,2 for diploid species. The current 3.0.2 version of the package can handle haploid and diploid data only.</p>
<p>First, we normalize the genotype matrix columnwise. For diploid data, we consider the usual normalization in population genomics where <span class="math inline">\(\tilde{G}_{ij} = (G_{ij} - p_j)/(2 \times p_j(1 - p_j))^{1/2}\)</span>, and <span class="math inline">\(p_j\)</span> denotes the minor allele frequency for locus <span class="math inline">\(j\)</span> <span class="citation">(N. Patterson et al., <a href="#ref-patterson2006population">2006</a>)</span>. The normalization for haploid data is similar except that the denominator is given by <span class="math inline">\((p_j(1 - p_j))^{1/2}\)</span></p>
<p>Then, we use the normalized genotype matrix math formula to ascertain population structure with PCA <span class="citation">(N. Patterson et al., <a href="#ref-patterson2006population">2006</a>)</span>. The number of principal components to consider is denoted <span class="math inline">\(K\)</span> and is a parameter that should be chosen by the user. In order to choose <span class="math inline">\(K\)</span>, we recommend to consider the graphical approach based on the scree plot <span class="citation">(Jackson, <a href="#ref-jackson1993stopping">1993</a>)</span>. The scree plot displays the eigenvalues of the covariance matrix <span class="math inline">\(\Omega\)</span> in descending order. Up to a constant, eigenvalues are proportional to the proportion of variance explained by each principal component. The eigenvalues that correspond to random variation lie on a straight line whereas the ones corresponding to population structure depart from the line. We recommend to use Cattell’s rule that states that components corresponding to eigenvalues to the left of the straight line should be kept <span class="citation">(Cattell, <a href="#ref-cattell1966scree">1966</a>)</span>.</p>
</div>
<div id="test-statistic" class="section level4 unnumbered">
<h4>Test statistic</h4>
<p>We now detail how the package computes the test statistic. We consider multiple linear regressions by regressing each of the <span class="math inline">\(p\)</span> SNPs by the <span class="math inline">\(K\)</span> principal components <span class="math inline">\(X_1, \dots, X_K\)</span></p>
<span class="math display" id="eq:multiple-reg">\[\begin{equation}
  G_j = \sum_{k=1}^K \beta_{jk} X_k + \epsilon_j, \; j = 1, \dots, p,
  \tag{3.5}
\end{equation}\]</span>
<p>where <span class="math inline">\(\beta_{jk}\)</span> is the regression coefficient corresponding to the <span class="math inline">\(j\)</span>-th SNP regressed by the <span class="math inline">\(k\)</span>-th principal component, and <span class="math inline">\(\epsilon_j\)</span> is the residuals vector. To summarize the result of the regression analysis for the <span class="math inline">\(j\)</span>-th SNP, we return a vector of <span class="math inline">\(z\)</span>-scores <span class="math inline">\(z_j = (z_{j1}, \dots, z_{jK})\)</span> where <span class="math inline">\(z_{jk}\)</span> corresponds to the <span class="math inline">\(z\)</span>-score obtained when regressing the <span class="math inline">\(j\)</span>-th SNP by the <span class="math inline">\(k\)</span>-th principal component.</p>
<p>The next step is to look for outliers based on the vector of <span class="math inline">\(z\)</span>-scores. We consider a classical approach in multivariate analysis for outlier detection. The test statistic is a robust Mahalanobis distance <span class="math inline">\(D\)</span> defined as</p>
<span class="math display" id="eq:maha-def">\[\begin{equation}
  D_j^2 = (z_j - \bar{z})^T \Sigma^{-1} (z_j - \bar{z}),
  \tag{3.6}
\end{equation}\]</span>
<p>where <span class="math inline">\(\Sigma\)</span> is the <span class="math inline">\((K \times K)\)</span> covariance matrix of the <span class="math inline">\(z\)</span>-scores and <span class="math inline">\(\bar{z}\)</span> is the vector of the <span class="math inline">\(K\)</span> <span class="math inline">\(z\)</span>-score means <span class="citation">(Maronna &amp; Zamar, <a href="#ref-maronna2002robust">2002</a>)</span>. When <span class="math inline">\(K &gt; 1\)</span>, the covariance matrix <span class="math inline">\(\Sigma\)</span> is estimated with the orthogonalized Gnanadesikan–Kettenring method that is a robust estimate of the covariance able to handle large-scale data <span class="citation">(Maronna &amp; Zamar, <a href="#ref-maronna2002robust">2002</a>)</span> (<em>covRob</em> function of the <em>robust</em> R package). When <span class="math inline">\(K = 1\)</span>, the variance is estimated with another robust estimate (<em>cov.rob</em> function of the <em>MASS</em> R package).</p>
</div>
<div id="genomic-inflation-factor" class="section level4 unnumbered">
<h4>Genomic inflation factor</h4>
<p>To perform multiple hypothesis testing, Mahalanobis distances should be transformed into <span class="math inline">\(P\)</span>-values. If the <span class="math inline">\(z\)</span>-scores were truly multivariate Gaussian, the Mahalanobis distances <span class="math inline">\(D\)</span> should be chi-square distributed with <span class="math inline">\(K\)</span> degrees of freedom. However, as usual for genome scans, there are confounding factors that inflate values of the test statistic and that would lead to an excess of false positives <span class="citation">(François et al., <a href="#ref-franccois2016controlling">2016</a>)</span>. To account for the inflation of test statistics, we divide Mahalanobis distances by a constant <span class="math inline">\(\lambda\)</span> to obtain a statistic that can be approximated by a chi-square distribution with <span class="math inline">\(K\)</span> degrees of freedom. This constant is estimated by the genomic inflation factor defined here as the median of the Mahalanobis distances divided by the median of the chi-square distribution with <span class="math inline">\(K\)</span> degrees of freedom <span class="citation">(Devlin &amp; Roeder, <a href="#ref-devlin1999genomic">1999</a>)</span>.</p>
</div>
<div id="control-of-the-false-discovery-rate-fdr" class="section level4 unnumbered">
<h4>Control of the false discovery rate (FDR)</h4>
<p>Once <span class="math inline">\(P\)</span>-values are computed, there is a problem of decision-making related to the choice of a threshold for <span class="math inline">\(P\)</span>-values. We recommend to use the FDR approach where the objective is to provide a list of candidate genes with an expected proportion of false discoveries smaller than a specified value. For controlling the FDR, we consider the <span class="math inline">\(q\)</span>-value procedure as implemented in the <em>qvalue</em> R package that is less conservative than Bonferroni or Benjamini–Hochberg correction <span class="citation">(Storey &amp; Tibshirani, <a href="#ref-storey2003statistical">2003</a>)</span>. The <em>qvalue</em> R package transforms the <span class="math inline">\(P\)</span>-values into <span class="math inline">\(q\)</span>-values and the user can control a specified value <span class="math inline">\(\alpha\)</span> of FDR by considering as candidates the SNPs with <span class="math inline">\(q\)</span>-values smaller than <span class="math inline">\(\alpha\)</span>.</p>
</div>
<div id="numerical-computations-1" class="section level4 unnumbered">
<h4>Numerical computations</h4>
<p>PCA is performed using a C routine that allows to compute scores and eigenvalues efficiently with minimum RAM access <span class="citation">(Duforet-Frebourg et al., <a href="#ref-duforet2015detecting">2015</a>)</span>. Computing the covariance matrix <span class="math inline">\(\Omega\)</span> is the most computationally demanding part. To provide a fast routine, we compute the <span class="math inline">\(n \times n\)</span> covariance matrix <span class="math inline">\(\Omega\)</span> instead of the much larger <span class="math inline">\(p \times p\)</span> covariance matrix. We compute the covariance <span class="math inline">\(\Omega\)</span> incrementally by adding small storable covariance blocks successively. Multiple linear regression is then solved directly by computing an explicit solution, written as a matrix product. Using the fact that the <span class="math inline">\((n,K)\)</span> score matrix <span class="math inline">\(X\)</span> is orthogonal, the <span class="math inline">\((p,K)\)</span> matrix <span class="math inline">\(\beta\)</span> of regression coefficients is given by <span class="math inline">\(G^TX\)</span> and the <span class="math inline">\((n,p)\)</span> matrix of residuals is given by <span class="math inline">\(G-XX^TG\)</span>. The <span class="math inline">\(z\)</span>-scores are then computed using the standard formula for multiple regression</p>
<span class="math display" id="eq:z-def">\[\begin{equation}
  z_{jk} = \hat{\beta}_{jk}\sqrt{\frac{\sum_{i=1}^n x_{ik}^2}{\sigma_j^2}}
  \tag{3.7}
\end{equation}\]</span>
<p>where <span class="math inline">\(\sigma_j^2\)</span> is an estimate of the residual variance for the <span class="math inline">\(j^{th}\)</span> SNP, and <span class="math inline">\(x_{ik}\)</span> is the score of the <span class="math inline">\(k^{th}\)</span> principal component for the <span class="math inline">\(i^{th}\)</span> individual.</p>
</div>
<div id="missing-data" class="section level4 unnumbered">
<h4>Missing data</h4>
<p>Missing data should be accounted for when computing principal components and when computing the matrix of <span class="math inline">\(z\)</span>-scores. There are many methods to account for missing data in PCA, and we consider the pairwise covariance approach <span class="citation">(Dray &amp; Josse, <a href="#ref-dray2015principal">2015</a>)</span>. It consists in estimating the covariance between each pair of individuals using only the markers that are available for both individuals. To compute <span class="math inline">\(z\)</span>-scores, we account for missing data in formula <a href="section.html#eq:z-def">(3.7)</a>. The term in the numerator <span class="math inline">\(\sum_{i=1}^n x_{ik}^2\)</span> depends on the quantity of missing data. If there are no missing data, it is equal to 1 by definition of the scores obtained with PCA. As the quantity of missing data grows, this term and the <span class="math inline">\(z\)</span>-score decrease such that it becomes more difficult to detect outlier markers.</p>
</div>
<div id="pooled-sequence-data" class="section level4 unnumbered">
<h4>Pooled sequence data</h4>
<p>When data are sequenced in pool, the Mahalanobis distance is based on the matrix of allele frequency computed in each pool instead of the matrix of z-scores.</p>
</div>
</div>
<div id="materials-and-methods-1" class="section level3 unnumbered">
<h3>Materials and methods</h3>
<div id="simulated-data" class="section level4 unnumbered">
<h4>Simulated data</h4>
<p>We simulated SNPs under an island model, under a divergence model and we downloaded simulations of range expansion <span class="citation">(Lotterhos &amp; Whitlock, <a href="#ref-lotterhos2015relative">2015</a>)</span>. All data we simulated were composed of 3 populations, each of them containing 50 sampled diploid individuals (Table <a href="section.html#tab:table2">3.3</a>). SNPs were simulated assuming no linkage disequilibrium. SNPs with minor allele frequencies lower than 5% were discarded from the data sets. The mean <span class="math inline">\(F_{ST}\)</span> for each simulation was comprised between 5% and 10%. Using the simulations based on an island and a divergence model, we also created data sets composed of admixed individuals. We assumed that an instantaneous admixture event occurs at the present time so that all sampled individuals are the results of this admixture event. Admixed individuals were generated by drawing randomly admixture proportions using a Dirichlet distribution of parameter <span class="math inline">\((\alpha, \alpha, \alpha)\)</span> (<span class="math inline">\(\alpha\)</span> ranging from 0.005 to 1 depending on the simulation).</p>

<table>
<caption><span id="tab:table2">Table 3.3: </span>Summary of the simulations. The table above shows the average number of individuals, of SNPs, of adaptive markers and the total number of simulations per scenario</caption>
<thead>
<tr class="header">
<th></th>
<th align="right">Individuals</th>
<th align="right">SNPs</th>
<th align="right">Adaptive SNPs</th>
<th align="right">Simulations</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Island model</td>
<td align="right">150</td>
<td align="right">472</td>
<td align="right">27</td>
<td align="right">35</td>
</tr>
<tr class="even">
<td>Divergence model</td>
<td align="right">150</td>
<td align="right">3000</td>
<td align="right">100</td>
<td align="right">6</td>
</tr>
<tr class="odd">
<td>Island model (hybrids)</td>
<td align="right">150</td>
<td align="right">472</td>
<td align="right">30</td>
<td align="right">27</td>
</tr>
<tr class="even">
<td>Divergence model (hybrids)</td>
<td align="right">150</td>
<td align="right">3000</td>
<td align="right">100</td>
<td align="right">9</td>
</tr>
<tr class="odd">
<td>Range expansion</td>
<td align="right">1200</td>
<td align="right">9999</td>
<td align="right">99</td>
<td align="right">6</td>
</tr>
</tbody>
</table>
</div>
<div id="island-model-1" class="section level4 unnumbered">
<h4>Island model</h4>
<p>We used <em>ms</em> to create simulations under an island model (Fig. <a href="informations-supplementaires.html#fig:FigureSI1">B.1</a>). We set a lower migration rate for the 50 adaptive SNPs compared with the 950 neutral ones to mimic diversifying selection <span class="citation">(Bazin et al., <a href="#ref-bazin2010likelihood">2010</a>)</span>. For a given locus, migration from population <span class="math inline">\(i\)</span> to <span class="math inline">\(j\)</span> was specified by choosing a value of the effective migration rate that is set to <span class="math inline">\(M_{\text{neutral}} = 10\)</span> for neutral SNPs and to <span class="math inline">\(M_{\text{adaptive}}\)</span> for adaptive ones. We simulated 35 data sets in the island model with different strengths of selection, where the strength of selection corresponds to the ratio <span class="math inline">\(M_{\text{neutral}} / M_{\text{adaptive}}\)</span> that varies from 10 to 1000. The <em>ms</em> command lines for neutral and adaptive SNPs are given by (<span class="math inline">\(M_{\text{adaptive}} = 0.01\)</span> and <span class="math inline">\(M_{\text{neutral}} = 10\)</span>).</p>
<div class="sourceCode"><pre class="sourceCode bash"><code class="sourceCode bash"><span class="kw">./ms</span> 300 950 -s 1 -I 3 100 100 100

<span class="kw">-ma</span> x 10 10 10 x 10 10 10 x

<span class="kw">./ms</span> 300 50 -s 1 -I 3 100 100 100

<span class="kw">-ma</span> x 0.01 0.01 0.01 x 0.01 0.01 0.01 x</code></pre></div>
</div>
<div id="divergence-model-1" class="section level4 unnumbered">
<h4>Divergence model</h4>
<p>To perform simulations under a divergence model, we used the package <em>simuPOP</em>, which is an individual-based population genetic simulation environment <span class="citation">(B. Peng &amp; Kimmel, <a href="#ref-peng2005simupop">2005</a>)</span>. We assumed that an ancestral panmictic population evolved during 20 generations before splitting into two subpopulations. The second subpopulation then split into subpopulations 2 and 3 at time <span class="math inline">\(T &gt; 20\)</span>. All 3 subpopulations continued to evolve until 200 generations have been reached, without migration between them (Figure <a href="informations-supplementaires.html#fig:FigureSI1">B.1</a>). A total of 50 diploid individuals were sampled in each population. Selection only occurred in the branch associated with population 2 and selection was simulated by assuming an additive model (fitness is equal to <span class="math inline">\(1 - 2s\)</span>, <span class="math inline">\(1 - s\)</span>, <span class="math inline">\(1\)</span> depending on the genotypes). We simulated a total of 3000 SNPs comprising of 100 adaptive ones for which the selection coefficient is of <span class="math inline">\(s = 0.1\)</span>.</p>
</div>
<div id="range-expansion" class="section level4 unnumbered">
<h4>Range expansion</h4>
<p>We downloaded in the <em>Dryad Digital Repository</em> six simulations of range expansion with two glacial refugia <span class="citation">(Lotterhos &amp; Whitlock, <a href="#ref-lotterhos2015relative">2015</a>)</span>. Adaptation occurred during the recolonization phase of the species range from the two refugia. We considered six different simulated data with 30 populations and a number of sampled individuals per location that varies from 20 to 60.</p>
</div>
<div id="parameter-settings-for-the-different-computer-programs" class="section level4 unnumbered">
<h4>Parameter settings for the different computer programs</h4>
<p>When using <em>hapflk</em>, we set <span class="math inline">\(K = 1\)</span> that corresponds to the computation of the <span class="math inline">\(F_{LK}\)</span> statistic. When using <em>BayeScan</em> and <em>OutFLANK</em>, we used the default parameter values. For <em>sNMF</em>, we used <span class="math inline">\(K = 3\)</span> for the island and divergence model and <span class="math inline">\(K = 5\)</span> for range expansion as indicated by the cross-entropy criterion. The regularization parameter of <em>sNMF</em> was set to <span class="math inline">\(\alpha = 1000\)</span>. For <em>sNMF</em> and <em>hapflk</em>, we used the genomic inflation factor to recalibrate <span class="math inline">\(p\)</span>-values. When using population-based methods with admixed individuals, we assigned each individual to the population with maximum amount of ancestry.</p>
</div>
</div>
<div id="results-1" class="section level3 unnumbered">
<h3>Results</h3>
<div id="choosing-the-number-of-principal-components-1" class="section level4 unnumbered">
<h4>Choosing the number of principal components</h4>
<p>We evaluate Cattell’s graphical rule to choose the number of principal components. For the island and divergence model, the choice of <span class="math inline">\(K\)</span> is evident (Fig. <a href="section.html#fig:choice">3.8</a>). For <span class="math inline">\(K \geq 3\)</span>, the eigenvalues follow a straight line. As a consequence, Cattell’s rule indicates <span class="math inline">\(K = 2\)</span>, which is expected because there are three populations <span class="citation">(N. Patterson et al., <a href="#ref-patterson2006population">2006</a>)</span>. For the model of range expansion, applying Cattell’s rule to choose <span class="math inline">\(K\)</span> is more difficult (Fig. <a href="section.html#fig:choice">3.8</a>). Ideally, the eigenvalues that correspond to random variation lie on a straight line whereas the ones corresponding to population structure depart from the line. However, there is no obvious point at which eigenvalues depart from the straight line. Choosing a value of <span class="math inline">\(K\)</span> between 5 and 8 is compatible with Cattell’s rule. Using the package qvalue to control 10% of FDR, we find that the actual proportion of false discoveries as well as statistical power is weakly impacted when varying the number of principal components from <span class="math inline">\(K = 5\)</span> to <span class="math inline">\(K = 8\)</span> (Figure <a href="informations-supplementaires.html#fig:FigureSI2">B.2</a>).</p>


<div class="figure" style="text-align: center"><span id="fig:choice"></span>
<img src="thesis_files/figure-html/choice-1.png" alt="Determining \(K\) with the scree plot. To choose \(K\), we recommend to use Cattell’s rule that states that components corresponding to eigenvalues to the left of the straight line should be kept. According to Cattell’s rule, the eigenvalues that correspond to random variation lie on the straight line whereas the ones corresponding to population structure depart from the line. For the island and divergence model, the choice of \(K\) is evident. For the model or range expansion, a value of \(K\) between 5 and 8 is compatible with Cattell’s rule." width="\textwidth" />
<p class="caption">
Figure 3.8: Determining <span class="math inline">\(K\)</span> with the scree plot. To choose <span class="math inline">\(K\)</span>, we recommend to use Cattell’s rule that states that components corresponding to eigenvalues to the left of the straight line should be kept. According to Cattell’s rule, the eigenvalues that correspond to random variation lie on the straight line whereas the ones corresponding to population structure depart from the line. For the island and divergence model, the choice of <span class="math inline">\(K\)</span> is evident. For the model or range expansion, a value of <span class="math inline">\(K\)</span> between 5 and 8 is compatible with Cattell’s rule.
</p>
</div>
</div>
<div id="an-example-of-genome-scans-performed-with-pcadapt" class="section level4 unnumbered">
<h4>An example of genome scans performed with <em>pcadapt</em></h4>
<p>To provide an example of results, we apply <em>pcadapt</em> with <span class="math inline">\(K = 6\)</span> in the model of range expansion. Population structure captured by the first two principal components is displayed in Fig. <a href="section.html#fig:scanex">3.9</a>. <span class="math inline">\(P\)</span>-values are well calibrated because they are distributed as a mixture of a uniform distribution and of a peaky distribution around 0, which corresponds to outlier loci (Fig. <a href="section.html#fig:scanex">3.9</a>). Using a FDR threshold of 10% with the <em>qvalue</em> package, we find 122 outliers among 10 000 SNPs, resulting in 23% actual false discoveries and a power of 95%.</p>

<div class="figure" style="text-align: center"><span id="fig:scanex"></span>
<img src="thesis_files/figure-html/scanex-1.png" alt="Population structure (first 2 principal components) and distribution of \(p\)-values obtained with pcadapt for a simulation of range expansion. \(P\)-values are well calibrated because they are distributed as a mixture of a uniform distribution and of a peaky distribution around 0, which corresponds to outlier loci. In the left panel, each colour corresponds to individuals sampled from the same population." width="\textwidth" />
<p class="caption">
Figure 3.9: Population structure (first 2 principal components) and distribution of <span class="math inline">\(p\)</span>-values obtained with <em>pcadapt</em> for a simulation of range expansion. <span class="math inline">\(P\)</span>-values are well calibrated because they are distributed as a mixture of a uniform distribution and of a peaky distribution around 0, which corresponds to outlier loci. In the left panel, each colour corresponds to individuals sampled from the same population.
</p>
</div>
</div>
<div id="control-of-the-false-discovery-rate" class="section level4">
<h4><span class="header-section-number">3.2.0.1</span> Control of the false discovery rate</h4>
<p>We evaluate to what extent using the packages <em>pcadapt</em> and <em>qvalue</em> control a FDR set at 10% (Fig. <a href="section.html#fig:jitter">3.10</a>). All SNPs with a <span class="math inline">\(q\)</span>-value smaller than 10% were considered as candidate SNPs. For the island model, we find that the proportion of false discoveries is 8% and it increases to 10% when including admixture. For the divergence model, the proportion of false discoveries is 11% and it increases to 22% when including admixture. The largest proportion of false discoveries is obtained under range expansion and is equal to 25%.</p>

<div class="figure" style="text-align: center"><span id="fig:jitter"></span>
<img src="thesis_files/figure-html/jitter-1.png" alt="Control of the FDR for different computer programs for genome scans. We find that the median proportion of false discoveries is around the nominal FDR set at 10% (6% for hapflk, 11% for both OutFLANK and pcadapt and 19% for sNMF) with the exception of BayeScan that generates 41% of false discoveries." width="\textwidth" />
<p class="caption">
Figure 3.10: Control of the FDR for different computer programs for genome scans. We find that the median proportion of false discoveries is around the nominal FDR set at 10% (6% for <em>hapflk</em>, 11% for both <em>OutFLANK</em> and <em>pcadapt</em> and 19% for <em>sNMF</em>) with the exception of <em>BayeScan</em> that generates 41% of false discoveries.
</p>
</div>
<p>We then evaluate the proportion of false discoveries obtained with <em>BayeScan</em>, <em>hapflk</em>, <em>OutFLANK</em> and <em>sNMF</em> (Fig. <a href="section.html#fig:jitter">3.10</a>). We find that <em>hapflk</em> is the most conservative approach (FDR = 6%) followed by <em>OutFLANK</em> and <em>pcadapt</em> (FDR = 11%). The computer program <em>sNMF</em> is more liberal (FDR = 19%) and <em>BayeScan</em> generates the largest proportion of false discoveries (FDR = 41%). When not recalibrating the p-values of <em>hapflk</em>, we find that the test is even more conservative (results not shown). For all programs, the range expansion scenario is the one that generates the largest proportion of false discoveries. Proportion of false discoveries under range expansion ranges from 22% (<em>OutFLANK</em>) to 93% (<em>BayeScan</em>).</p>
</div>
<div id="statistical-power" class="section level4 unnumbered">
<h4>Statistical power</h4>
<p>To provide a fair comparison between methods and computer programs, we compare statistical power for equal values of the observed proportion of false discoveries. Then we compute statistical power averaged over observed proportion of false discoveries ranging from 0% to 50%.</p>
<p>We first compare statistical power obtained with the different statistical methods that have been implemented in <em>pcadapt</em> (Table <a href="section.html#tab:table1">3.2</a>). For the island model, Bayes factor, communality statistic and Mahalanobis distance have similar power (Fig. <a href="section.html#fig:bayes-comm-maha">3.11</a>). For the divergence model, the power obtained with Mahalanobis distance is 20% whereas the power obtained with the communality statistic and with the Bayes factor is, respectively, 4% and 2% (Fig. <a href="section.html#fig:bayes-comm-maha">3.11</a>). Similarly, for range expansion, the power obtained with Mahalanobis distance is 46% whereas the power obtained with the communality statistic and with the Bayes factor is 34% and 13%. We additionally investigate to what extent increasing sample size in each population from 20 to 60 individuals affects power. For range expansion, the power obtained with the Mahalanobis distance hardly changes ranging from 44% to 47%. However, the power obtained with the other two statistics changes importantly. The power obtained with the communality statistic increases from 27% to 39% when increasing the sample size and the power obtained with the Bayes factor increases from 0% to 44%.</p>
<p></p>

<div class="figure" style="text-align: center"><span id="fig:bayes-comm-maha"></span>
<img src="thesis_files/figure-html/bayes-comm-maha-1.png" alt="Bayes factor corresponds to the test statistic implemented in the Bayesian version of pcadapt (Duforet-Frebourg et al., 2014); the communality statistic was the default statistic in version 1.x of the R package pcadapt (Duforet-Frebourg et al., 2015), and Mahalanobis distances are available since the release of the 2.0 version of the package. When there is hierarchical population structure (divergence model and range expansion), the Mahalanobis distance provides more powerful genome scans compared with the test statistic previously implemented in pcadapt. The abbreviation dist. stands for distance. Statistical power is averaged over the observed proportion of false discoveries (ranging between 0% and 50%)." width="\textwidth" />
<p class="caption">
Figure 3.11: Bayes factor corresponds to the test statistic implemented in the Bayesian version of <em>pcadapt</em> <span class="citation">(Duforet-Frebourg et al., <a href="#ref-duforet2014genome">2014</a>)</span>; the communality statistic was the default statistic in version 1.x of the R package <em>pcadapt</em> <span class="citation">(Duforet-Frebourg et al., <a href="#ref-duforet2015detecting">2015</a>)</span>, and Mahalanobis distances are available since the release of the 2.0 version of the package. When there is hierarchical population structure (divergence model and range expansion), the Mahalanobis distance provides more powerful genome scans compared with the test statistic previously implemented in pcadapt. The abbreviation dist. stands for distance. Statistical power is averaged over the observed proportion of false discoveries (ranging between 0% and 50%).
</p>
</div>
<p>Then we describe our comparison of computer programs for genome scans. For the simulations obtained with the island model where there is no hierarchical population structure, the statistical power is similar for all programs (Figure <a href="informations-supplementaires.html#fig:FigureSI3">B.3</a> and <a href="informations-supplementaires.html#fig:FigureSI4">B.4</a>). Including admixed individuals hardly changes their statistical power (Figure <a href="informations-supplementaires.html#fig:FigureSI3">B.3</a>).</p>
<p>Then, we compare statistical power in a divergence model where adaptation took place in one of the external branches of the population divergence tree. The programs <em>pcadapt</em> and <em>hapflk</em>, which account for hierarchical population structure, as well as <em>BayeScan</em> are the most powerful in that setting (Fig. <a href="section.html#fig:power-div">3.12</a> and Figure <a href="informations-supplementaires.html#fig:FigureSI5">B.5</a>). The values of power in decreasing order are of 23% for <em>BayeScan</em>, of 20% for <em>pcadapt</em>, of 17% for <em>hapflk</em>, of 7% for <em>sNMF</em> and of 1% for <em>OutFLANK</em>. When including admixed individuals, the power of <em>hapflk</em> and of <em>pcadapt</em> hardly decreases whereas the power of <em>BayeScan</em> decreases to 6% (Fig. <a href="section.html#fig:power-div">3.12</a>).</p>

<div class="figure" style="text-align: center"><span id="fig:power-div"></span>
<img src="thesis_files/figure-html/power-div-1.png" alt="Statistical power averaged over the expected proportion of false discoveries (ranging between 0% and 50%) for the divergence model with three populations. We assume that adaptation took place in an external branch that follows the most recent population divergence event." width="\textwidth" />
<p class="caption">
Figure 3.12: Statistical power averaged over the expected proportion of false discoveries (ranging between 0% and 50%) for the divergence model with three populations. We assume that adaptation took place in an external branch that follows the most recent population divergence event.
</p>
</div>
<p>The last model we consider is the model of range expansion. The package <em>pcadapt</em> is the most powerful approach in this setting (Fig. <a href="section.html#fig:power-rexp">3.13</a> and <a href="informations-supplementaires.html#fig:FigureSI6">B.6</a>). Other computer programs also discover many true-positive loci with the exception of <em>BayeScan</em> that provides no true discovery when the observed FDR is smaller than 50% (Fig. <a href="section.html#fig:power-rexp">3.13</a> and <a href="informations-supplementaires.html#fig:FigureSI6">B.6</a>). The values of power in decreasing order are of 46% for <em>pcadapt</em>, of 41% for <em>hapflk</em>, of 37% for <em>OutFLANK</em>, of 30% for <em>sNMF</em> and of 0% for <em>BayeScan</em>.</p>

<div class="figure" style="text-align: center"><span id="fig:power-rexp"></span>
<img src="thesis_files/figure-html/power-rexp-1.png" alt="Statistical power averaged over the expected proportion of false discoveries (ranging between 0% and 50%) for a range expansion model with two refugia. Adaptation took place during the recolonization event." width="\textwidth" />
<p class="caption">
Figure 3.13: Statistical power averaged over the expected proportion of false discoveries (ranging between 0% and 50%) for a range expansion model with two refugia. Adaptation took place during the recolonization event.
</p>
</div>
</div>
<div id="running-time-of-the-different-computer-programs" class="section level4 unnumbered">
<h4>Running time of the different computer programs</h4>
<p>Last, we compare running times. The characteristics of the computer we used to perform comparisons are the following: OSX El Capitan 10.11.3, 2,5 GHz Intel Core i5, 8 Go 1600 MHz DDR3. We discard <em>BayeScan</em> as it is too time-consuming. For instance, running <em>BayeScan</em> on a genotype matrix containing 150 individuals and 3000 SNPs takes 9h whereas it takes less than one second with <em>pcadapt</em>. The different programs were run on genotype matrices containing 300 individuals and from 500 to 50 000 SNPs. <em>OutFLANK</em> is the computer program for which the runtime increases the most rapidly with the number of markers. <em>OutFLANK</em> takes around 25 min to analyse 50 000 SNPs (Figure <a href="informations-supplementaires.html#fig:FigureSI7">B.7</a>). For the other 3 computer programs (<em>hapflk</em>, <em>pcadapt</em>, <em>sNMF</em>), analysing 50 000 SNPs takes less than 3 min.</p>
</div>
</div>
<div id="discussion-1" class="section level3 unnumbered">
<h3>Discussion</h3>
<p>The R package <em>pcadapt</em> implements a fast method to perform genome scans with next-generation sequencing data. It can handle data sets where population structure is continuous or data sets containing admixed individuals. It can handle missing data as well as pooled sequencing data. The 2.0 and later versions of the R package implements a robust Mahalanobis distance as a test statistic. When hierarchical population structure occurs, Mahalanobis distance provides more powerful genome scans compared with the communality statistic that was implemented in the first version of the package <span class="citation">(Duforet-Frebourg et al., <a href="#ref-duforet2015detecting">2015</a>)</span>. In the divergence model, adaptation occurs along an external branch of the divergence tree that corresponds to the second principal component. When outlier SNPs are not related to the first principal component, the Mahalanobis distance provides a better ranking of the SNPs compared with the communality statistic.</p>
<p>Simulations show that the R package <em>pcadapt</em> compares favourably to other computer programs for genome scans. When data were simulated under an island model, population structure is not hierarchical because genetic differentiation is the same for all pairs of populations. Statistical power and control of the FDR were similar for all computer programs. In the presence of hierarchical population structure (divergence model) where genetic differentiation varies between pairs of populations, the ranking of the SNPs depends on the computer program. <em>pcadapt</em> and <em>hapflk</em> provide the most powerful scans whether or not simulations include admixed individuals. <em>OutFLANK</em> implements a <span class="math inline">\(F_{ST}\)</span> statistic and because adaptation does not correspond to the most differentiated populations, it fails to capture adaptive SNPs (Fig. <a href="section.html#fig:power-div">3.12</a>) <span class="citation">(Bonhomme et al., <a href="#ref-bonhomme2010detecting">2010</a>; Duforet-Frebourg et al., <a href="#ref-duforet2015detecting">2015</a>)</span>. <em>BayeScan</em> does not assume equal differentiation between all pairs of populations, which may explain why it has a good statistical power for the divergence model. However, its statistical power is severely impacted by the presence of admixed individuals because its power decreases from 24% to 6% (Fig. <a href="section.html#fig:power-div">3.12</a>). Understanding why <em>BayeScan</em> is severely impacted by admixture is out of the scope of this study. In the range expansion model, <em>BayeScan</em> returns many null q-values (between 376 and 809 SNPs of 9899 neutral and 100 adaptive SNPs) such that the observed FDR is always larger than 50%. Overall, we find that <em>pcadapt</em> and <em>hapflk</em> provide comparable statistical power. They provide optimal or near optimal ranking of the SNPs in different scenarios including hierarchical population structure and admixed individuals. The main difference between the two computer programs concerns the control of the FDR because hapflk is found to be more conservative.</p>
<p>Because NGS data become more and more massive, careful numerical implementation is crucial. There are different options to implement PCA and <em>pcadapt</em> uses a numerical routine based on the computation of the covariance matrix <span class="math inline">\(\Omega\)</span>. The algorithmic complexity to compute the covariance matrix is proportional to <span class="math inline">\(pn^2\)</span> where <span class="math inline">\(p\)</span> is the number of markers and <span class="math inline">\(n\)</span> is the number of individuals. The computation of the first <span class="math inline">\(K\)</span> eigenvectors of the covariance matrix <span class="math inline">\(\Omega\)</span> has a complexity proportional to <span class="math inline">\(n^3\)</span>. This second step is usually more rapid than the computation of the covariance because the number of markers is usually large compared with the number of individuals. In brief, computing the covariance matrix <span class="math inline">\(\Omega\)</span> is by far the most costly operation when computing principal components. Although we have implemented PCA in C to obtain fast computations, an improvement in speed could be envisioned for future versions. When the number of individuals becomes large (e.g. <span class="math inline">\(n \geq 10 000\)</span>), there are faster algorithms to compute principal components <span class="citation">(Abraham &amp; Inouye, <a href="#ref-abraham2014fast">2014</a>; Halko, Martinsson, &amp; Tropp, <a href="#ref-halko2011finding">2011</a>)</span>. In addition to running time, numerical implementations also impact the effect of missing data on principal components <span class="citation">(Dray &amp; Josse, <a href="#ref-dray2015principal">2015</a>)</span>. Achieving a good trade-off between fast computations and accurate evaluation of population structure in the face of large amount of missing data is a challenge for modern numerical methods in molecular ecology.</p>
</div>
<div id="acknowledgements" class="section level3 unnumbered">
<h3>Acknowledgements</h3>
<p>This work has been supported by the LabEx PERSYVAL-Lab (ANR-11-LABX-0025-01) and the ANR AGRHUM project (ANR-14-CE02-0003-01). We want to thank two anonymous reviewers and Stéphane Dray for their critical reading of our manuscript.</p>
<p>K.L., E.B. and M.G.B.B. designed and performed the research.</p>
</div>
<div id="data-accessibility" class="section level3 unnumbered">
<h3>Data accessibility</h3>
<p>Island and divergence model data: doi: 10.5061/dryad.8290n.</p>
<p>Range expansion simulated data: doi: 10.5061/dryad.mh67v. Files:</p>
<p><code>2R_R30_1351142954_453_2_NumPops=30_NumInd=20</code></p>
<p><code>2R_R30_1351142954_453_2_NumPops=30_NumInd=60</code></p>
<p><code>2R_R30_1351142970_988_6_NumPops=30_NumInd=20</code></p>
<p><code>2R_R30_1351142970_988_6_NumPops=30_NumInd=60</code></p>
<p><code>2R_R30_1351142986_950_10_NumPops=30_NumInd=20</code></p>
<p><code>2R_R30_1351142986_950_10_NumPops=30_NumInd=60</code></p>

</div>
</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-duforet2014statistiques">
<p>Duforet-Frebourg, N. (2014). <em>Statistiques bayésiennes en génétique des populations: Modèle à facteurs et processus gaussiens pour étudier la variation génétique neutre et adaptative</em> (PhD thesis). Grenoble.</p>
</div>
<div id="ref-barreiro2008natural">
<p>Barreiro, L. B., Laval, G., Quach, H., Patin, E., &amp; Quintana-Murci, L. (2008). Natural selection has driven population differentiation in modern humans. <em>Nature Genetics</em>, <em>40</em>(3), 340–345.</p>
</div>
<div id="ref-grossman2013identifying">
<p>Grossman, S. R., Andersen, K. G., Shlyakhter, I., Tabrizi, S., Winnicki, S., Yen, A., … others. (2013). Identifying recent adaptations in large-scale genomic data. <em>Cell</em>, <em>152</em>(4), 703–713.</p>
</div>
<div id="ref-pickrell2009signals">
<p>Pickrell, J. K., Coop, G., Novembre, J., Kudaravalli, S., Li, J. Z., Absher, D., … others. (2009). Signals of recent positive selection in a worldwide sample of human populations. <em>Genome Research</em>, <em>19</em>(5), 826–837.</p>
</div>
<div id="ref-sabeti2007genome">
<p>Sabeti, P. C., Varilly, P., Fry, B., Lohmueller, J., Hostetter, E., Cotsapas, C., … others. (2007). Genome-wide detection and characterization of positive selection in human populations. <em>Nature</em>, <em>449</em>(7164), 913.</p>
</div>
<div id="ref-colonna2014human">
<p>Colonna, V., Ayub, Q., Chen, Y., Pagani, L., Luisi, P., Pybus, M., … Tyler-Smith, C. (2014). Human genomic regions with exceptionally high levels of population differentiation identified from 911 whole-genome sequences. <em>Genome Biology</em>, <em>15</em>(6), R88.</p>
</div>
<div id="ref-nielsen2005molecular">
<p>Nielsen, R. (2005). Molecular signatures of natural selection. <em>Annu. Rev. Genet.</em>, <em>39</em>, 197–218.</p>
</div>
<div id="ref-sabeti2006positive">
<p>Sabeti, P. C., Schaffner, S. F., Fry, B., Lohmueller, J., Varilly, P., Shamovsky, O., … Lander, E. (2006). Positive natural selection in the human lineage. <em>Science</em>, <em>312</em>(5780), 1614–1620.</p>
</div>
<div id="ref-hamblin2002complex">
<p>Hamblin, M. T., Thompson, E. E., &amp; Di Rienzo, A. (2002). Complex signatures of natural selection at the duffy blood group locus. <em>The American Journal of Human Genetics</em>, <em>70</em>(2), 369–383.</p>
</div>
<div id="ref-hancock2010human">
<p>Hancock, A. M., Witonsky, D. B., Ehler, E., Alkorta-Aranburu, G., Beall, C., Gebremedhin, A., … others. (2010). Human adaptations to diet, subsistence, and ecoregion are due to subtle shifts in allele frequency. <em>Proceedings of the National Academy of Sciences</em>, <em>107</em>(Supplement 2), 8924–8930.</p>
</div>
<div id="ref-luikart2003power">
<p>Luikart, G., England, P. R., Tallmon, D., Jordan, S., &amp; Taberlet, P. (2003). The power and promise of population genomics: From genotyping to genome typing. <em>Nature Reviews. Genetics</em>, <em>4</em>(12), 981.</p>
</div>
<div id="ref-excoffier1992analysis">
<p>Excoffier, L., Smouse, P. E., &amp; Quattro, J. M. (1992). Analysis of molecular variance inferred from metric distances among dna haplotypes: Application to human mitochondrial dna restriction data. <em>Genetics</em>, <em>131</em>(2), 479–491.</p>
</div>
<div id="ref-novembre2008genes">
<p>Novembre, J., Johnson, T., Bryc, K., Kutalik, Z., Boyko, A. R., Auton, A., … others. (2008). Genes mirror geography within europe. <em>Nature</em>, <em>456</em>(7218), 98.</p>
</div>
<div id="ref-bonhomme2010detecting">
<p>Bonhomme, M., Chevalet, C., Servin, B., Boitard, S., Abdallah, J., Blott, S., &amp; SanCristobal, M. (2010). Detecting selection in population trees: The lewontin and krakauer test extended. <em>Genetics</em>, <em>186</em>(1), 241–262.</p>
</div>
<div id="ref-fariello2013detecting">
<p>Fariello, M. I., Boitard, S., Naya, H., SanCristobal, M., &amp; Servin, B. (2013). Detecting signatures of selection through haplotype differentiation among hierarchically structured populations. <em>Genetics</em>, <em>193</em>(3), 929–941.</p>
</div>
<div id="ref-gunther2013robust">
<p>Günther, T., &amp; Coop, G. (2013). Robust identification of local adaptation from allele frequencies. <em>Genetics</em>, <em>195</em>(1), 205–220.</p>
</div>
<div id="ref-yang2012model">
<p>Yang, W.-Y., Novembre, J., Eskin, E., &amp; Halperin, E. (2012). A model-based approach for analysis of spatial structure in genetic data. <em>Nature Genetics</em>, <em>44</em>(6), 725–731.</p>
</div>
<div id="ref-duforet2014genome">
<p>Duforet-Frebourg, N., Bazin, E., &amp; Blum, M. G. (2014). Genome scans for detecting footprints of local adaptation using a bayesian factor model. <em>Molecular Biology and Evolution</em>, <em>31</em>(9), 2483–2495.</p>
</div>
<div id="ref-patterson2006population">
<p>Patterson, N., Price, A. L., &amp; Reich, D. (2006). Population structure and eigenanalysis. <em>PLoS Genetics</em>, <em>2</em>(12), e190.</p>
</div>
<div id="ref-mcvean2009genealogical">
<p>McVean, G. (2009). A genealogical interpretation of principal components analysis. <em>PLoS Genetics</em>, <em>5</em>(10), e1000686.</p>
</div>
<div id="ref-foll2008genome">
<p>Foll, M., &amp; Gaggiotti, O. (2008). A genome-scan method to identify selected loci appropriate for both dominant and codominant markers: A bayesian perspective. <em>Genetics</em>, <em>180</em>(2), 977–993.</p>
</div>
<div id="ref-riebler2008bayesian">
<p>Riebler, A., Held, L., &amp; Stephan, W. (2008). Bayesian variable selection for detecting adaptive genomic differences among populations. <em>Genetics</em>, <em>178</em>(3), 1817–1829.</p>
</div>
<div id="ref-jolliffe1986principal">
<p>Jolliffe, I. T. (1986). Principal component analysis and factor analysis. In <em>Principal component analysis</em> (pp. 115–128). Springer.</p>
</div>
<div id="ref-cadima1995loading">
<p>Cadima, J., &amp; Jolliffe, I. T. (1995). Loading and correlations in the interpretation of principle compenents. <em>Journal of Applied Statistics</em>, <em>22</em>(2), 203–214.</p>
</div>
<div id="ref-anderson1999lapack">
<p>Anderson, E., Bai, Z., Bischof, C., Blackford, S., Demmel, J., Dongarra, J., … others. (1999). LAPACK users’ guide 3rd edn (philadelphia, pa: Society for industrial and applied mathematics).</p>
</div>
<div id="ref-daub2013evidence">
<p>Daub, J. T., Hofer, T., Cutivet, E., Dupanloup, I., Quintana-Murci, L., Robinson-Rechavi, M., &amp; Excoffier, L. (2013). Evidence for polygenic adaptation to pathogens in the human genome. <em>Molecular Biology and Evolution</em>, <em>30</em>(7), 1544–1558.</p>
</div>
<div id="ref-fagny2014exploring">
<p>Fagny, M., Patin, E., Enard, D., Barreiro, L. B., Quintana-Murci, L., &amp; Laval, G. (2014). Exploring the occurrence of classic selective sweeps in humans using whole-genome sequencing data sets. <em>Molecular Biology and Evolution</em>, <em>31</em>(7), 1850–1868.</p>
</div>
<div id="ref-kudaravalli2008gene">
<p>Kudaravalli, S., Veyrieras, J.-B., Stranger, B. E., Dermitzakis, E. T., &amp; Pritchard, J. K. (2008). Gene expression levels are a target of recent natural selection in the human genome. <em>Molecular Biology and Evolution</em>, <em>26</em>(3), 649–658.</p>
</div>
<div id="ref-vitti2013detecting">
<p>Vitti, J. J., Grossman, S. R., &amp; Sabeti, P. C. (2013). Detecting natural selection in genomic data. <em>Annual Review of Genetics</em>, <em>47</em>, 97–120.</p>
</div>
<div id="ref-holsinger2009genetics">
<p>Holsinger, K. E., &amp; Weir, B. S. (2009). Genetics in geographically structured populations: Defining, estimating and interpreting fst. <em>Nature Reviews. Genetics</em>, <em>10</em>(9), 639.</p>
</div>
<div id="ref-nelson2008population">
<p>Nelson, M. R., Bryc, K., King, K. S., Indap, A., Boyko, A. R., Novembre, J., … others. (2008). The population reference sample, popres: A resource for population, disease, and pharmacological genetics research. <em>The American Journal of Human Genetics</em>, <em>83</em>(3), 347–358.</p>
</div>
<div id="ref-jay2012anisotropic">
<p>Jay, F., Sjödin, P., Jakobsson, M., &amp; Blum, M. G. (2012). Anisotropic isolation by distance: The main orientations of human genetic differentiation. <em>Molecular Biology and Evolution</em>, <em>30</em>(3), 513–525.</p>
</div>
<div id="ref-ahmed2009newly">
<p>Ahmed, S., Thomas, G., Ghoussaini, M., Healey, C. S., Humphreys, M. K., Platte, R., … others. (2009). Newly discovered breast cancer susceptibility loci on 3p24 and 17q23. 2. <em>Nature Genetics</em>, <em>41</em>(5), 585–590.</p>
</div>
<div id="ref-bersaglieri2004genetic">
<p>Bersaglieri, T., Sabeti, P. C., Patterson, N., Vanderploeg, T., Schaffner, S. F., Drake, J. A., … Hirschhorn, J. N. (2004). Genetic signatures of strong recent positive selection at the lactase gene. <em>The American Journal of Human Genetics</em>, <em>74</em>(6), 1111–1120.</p>
</div>
<div id="ref-han2007evidence">
<p>Han, Y., Gu, S., Oota, H., Osier, M. V., Pakstis, A. J., Speed, W. C., … Kidd, K. K. (2007). Evidence of positive selection on a class i adh locus. <em>The American Journal of Human Genetics</em>, <em>80</em>(3), 441–456.</p>
</div>
<div id="ref-wilde2014direct">
<p>Wilde, S., Timpson, A., Kirsanow, K., Kaiser, E., Kayser, M., Unterländer, M., … others. (2014). Direct evidence for positive selection of skin, hair, and eye pigmentation in europeans during the last 5,000 y. <em>Proceedings of the National Academy of Sciences</em>, <em>111</em>(13), 4832–4837.</p>
</div>
<div id="ref-bierne2013pervasive">
<p>Bierne, N., Roze, D., &amp; Welch, J. J. (2013). Pervasive selection or is it…? Why are fst outliers sometimes so frequent? <em>Molecular Ecology</em>, <em>22</em>(8), 2061–2064.</p>
</div>
<div id="ref-lotterhos2014evaluation">
<p>Lotterhos, K. E., &amp; Whitlock, M. C. (2014). Evaluation of demographic history and neutral parameterization on the performance of fst outlier tests. <em>Molecular Ecology</em>, <em>23</em>(9), 2178–2192.</p>
</div>
<div id="ref-foll2014widespread">
<p>Foll, M., Gaggiotti, O. E., Daub, J. T., Vatsiou, A., &amp; Excoffier, L. (2014). Widespread signals of convergent adaptation to high altitude in asia and america. <em>The American Journal of Human Genetics</em>, <em>95</em>(4), 394–407.</p>
</div>
<div id="ref-galinsky2016fast">
<p>Galinsky, K. J., Bhatia, G., Loh, P.-R., Georgiev, S., Mukherjee, S., Patterson, N. J., &amp; Price, A. L. (2016). Fast principal-component analysis reveals convergent evolution of adh1b in europe and east asia. <em>The American Journal of Human Genetics</em>, <em>98</em>(3), 456–472.</p>
</div>
<div id="ref-yuan2010two">
<p>Yuan, K.-H., &amp; Bentler, P. M. (2010). Two simple approximations to the distributions of quadratic forms. <em>British Journal of Mathematical and Statistical Psychology</em>, <em>63</em>(2), 273–291.</p>
</div>
<div id="ref-lewontin1973distribution">
<p>Lewontin, R., &amp; Krakauer, J. (1973). Distribution of gene frequency as a test of the theory of the selective neutrality of polymorphisms. <em>Genetics</em>, <em>74</em>(1), 175–195.</p>
</div>
<div id="ref-whitlock2015reliable">
<p>Whitlock, M. C., &amp; Lotterhos, K. E. (2015). Reliable detection of loci responsible for local adaptation: Inference of a null model through trimming the distribution of f st. <em>The American Naturalist</em>, <em>186</em>(S1), S24–S36.</p>
</div>
<div id="ref-hernandez2011classic">
<p>Hernandez, R. D., Kelley, J. L., Elyashiv, E., Melton, S. C., Auton, A., McVean, G., … others. (2011). Classic selective sweeps were rare in recent human evolution. <em>Science</em>, <em>331</em>(6019), 920–924.</p>
</div>
<div id="ref-li2012evidence">
<p>Li, J., Liu, Y., Xin, X., Kim, T. S., Cabeza, E. A., Ren, J., … Zhang, Z. (2012). Evidence for positive selection on a number of microrna regulatory interactions during recent human evolution. <em>PLoS Genetics</em>, <em>8</em>(3), e1002578.</p>
</div>
<div id="ref-williamson2007localizing">
<p>Williamson, S. H., Hubisz, M. J., Clark, A. G., Payseur, B. A., Bustamante, C. D., &amp; Nielsen, R. (2007). Localizing recent adaptive evolution in the human genome. <em>PLoS Genetics</em>, <em>3</em>(6), e90.</p>
</div>
<div id="ref-carlson2005genomic">
<p>Carlson, C. S., Thomas, D. J., Eberle, M. A., Swanson, J. E., Livingston, R. J., Rieder, M. J., &amp; Nickerson, D. A. (2005). Genomic regions exhibiting positive selection identified from dense genotype data. <em>Genome Research</em>, <em>15</em>(11), 1553–1565.</p>
</div>
<div id="ref-wu2010positive">
<p>Wu, D.-D., &amp; Zhang, Y.-P. (2010). Positive selection drives population differentiation in the skeletal genes in modern humans. <em>Human Molecular Genetics</em>, <em>19</em>(12), 2341–2346.</p>
</div>
<div id="ref-gurdasani2015african">
<p>Gurdasani, D., Carstensen, T., Tekola-Ayele, F., Pagani, L., Tachmazidou, I., Hatzikotoulas, K., … others. (2015). The african genome variation project shapes medical genetics in africa. <em>Nature</em>, <em>517</em>(7534), 327.</p>
</div>
<div id="ref-jiao2011genome">
<p>Jiao, H., Arner, P., Hoffstedt, J., Brodin, D., Dubern, B., Czernichow, S., … others. (2011). Genome wide association study identifies kcnma1 contributing to human obesity. <em>BMC Medical Genomics</em>, <em>4</em>(1), 51.</p>
</div>
<div id="ref-oeggerli2012role">
<p>Oeggerli, M., Tian, Y., Ruiz, C., Wijker, B., Sauter, G., Obermann, E., … others. (2012). Role of kcnma1 in breast cancer. <em>PloS One</em>, <em>7</em>(8), e41664.</p>
</div>
<div id="ref-chen2010population">
<p>Chen, H., Patterson, N., &amp; Reich, D. (2010). Population differentiation as a test for selective sweeps. <em>Genome Research</em>, <em>20</em>(3), 393–402.</p>
</div>
<div id="ref-fumagalli2010genome">
<p>Fumagalli, M., Pozzoli, U., Cagliani, R., Comi, G. P., Bresolin, N., Clerici, M., &amp; Sironi, M. (2010). Genome-wide identification of susceptibility alleles for viral infections through a population genetics approach. <em>PLoS Genetics</em>, <em>6</em>(2), e1000849.</p>
</div>
<div id="ref-hollox2008directional">
<p>Hollox, E. J., &amp; Armour, J. A. (2008). Directional and balancing selection in human beta-defensins. <em>BMC Evolutionary Biology</em>, <em>8</em>(1), 113.</p>
</div>
<div id="ref-cagliani2008signature">
<p>Cagliani, R., Fumagalli, M., Riva, S., Pozzoli, U., Comi, G. P., Menozzi, G., … Sironi, M. (2008). The signature of long-standing balancing selection at the human defensin <span class="math inline">\(\beta\)</span>-1 promoter. <em>Genome Biology</em>, <em>9</em>(9), R143.</p>
</div>
<div id="ref-peng2010adh1b">
<p>Peng, Y., Shi, H., Qi, X.-b., Xiao, C.-j., Zhong, H., Run-lin, Z. M., &amp; Su, B. (2010). The adh1b arg47his polymorphism in east asian populations and expansion of rice domestication in history. <em>BMC Evolutionary Biology</em>, <em>10</em>(1), 15.</p>
</div>
<div id="ref-barreiro2010evolutionary">
<p>Barreiro, L. B., &amp; Quintana-Murci, L. (2010). From evolutionary genetics to human immunology: How selection shapes host defence genes. <em>Nature Reviews. Genetics</em>, <em>11</em>(1), 17.</p>
</div>
<div id="ref-fumagalli2011signatures">
<p>Fumagalli, M., Sironi, M., Pozzoli, U., Ferrer-Admettla, A., Pattini, L., &amp; Nielsen, R. (2011). Signatures of environmental genetic adaptation pinpoint pathogens as the main selective pressure through human evolution. <em>PLoS Genetics</em>, <em>7</em>(11), e1002355.</p>
</div>
<div id="ref-hancock2008adaptations">
<p>Hancock, A. M., Witonsky, D. B., Gordon, A. S., Eshel, G., Pritchard, J. K., Coop, G., &amp; Di Rienzo, A. (2008). Adaptations to climate in candidate genes for common metabolic disorders. <em>PLoS Genetics</em>, <em>4</em>(2), e32.</p>
</div>
<div id="ref-hudson2002generating">
<p>Hudson, R. R. (2002). Generating samples under a wright–Fisher neutral model of genetic variation. <em>Bioinformatics</em>, <em>18</em>(2), 337–338.</p>
</div>
<div id="ref-peng2005simupop">
<p>Peng, B., &amp; Kimmel, M. (2005). SimuPOP: A forward-time population genetics simulation environment. <em>Bioinformatics</em>, <em>21</em>(18), 3686–3687.</p>
</div>
<div id="ref-weir1984estimating">
<p>Weir, B. S., &amp; Cockerham, C. C. (1984). Estimating f-statistics for the analysis of population structure. <em>Evolution</em>, <em>38</em>(6), 1358–1370.</p>
</div>
<div id="ref-10002012integrated">
<p>Consortium, 1. G. P., &amp; others. (2012). An integrated map of genetic variation from 1,092 human genomes. <em>Nature</em>, <em>491</em>(7422), 56.</p>
</div>
<div id="ref-kofler2012gowinda">
<p>Kofler, R., &amp; Schlötterer, C. (2012). Gowinda: Unbiased analysis of gene set enrichment for genome-wide association studies. <em>Bioinformatics</em>, <em>28</em>(15), 2084–2085.</p>
</div>
<div id="ref-franccois2016controlling">
<p>François, O., Martins, H., Caye, K., &amp; Schoville, S. D. (2016). Controlling false discoveries in genome scans for selection. <em>Molecular Ecology</em>, <em>25</em>(2), 454–469.</p>
</div>
<div id="ref-excoffier2009detecting">
<p>Excoffier, L., Hofer, T., &amp; Foll, M. (2009). Detecting loci under selection in a hierarchically structured population. <em>Heredity</em>, <em>103</em>(4), 285.</p>
</div>
<div id="ref-waples2006invited">
<p>Waples, R. S., &amp; Gaggiotti, O. (2006). INVITED review: What is a population? An empirical evaluation of some genetic methods for identifying the number of gene pools and their degree of connectivity. <em>Molecular Ecology</em>, <em>15</em>(6), 1419–1439.</p>
</div>
<div id="ref-lotterhos2015relative">
<p>Lotterhos, K. E., &amp; Whitlock, M. C. (2015). The relative power of genome scans to detect local adaptation depends on sampling design and statistical method. <em>Molecular Ecology</em>, <em>24</em>(5), 1031–1046.</p>
</div>
<div id="ref-pritchard2000inference">
<p>Pritchard, J. K., Stephens, M., &amp; Donnelly, P. (2000). Inference of population structure using multilocus genotype data. <em>Genetics</em>, <em>155</em>(2), 945–959.</p>
</div>
<div id="ref-chen2016eigengwas">
<p>Chen, G., Lee, S., Zhu, Z., Benyamin, B., &amp; Robinson, M. (2016). EigenGWAS: Finding loci under selection through genome-wide association studies of eigenvectors in structured populations. <em>Heredity</em>, <em>117</em>(1), 51.</p>
</div>
<div id="ref-duforet2015detecting">
<p>Duforet-Frebourg, N., Luu, K., Laval, G., Bazin, E., &amp; Blum, M. G. (2015). Detecting genomic signatures of natural selection with principal component analysis: Application to the 1000 genomes data. <em>Molecular Biology and Evolution</em>, <em>33</em>(4), 1082–1093.</p>
</div>
<div id="ref-hao2015probabilistic">
<p>Hao, W., Song, M., &amp; Storey, J. D. (2015). Probabilistic models of genetic variation in structured populations applied to global human studies. <em>Bioinformatics</em>, <em>32</em>(5), 713–721.</p>
</div>
<div id="ref-martins2016identifying">
<p>Martins, H., Caye, K., Luu, K., Blum, M. G., &amp; Francois, O. (2016). Identifying outlier loci in admixed and in continuous populations using ancestral population differentiation statistics. <em>Molecular Ecology</em>, <em>25</em>(20), 5029–5042.</p>
</div>
<div id="ref-lange2014next">
<p>Lange, K., Papp, J. C., Sinsheimer, J. S., &amp; Sobel, E. M. (2014). Next-generation statistical genetics: Modeling, penalization, and optimization in high-dimensional data. <em>Annual Review of Statistics and Its Application</em>, <em>1</em>, 279–300.</p>
</div>
<div id="ref-arnold2013radseq">
<p>Arnold, B., Corbett-Detig, R. B., Hartl, D., &amp; Bomblies, K. (2013). RADseq underestimates diversity and introduces genealogical biases due to nonrandom haplotype sampling. <em>Molecular Ecology</em>, <em>22</em>(11), 3179–3190.</p>
</div>
<div id="ref-gautier2013effect">
<p>Gautier, M., Gharbi, K., Cezard, T., Foucaud, J., Kerdelhué, C., Pudlo, P., … Estoup, A. (2013). The effect of rad allele dropout on the estimation of genetic variation within and between populations. <em>Molecular Ecology</em>, <em>22</em>(11), 3165–3178.</p>
</div>
<div id="ref-team2015r">
<p>Team, R. C. (2015). R: A language and environment for statistical computing [internet]. vienna, austria: R foundation for statistical computing; 2014.</p>
</div>
<div id="ref-maronna2002robust">
<p>Maronna, R. A., &amp; Zamar, R. H. (2002). Robust estimates of location and dispersion for high-dimensional datasets. <em>Technometrics</em>, <em>44</em>(4), 307–317.</p>
</div>
<div id="ref-frichot2014fast">
<p>Frichot, E., Mathieu, F., Trouillon, T., Bouchard, G., &amp; François, O. (2014). Fast and efficient estimation of individual ancestry coefficients. <em>Genetics</em>, <em>196</em>(4), 973–983.</p>
</div>
<div id="ref-jackson1993stopping">
<p>Jackson, D. A. (1993). Stopping rules in principal components analysis: A comparison of heuristical and statistical approaches. <em>Ecology</em>, <em>74</em>(8), 2204–2214.</p>
</div>
<div id="ref-cattell1966scree">
<p>Cattell, R. B. (1966). The scree test for the number of factors. <em>Multivariate Behavioral Research</em>, <em>1</em>(2), 245–276.</p>
</div>
<div id="ref-devlin1999genomic">
<p>Devlin, B., &amp; Roeder, K. (1999). Genomic control for association studies. <em>Biometrics</em>, <em>55</em>(4), 997–1004.</p>
</div>
<div id="ref-storey2003statistical">
<p>Storey, J. D., &amp; Tibshirani, R. (2003). Statistical significance for genomewide studies. <em>Proceedings of the National Academy of Sciences</em>, <em>100</em>(16), 9440–9445.</p>
</div>
<div id="ref-dray2015principal">
<p>Dray, S., &amp; Josse, J. (2015). Principal component analysis with missing values: A comparative survey of methods. <em>Plant Ecology</em>, <em>216</em>(5), 657–667.</p>
</div>
<div id="ref-bazin2010likelihood">
<p>Bazin, E., Dawson, K. J., &amp; Beaumont, M. A. (2010). Likelihood-free inference of population structure and local adaptation in a bayesian hierarchical model. <em>Genetics</em>, <em>185</em>(2), 587–602.</p>
</div>
<div id="ref-abraham2014fast">
<p>Abraham, G., &amp; Inouye, M. (2014). Fast principal component analysis of large-scale genome-wide data. <em>PloS One</em>, <em>9</em>(4), e93766.</p>
</div>
<div id="ref-halko2011finding">
<p>Halko, N., Martinsson, P.-G., &amp; Tropp, J. A. (2011). Finding structure with randomness: Probabilistic algorithms for constructing approximate matrix decompositions. <em>SIAM Review</em>, <em>53</em>(2), 217–288.</p>
</div>
</div>
<div class="footnotes">
<hr />
<ol start="15">
<li id="fn15"><p>ensemble des mécanismes grâce auxquels un organisme est capable de dégrader les graisses et de les assimiler<a href="section.html#fnref15">↩</a></p></li>
</ol>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="adaptation-locale-1.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="introgression-adaptative.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"download": [["thesis.pdf", "PDF"], ["thesis.epub", "EPUB"], ["thesis.docx", "Word"]],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(script.src))
      script.src  = script.src.replace(/^https?:/, '');
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
