<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>THÈSE</title>
  <meta name="description" content="THÈSE">
  <meta name="generator" content="bookdown 0.5 and GitBook 2.6.7">

  <meta property="og:title" content="THÈSE" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="THÈSE" />
  
  
  

<meta name="author" content="Keurcien LUU">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="conclusion.html">
<link rel="next" href="r-python.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />









<style type="text/css">
div.sourceCode { overflow-x: auto; }
table.sourceCode, tr.sourceCode, td.lineNumbers, td.sourceCode {
  margin: 0; padding: 0; vertical-align: baseline; border: none; }
table.sourceCode { width: 100%; line-height: 100%; }
td.lineNumbers { text-align: right; padding-right: 4px; padding-left: 4px; color: #aaaaaa; border-right: 1px solid #aaaaaa; }
td.sourceCode { padding-left: 5px; }
code > span.kw { color: #007020; font-weight: bold; } /* Keyword */
code > span.dt { color: #902000; } /* DataType */
code > span.dv { color: #40a070; } /* DecVal */
code > span.bn { color: #40a070; } /* BaseN */
code > span.fl { color: #40a070; } /* Float */
code > span.ch { color: #4070a0; } /* Char */
code > span.st { color: #4070a0; } /* String */
code > span.co { color: #60a0b0; font-style: italic; } /* Comment */
code > span.ot { color: #007020; } /* Other */
code > span.al { color: #ff0000; font-weight: bold; } /* Alert */
code > span.fu { color: #06287e; } /* Function */
code > span.er { color: #ff0000; font-weight: bold; } /* Error */
code > span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
code > span.cn { color: #880000; } /* Constant */
code > span.sc { color: #4070a0; } /* SpecialChar */
code > span.vs { color: #4070a0; } /* VerbatimString */
code > span.ss { color: #bb6688; } /* SpecialString */
code > span.im { } /* Import */
code > span.va { color: #19177c; } /* Variable */
code > span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code > span.op { color: #666666; } /* Operator */
code > span.bu { } /* BuiltIn */
code > span.ex { } /* Extension */
code > span.pp { color: #bc7a00; } /* Preprocessor */
code > span.at { color: #7d9029; } /* Attribute */
code > span.do { color: #ba2121; font-style: italic; } /* Documentation */
code > span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code > span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code > span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#la-genetique-des-populations"><i class="fa fa-check"></i><b>1.1</b> La génétique des populations</a><ul>
<li class="chapter" data-level="1.1.1" data-path="introduction.html"><a href="introduction.html#levolution-comme-point-de-depart"><i class="fa fa-check"></i><b>1.1.1</b> L’évolution comme point de départ</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#a-lorigine-de-la-variabilite-genetique"><i class="fa fa-check"></i><b>1.2</b> À l’origine de la variabilité génétique</a><ul>
<li class="chapter" data-level="1.2.1" data-path="introduction.html"><a href="introduction.html#la-theorie-de-levolution"><i class="fa fa-check"></i><b>1.2.1</b> La théorie de l’évolution</a></li>
<li class="chapter" data-level="1.2.2" data-path="introduction.html"><a href="introduction.html#levolution-dune-theorie"><i class="fa fa-check"></i><b>1.2.2</b> L’évolution d’une théorie</a></li>
<li class="chapter" data-level="1.2.3" data-path="introduction.html"><a href="introduction.html#forces-evolutives"><i class="fa fa-check"></i><b>1.2.3</b> Forces évolutives</a></li>
<li class="chapter" data-level="1.2.4" data-path="introduction.html"><a href="introduction.html#adaptation-locale"><i class="fa fa-check"></i><b>1.2.4</b> Adaptation locale</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#donnees-de-polymorphismes-genetiques"><i class="fa fa-check"></i><b>1.3</b> Données de polymorphismes génétiques</a><ul>
<li class="chapter" data-level="1.3.1" data-path="introduction.html"><a href="introduction.html#des-donnees-en-grande-dimension"><i class="fa fa-check"></i><b>1.3.1</b> Des données en grande dimension</a></li>
<li class="chapter" data-level="1.3.2" data-path="introduction.html"><a href="introduction.html#les-marqueurs-genetiques"><i class="fa fa-check"></i><b>1.3.2</b> Les marqueurs génétiques</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="introduction.html"><a href="introduction.html#les-motivations-de-la-these"><i class="fa fa-check"></i><b>1.4</b> Les motivations de la thèse</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="etat-de-lart.html"><a href="etat-de-lart.html"><i class="fa fa-check"></i><b>2</b> État de l’art</a><ul>
<li class="chapter" data-level="2.1" data-path="etat-de-lart.html"><a href="etat-de-lart.html#adaptation-locale-1"><i class="fa fa-check"></i><b>2.1</b> Adaptation locale</a><ul>
<li class="chapter" data-level="2.1.1" data-path="etat-de-lart.html"><a href="etat-de-lart.html#les-scans-genomiques"><i class="fa fa-check"></i><b>2.1.1</b> Les scans génomiques</a></li>
<li class="chapter" data-level="2.1.2" data-path="etat-de-lart.html"><a href="etat-de-lart.html#lindice-de-fixation"><i class="fa fa-check"></i><b>2.1.2</b> L’indice de fixation</a></li>
<li class="chapter" data-level="2.1.3" data-path="etat-de-lart.html"><a href="etat-de-lart.html#modele-de-flk"><i class="fa fa-check"></i><b>2.1.3</b> Modèle de FLK</a></li>
<li class="chapter" data-level="2.1.4" data-path="etat-de-lart.html"><a href="etat-de-lart.html#modele-de-outflank"><i class="fa fa-check"></i><b>2.1.4</b> Modèle de OutFLANK</a></li>
<li class="chapter" data-level="2.1.5" data-path="etat-de-lart.html"><a href="etat-de-lart.html#modele-du-logiciel-bayescan"><i class="fa fa-check"></i><b>2.1.5</b> Modèle du logiciel Bayescan</a></li>
<li class="chapter" data-level="2.1.6" data-path="etat-de-lart.html"><a href="etat-de-lart.html#fast-pca"><i class="fa fa-check"></i><b>2.1.6</b> Fast PCA</a></li>
<li class="chapter" data-level="2.1.7" data-path="etat-de-lart.html"><a href="etat-de-lart.html#analyse-en-composantes-principales-parcimonieuse"><i class="fa fa-check"></i><b>2.1.7</b> Analyse en Composantes Principales parcimonieuse</a></li>
<li class="chapter" data-level="2.1.8" data-path="etat-de-lart.html"><a href="etat-de-lart.html#bootstrap-acp"><i class="fa fa-check"></i><b>2.1.8</b> Bootstrap ACP</a></li>
<li class="chapter" data-level="2.1.9" data-path="etat-de-lart.html"><a href="etat-de-lart.html#contexte"><i class="fa fa-check"></i><b>2.1.9</b> Contexte</a></li>
<li class="chapter" data-level="2.1.10" data-path="etat-de-lart.html"><a href="etat-de-lart.html#tests-multiples"><i class="fa fa-check"></i><b>2.1.10</b> Tests multiples</a></li>
<li class="chapter" data-level="2.1.11" data-path="etat-de-lart.html"><a href="etat-de-lart.html#controle-du-taux-de-fausse-decouverte"><i class="fa fa-check"></i><b>2.1.11</b> Contrôle du taux de fausse découverte</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="etat-de-lart.html"><a href="etat-de-lart.html#introgression-adaptative"><i class="fa fa-check"></i><b>2.2</b> Introgression adaptative</a><ul>
<li class="chapter" data-level="2.2.1" data-path="etat-de-lart.html"><a href="etat-de-lart.html#coefficients-de-metissage-globaux-et-locaux"><i class="fa fa-check"></i><b>2.2.1</b> Coefficients de métissage globaux et locaux</a></li>
<li class="chapter" data-level="2.2.2" data-path="etat-de-lart.html"><a href="etat-de-lart.html#lien-entre-analyse-en-composantes-principales-et-metissage-global."><i class="fa fa-check"></i><b>2.2.2</b> Lien entre Analyse en Composantes Principales et métissage global.</a></li>
<li class="chapter" data-level="2.2.3" data-path="etat-de-lart.html"><a href="etat-de-lart.html#analyse-en-composantes-principales-locale"><i class="fa fa-check"></i><b>2.2.3</b> Analyse en Composantes Principales locale</a></li>
<li class="chapter" data-level="2.2.4" data-path="etat-de-lart.html"><a href="etat-de-lart.html#sensibilite-a-limputation-des-donnees-manquantes"><i class="fa fa-check"></i><b>2.2.4</b> Sensibilité à l’imputation des données manquantes</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="la-communalite-une-statistique-proche-de-la-f-st.html"><a href="la-communalite-une-statistique-proche-de-la-f-st.html"><i class="fa fa-check"></i><b>3</b> La communalité : une statistique proche de la <span class="math inline">\(F_{ST}\)</span></a><ul>
<li class="chapter" data-level="3.1" data-path="la-communalite-une-statistique-proche-de-la-f-st.html"><a href="la-communalite-une-statistique-proche-de-la-f-st.html#contributions"><i class="fa fa-check"></i><b>3.1</b> Contributions</a></li>
<li class="chapter" data-level="3.2" data-path="la-communalite-une-statistique-proche-de-la-f-st.html"><a href="la-communalite-une-statistique-proche-de-la-f-st.html#resume-de-larticle"><i class="fa fa-check"></i><b>3.2</b> Résumé de l’article</a></li>
<li class="chapter" data-level="3.3" data-path="la-communalite-une-statistique-proche-de-la-f-st.html"><a href="la-communalite-une-statistique-proche-de-la-f-st.html#article-1"><i class="fa fa-check"></i><b>3.3</b> Article 1</a><ul>
<li class="chapter" data-level="" data-path="la-communalite-une-statistique-proche-de-la-f-st.html"><a href="la-communalite-une-statistique-proche-de-la-f-st.html#abstract"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="la-communalite-une-statistique-proche-de-la-f-st.html"><a href="la-communalite-une-statistique-proche-de-la-f-st.html#significance-statement"><i class="fa fa-check"></i>Significance Statement</a></li>
<li class="chapter" data-level="" data-path="la-communalite-une-statistique-proche-de-la-f-st.html"><a href="la-communalite-une-statistique-proche-de-la-f-st.html#introduction-1"><i class="fa fa-check"></i>Introduction</a></li>
<li class="chapter" data-level="" data-path="la-communalite-une-statistique-proche-de-la-f-st.html"><a href="la-communalite-une-statistique-proche-de-la-f-st.html#new-method"><i class="fa fa-check"></i>New Method</a></li>
<li class="chapter" data-level="" data-path="la-communalite-une-statistique-proche-de-la-f-st.html"><a href="la-communalite-une-statistique-proche-de-la-f-st.html#results"><i class="fa fa-check"></i>Results</a></li>
<li class="chapter" data-level="" data-path="la-communalite-une-statistique-proche-de-la-f-st.html"><a href="la-communalite-une-statistique-proche-de-la-f-st.html#discussion"><i class="fa fa-check"></i>Discussion</a></li>
<li class="chapter" data-level="" data-path="la-communalite-une-statistique-proche-de-la-f-st.html"><a href="la-communalite-une-statistique-proche-de-la-f-st.html#materials-and-methods"><i class="fa fa-check"></i>Materials and Methods</a></li>
<li class="chapter" data-level="" data-path="la-communalite-une-statistique-proche-de-la-f-st.html"><a href="la-communalite-une-statistique-proche-de-la-f-st.html#acknowledgments"><i class="fa fa-check"></i>Acknowledgments</a></li>
<li class="chapter" data-level="3.3.1" data-path="la-communalite-une-statistique-proche-de-la-f-st.html"><a href="la-communalite-une-statistique-proche-de-la-f-st.html#analyses-supplementaires"><i class="fa fa-check"></i><b>3.3.1</b> Analyses supplémentaires</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="detection-doutliers.html"><a href="detection-doutliers.html"><i class="fa fa-check"></i><b>4</b> Détection d’outliers</a><ul>
<li class="chapter" data-level="4.1" data-path="detection-doutliers.html"><a href="detection-doutliers.html#article-2"><i class="fa fa-check"></i><b>4.1</b> Article 2</a><ul>
<li class="chapter" data-level="" data-path="detection-doutliers.html"><a href="detection-doutliers.html#abstract-1"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="detection-doutliers.html"><a href="detection-doutliers.html#introduction-2"><i class="fa fa-check"></i>Introduction</a></li>
<li class="chapter" data-level="" data-path="detection-doutliers.html"><a href="detection-doutliers.html#statistical-and-computational-approach"><i class="fa fa-check"></i>Statistical and computational approach</a></li>
<li class="chapter" data-level="" data-path="detection-doutliers.html"><a href="detection-doutliers.html#materials-and-methods-1"><i class="fa fa-check"></i>Materials and methods</a></li>
<li class="chapter" data-level="" data-path="detection-doutliers.html"><a href="detection-doutliers.html#results-1"><i class="fa fa-check"></i>Results</a></li>
<li class="chapter" data-level="" data-path="detection-doutliers.html"><a href="detection-doutliers.html#discussion-1"><i class="fa fa-check"></i>Discussion</a></li>
<li class="chapter" data-level="" data-path="detection-doutliers.html"><a href="detection-doutliers.html#acknowledgements"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="detection-doutliers.html"><a href="detection-doutliers.html#data-accessibility"><i class="fa fa-check"></i>Data accessibility</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="introgression-adaptative-1.html"><a href="introgression-adaptative-1.html"><i class="fa fa-check"></i><b>5</b> Introgression adaptative</a><ul>
<li class="chapter" data-level="5.1" data-path="introgression-adaptative-1.html"><a href="introgression-adaptative-1.html#simulations"><i class="fa fa-check"></i><b>5.1</b> Simulations</a><ul>
<li class="chapter" data-level="5.1.1" data-path="introgression-adaptative-1.html"><a href="introgression-adaptative-1.html#donnees-de-peupliers"><i class="fa fa-check"></i><b>5.1.1</b> Données de peupliers</a></li>
<li class="chapter" data-level="5.1.2" data-path="introgression-adaptative-1.html"><a href="introgression-adaptative-1.html#generation-aleatoire-dindividus-hybrides"><i class="fa fa-check"></i><b>5.1.2</b> Génération aléatoire d’individus hybrides</a></li>
<li class="chapter" data-level="5.1.3" data-path="introgression-adaptative-1.html"><a href="introgression-adaptative-1.html#simulations-a-partir-de-ms-et-seq-gen"><i class="fa fa-check"></i><b>5.1.3</b> Simulations à partir de ms et Seq-Gen</a></li>
<li class="chapter" data-level="5.1.4" data-path="introgression-adaptative-1.html"><a href="introgression-adaptative-1.html#resultats-de-la-comparaison-des-logiciels"><i class="fa fa-check"></i><b>5.1.4</b> Résultats de la comparaison des logiciels</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="introgression-adaptative-1.html"><a href="introgression-adaptative-1.html#article-3"><i class="fa fa-check"></i><b>5.2</b> Article 3</a><ul>
<li class="chapter" data-level="" data-path="introgression-adaptative-1.html"><a href="introgression-adaptative-1.html#introduction-3"><i class="fa fa-check"></i>Introduction</a></li>
<li class="chapter" data-level="" data-path="introgression-adaptative-1.html"><a href="introgression-adaptative-1.html#materials-and-methods-2"><i class="fa fa-check"></i>Materials and methods</a></li>
<li class="chapter" data-level="" data-path="introgression-adaptative-1.html"><a href="introgression-adaptative-1.html#results-2"><i class="fa fa-check"></i>Results</a></li>
<li class="chapter" data-level="" data-path="introgression-adaptative-1.html"><a href="introgression-adaptative-1.html#discussion-2"><i class="fa fa-check"></i>Discussion</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="aspect-computationnel.html"><a href="aspect-computationnel.html"><i class="fa fa-check"></i><b>6</b> Aspect computationnel</a><ul>
<li class="chapter" data-level="6.1" data-path="aspect-computationnel.html"><a href="aspect-computationnel.html#donnees-dentree"><i class="fa fa-check"></i><b>6.1</b> Données d’entrée</a></li>
<li class="chapter" data-level="6.2" data-path="aspect-computationnel.html"><a href="aspect-computationnel.html#memory-mapping"><i class="fa fa-check"></i><b>6.2</b> Memory-mapping</a></li>
<li class="chapter" data-level="6.3" data-path="aspect-computationnel.html"><a href="aspect-computationnel.html#interface-shiny"><i class="fa fa-check"></i><b>6.3</b> Interface Shiny</a></li>
<li class="chapter" data-level="6.4" data-path="aspect-computationnel.html"><a href="aspect-computationnel.html#acp-et-donnees-manquantes"><i class="fa fa-check"></i><b>6.4</b> ACP et données manquantes</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="conclusion.html"><a href="conclusion.html"><i class="fa fa-check"></i>Conclusion</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="supplementary-information.html"><a href="supplementary-information.html"><i class="fa fa-check"></i><b>A</b> Supplementary Information</a><ul>
<li class="chapter" data-level="" data-path="supplementary-information.html"><a href="supplementary-information.html#article-1-1"><i class="fa fa-check"></i>Article 1 </a></li>
<li class="chapter" data-level="" data-path="supplementary-information.html"><a href="supplementary-information.html#article-2-1"><i class="fa fa-check"></i>Article 2</a></li>
<li class="chapter" data-level="" data-path="supplementary-information.html"><a href="supplementary-information.html#article-3-1"><i class="fa fa-check"></i>Article 3</a></li>
</ul></li>
<li class="chapter" data-level="B" data-path="r-python.html"><a href="r-python.html"><i class="fa fa-check"></i><b>B</b> R &amp; Python</a><ul>
<li class="chapter" data-level="B.1" data-path="r-python.html"><a href="r-python.html#simulations-et-modeles-demographiques"><i class="fa fa-check"></i><b>B.1</b> Simulations et modèles démographiques</a><ul>
<li class="chapter" data-level="B.1.1" data-path="r-python.html"><a href="r-python.html#modele-en-ile"><i class="fa fa-check"></i><b>B.1.1</b> Modèle en île</a></li>
<li class="chapter" data-level="B.1.2" data-path="r-python.html"><a href="r-python.html#modele-de-divergence"><i class="fa fa-check"></i><b>B.1.2</b> Modèle de divergence</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="bibliographie.html"><a href="bibliographie.html"><i class="fa fa-check"></i>Bibliographie</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">THÈSE</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="supplementary-information" class="section level1">
<h1><span class="header-section-number">A</span> Supplementary Information</h1>
<div id="article-1-1" class="section level2 unnumbered">
<h2>Article 1 </h2>
</div>
<div id="article-2-1" class="section level2 unnumbered">
<h2>Article 2</h2>

<div class="figure" style="text-align: center"><span id="fig:FigureSI1"></span>
<img src="figure/FigureSI1.png" alt="Schematic description of the island and divergence model. For the island model, adaptation occurs simultaneously in each population. For the island model, adaptation takes place in the branch leading to the second population." width="\textwidth" />
<p class="caption">
Figure A.1: Schematic description of the island and divergence model. For the island model, adaptation occurs simultaneously in each population. For the island model, adaptation takes place in the branch leading to the second population.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:FigureSI2"></span>
<img src="thesis_files/figure-html/FigureSI2-1.png" alt="Proportion of false discoveries and statistical power as a function of the number of principal components in a model of range expansion." width="\textwidth" />
<p class="caption">
Figure A.2: Proportion of false discoveries and statistical power as a function of the number of principal components in a model of range expansion.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:FigureSI3"></span>
<img src="thesis_files/figure-html/FigureSI3-1.png" alt="Statistical power averaged over the expected proportion of false discoveries (ranging between 0% and 50%) for the island model." width="\textwidth" />
<p class="caption">
Figure A.3: Statistical power averaged over the expected proportion of false discoveries (ranging between 0% and 50%) for the island model.
</p>
</div>


<div class="figure" style="text-align: center"><span id="fig:FigureSI4"></span>
<img src="thesis_files/figure-html/FigureSI4-1.png" alt="Statistical power as a function of the proportion of false discoveries for the island model." width="\textwidth" />
<p class="caption">
Figure A.4: Statistical power as a function of the proportion of false discoveries for the island model.
</p>
</div>


<div class="figure" style="text-align: center"><span id="fig:FigureSI5"></span>
<img src="thesis_files/figure-html/FigureSI5-1.png" alt="Statistical power as a function of the proportion of false discoveries for the divergence model." width="\textwidth" />
<p class="caption">
Figure A.5: Statistical power as a function of the proportion of false discoveries for the divergence model.
</p>
</div>


<div class="figure" style="text-align: center"><span id="fig:FigureSI6"></span>
<img src="thesis_files/figure-html/FigureSI6-1.png" alt="Statistical power as a function of the proportion of false discoveries for the model of range expansion." width="\textwidth" />
<p class="caption">
Figure A.6: Statistical power as a function of the proportion of false discoveries for the model of range expansion.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:FigureSI7"></span>
<img src="thesis_files/figure-html/FigureSI7-1.png" alt="Running times of the different computer programs. The different programs were run on genotype matrices containing 300 individuals and from 500 to 50,000 SNPs. The characteristics of the computer we used to perform comparisons are the following: OSX El Capitan 10.11.3, 2,5 GHz Intel Core i5, 8 Go 1600 MHz DDR3." width="\textwidth" />
<p class="caption">
Figure A.7: Running times of the different computer programs. The different programs were run on genotype matrices containing 300 individuals and from 500 to 50,000 SNPs. The characteristics of the computer we used to perform comparisons are the following: OSX El Capitan 10.11.3, 2,5 GHz Intel Core i5, 8 Go 1600 MHz DDR3.
</p>
</div>
</div>
<div id="article-3-1" class="section level2 unnumbered">
<h2>Article 3</h2>

<div class="figure" style="text-align: center"><span id="fig:ancestry"></span>
<img src="figure/ancestry.png" alt="Standardized average ancestry coefficient computed with pcadapt and different LAI methods (EILA, Loter and RFMix) for a population of simulated admixed individuals. Simulations use phased genotype data from \(25\) Populus balsamifera individuals and \(25\) Populus trichocarpa individuals and assume that admixture took place \(\lambda=100\) generations ago. A total of \(25\) admixed individuals were generated assuming \(30\%\) of Populus balsamifera ancestry except from the 5 outlier \(500\) SNP regions where Populus balsamifera ancestry is of \(50\%\)." width="\textwidth" />
<p class="caption">
Figure A.8: Standardized average ancestry coefficient computed with pcadapt and different LAI methods (EILA, Loter and RFMix) for a population of simulated admixed individuals. Simulations use phased genotype data from <span class="math inline">\(25\)</span> <em>Populus balsamifera</em> individuals and <span class="math inline">\(25\)</span> <em>Populus trichocarpa</em> individuals and assume that admixture took place <span class="math inline">\(\lambda=100\)</span> generations ago. A total of <span class="math inline">\(25\)</span> admixed individuals were generated assuming <span class="math inline">\(30\%\)</span> of <em>Populus balsamifera</em> ancestry except from the 5 outlier <span class="math inline">\(500\)</span> SNP regions where <em>Populus balsamifera</em> ancestry is of <span class="math inline">\(50\%\)</span>.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:losspower-admix"></span>
<img src="figure/meanloss_admixture.png" alt="Relative loss of power of the different methods when compared to an ideal method called oracle, which would know ancestry chunks for each admixed individual. The relative power is averaged over the difference \(\Delta_q\) of ancestry between neutral and outlier regions. Simulations use phased genotype data from \(25\) Populus balsamifera individuals and \(25\) Populus trichocarpa individuals and assume that admixture took place \(\lambda\) generations ago. A total of \(25\) admixed individuals were generated assuming \(70\%\) of Populus balsamifera ancestry except from the 5 outlier \(500\) SNP regions where Populus balsamifera ancestry is of \(70\%-\Delta_q\) where \(\Delta_q\) is equal to \(0.05\), \(0.10\), \(0.15\), \(0.20\), \(0.30\), \(0.40\), or \(0.50\)." width="\textwidth" />
<p class="caption">
Figure A.9: Relative loss of power of the different methods when compared to an ideal method called <em>oracle</em>, which would know ancestry chunks for each admixed individual. The relative power is averaged over the difference <span class="math inline">\(\Delta_q\)</span> of ancestry between neutral and outlier regions. Simulations use phased genotype data from <span class="math inline">\(25\)</span> <em>Populus balsamifera</em> individuals and <span class="math inline">\(25\)</span> <em>Populus trichocarpa</em> individuals and assume that admixture took place <span class="math inline">\(\lambda\)</span> generations ago. A total of <span class="math inline">\(25\)</span> admixed individuals were generated assuming <span class="math inline">\(70\%\)</span> of <em>Populus balsamifera</em> ancestry except from the 5 outlier <span class="math inline">\(500\)</span> SNP regions where <em>Populus balsamifera</em> ancestry is of <span class="math inline">\(70\%-\Delta_q\)</span> where <span class="math inline">\(\Delta_q\)</span> is equal to <span class="math inline">\(0.05\)</span>, <span class="math inline">\(0.10\)</span>, <span class="math inline">\(0.15\)</span>, <span class="math inline">\(0.20\)</span>, <span class="math inline">\(0.30\)</span>, <span class="math inline">\(0.40\)</span>, or <span class="math inline">\(0.50\)</span>.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:power-admix-max"></span>
<img src="figure/power_Dq_max.png" alt="Proportion of true outlier peaks among the five top peaks found with pcadapt and different LAI methods (EILA, Loter and RFMix) in a scenario where 2 Populus populations experienced admixture. Compared to Figure 5.7, we compute maximum of ancestry coefficients within each genomic region instead of mean of ancestry coefficients. Proportion of true outlier peaks are displayed as a function of the difference \(\Delta_q\) of ancestry between outlier and neutral regions. The three panels correspond to the three different possible values (\(\lambda=10\) or \(100\) or \(1000\)) of the number of generations since admixture." width="\textwidth" />
<p class="caption">
Figure A.10: Proportion of true outlier peaks among the five top peaks found with pcadapt and different LAI methods (EILA, Loter and RFMix) in a scenario where 2 Populus populations experienced admixture. Compared to Figure <a href="introgression-adaptative-1.html#fig:power-admix">5.7</a>, we compute maximum of ancestry coefficients within each genomic region instead of mean of ancestry coefficients. Proportion of true outlier peaks are displayed as a function of the difference <span class="math inline">\(\Delta_q\)</span> of ancestry between outlier and neutral regions. The three panels correspond to the three different possible values (<span class="math inline">\(\lambda=10\)</span> or <span class="math inline">\(100\)</span> or <span class="math inline">\(1000\)</span>) of the number of generations since admixture.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:si-power-admix"></span>
<img src="figure/power_Dq_curves.png" alt="Proportion of true outlier peaks as a function of the number of top peaks found with pcadapt and different LAI methods (EILA, Loter and RFMix) in a scenario where 2 Populus populations experienced admixture. The different panels correspond to the different possible values for the number of generations since admixture occurred (\(\lambda=\{10,100,1000\}\)) and to the different values of the difference of ancestry \(\Delta_q\) between neutral and outlier regions." width="\textwidth" />
<p class="caption">
Figure A.11: Proportion of true outlier peaks as a function of the number of top peaks found with pcadapt and different LAI methods (EILA, Loter and RFMix) in a scenario where 2 Populus populations experienced admixture. The different panels correspond to the different possible values for the number of generations since admixture occurred (<span class="math inline">\(\lambda=\{10,100,1000\}\)</span>) and to the different values of the difference of ancestry <span class="math inline">\(\Delta_q\)</span> between neutral and outlier regions.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:introgression"></span>
<img src="figure/introgression.png" alt="Schematic description of simulations under an introgression scenario." width="\textwidth" />
<p class="caption">
Figure A.12: Schematic description of simulations under an introgression scenario.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:admixture"></span>
<img src="figure/admixture.png" alt="Schematic description of simulations under an admixture scenario." width="\textwidth" />
<p class="caption">
Figure A.13: Schematic description of simulations under an admixture scenario.
</p>
</div>

<table>
<caption><span id="tab:annotation">Table A.1: </span>Transcripts detected with pcadapt in chromosome 6 and not detected in <span class="citation">(Suarez-Gonzalez, <a href="#ref-suarez2016">2016</a>)</span> located in regions with excess of <em>Populus balsamifera</em> ancestry.</caption>
<thead>
<tr class="header">
<th align="left">Transcript</th>
<th align="left">Region</th>
<th align="left">PANTHER annotation</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Potri.006G156000.1</td>
<td align="left">14266001:14305734</td>
<td align="left">Sulfate transporter</td>
</tr>
<tr class="even">
<td align="left">Potri.006G156100.1</td>
<td align="left">14266001:14305734</td>
<td align="left">Not named</td>
</tr>
<tr class="odd">
<td align="left">Potri.006G156200.1</td>
<td align="left">14266001:14305734</td>
<td align="left">Not named</td>
</tr>
<tr class="even">
<td align="left">Potri.006G161800.1</td>
<td align="left">15556790:15791830</td>
<td align="left">Not named</td>
</tr>
<tr class="odd">
<td align="left">Potri.006G161900.1</td>
<td align="left">15556790:15791830</td>
<td align="left">Vernalization intensive protein</td>
</tr>
</tbody>
</table>
</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-suarez2016">
<p>Suarez-Gonzalez, et a., Adriana. (2016). Genomic and functional approaches reveal a case of adaptive introgression from populus balsamifera (balsam poplar) in p. trichocarpa (black cottonwood). <em>Molecular Ecology</em>, 2427–2442.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="conclusion.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="r-python.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"download": [["thesis.pdf", "PDF"], ["thesis.epub", "EPUB"], ["thesis.docx", "Word"]],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(script.src))
      script.src  = script.src.replace(/^https?:/, '');
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
